<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html dir="ltr" lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
 <head>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
  <meta content="Henipavirus,Autopsy,Meningitis,Mackay, Queensland,Outbreak,Australia,Seroprevalence,Cairns, Queensland,Haemorrhage,Pulmonary oedema,Sunshine Coast, Queensland" name="keywords" />
  <link href="../../favicon.ico" rel="shortcut icon" />
  <link href="../../wp/w/Wikipedia_Text_of_the_GNU_Free_Documentation_License.htm" rel="copyright" />
  <title>Henipavirus</title>
  <style media="screen,projection" type="text/css">/*<![CDATA[*/ @import "../../css/wp-monobook-main.css"; /*]]>*/</style>
  <link href="../../css/wp-commonPrint.css" media="print" rel="stylesheet" type="text/css" />
  <!--[if lt IE 5.5000]><style type="text/css">@import "../../css/IE50Fixes.css";</style><![endif]-->
  <!--[if IE 5.5000]><style type="text/css">@import "../../css/IE55Fixes.css";</style><![endif]-->
  <!--[if IE 6]><style type="text/css">@import "../../css/IE60Fixes.css";</style><![endif]-->
  <!--[if IE 7]><style type="text/css">@import "../../css/IE70Fixes.css";</style><![endif]-->
  <!--[if lt IE 7]><script type="text/javascript" src="../../js/IEFixes.js"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->
  <script type="text/javascript">
			var skin = "monobook";
			var stylepath = "/skins-1.5";

			var wgArticlePath = "/wiki/$1";
			var wgScriptPath = "/w";
			var wgServer = "http://en.wikipedia.org";
                        
			var wgCanonicalNamespace = "";
			var wgNamespaceNumber = 0;
			var wgPageName = "Henipavirus";
			var wgTitle = "Henipavirus";
			var wgArticleId = 333398;
			var wgIsArticle = true;
                        
			var wgUserName = null;
			var wgUserLanguage = "en";
			var wgContentLanguage = "en";
		</script>
  <script src="../../js/wikibits.js" type="text/javascript"><!-- wikibits js --></script>
  <script src="../../js/wp.js" type="text/javascript"><!-- site js --></script>
  <style type="text/css">/*<![CDATA[*/
@import "../../css/wp-common.css";
@import "../../css/wp-monobook.css";
@import "../../css/wp.css";
/*]]>*/</style>
  <!-- Head Scripts -->
 </head>
 <body class="mediawiki ns-0 ltr page-Henipavirus">
  <div id="globalWrapper">
   <div id="column-content">
    <div id="content"><a id="top" name="top"></a><h1 class="firstHeading">Henipavirus</h1>
     <div id="bodyContent">
      <h3 id="siteSub"><a href="../../index.htm">2007 Schools Wikipedia Selection</a>. Related subjects: <a href="../index/subject.Science.Biology.Organisms.htm">Organisms</a></h3>
      <!-- start content -->
      <table border="1" cellpadding="0" style="position:relative; margin: 0 0 0.5em 1em; border-collapse: collapse; float:right; clear:right; width:200px;">
       <tr style="text-align:center;">
        <th style="background: violet;">
         <div style="float:right; font-size:70%; padding:0 .5em 0 2em"><!--del_lnk--> ?</div><b>Henipaviruses</b></th>
       </tr>
       <tr style="text-align:center;">
        <th style="background: violet;"><b><!--del_lnk--> Virus classification</b></th>
       </tr>
       <tr style="text-align:center;">
        <td>
         <table cellpadding="2" style="margin:0 auto; text-align:left; background:none;">
          <tr>
           <td>Group:</td>
           <td>Group V <small>(<!--del_lnk--> (-)ssRNA)</small></td>
          </tr>
          <tr valign="top">
           <td>Order:</td>
           <td><i><!--del_lnk--> Mononegavirales</i></td>
          </tr>
          <tr valign="top">
           <td>Family:</td>
           <td><i><!--del_lnk--> Paramyxoviridae</i></td>
          </tr>
          <tr valign="top">
           <td>Genus:</td>
           <td><i><b>Henipavirus</b></i></td>
          </tr>
         </table>
        </td>
       </tr>
       <tr style="text-align:center; background:violet;">
        <th><b>Species</b></th>
       </tr>
       <tr>
        <td style="padding: 0 .5em;">
         <p><i><!--del_lnk--> Hendravirus</i><br /><i><!--del_lnk--> Nipahvirus</i></td>
       </tr>
      </table>
      <p><b>Henipavirus</b> is a <!--del_lnk--> genus of the <!--del_lnk--> family <i><!--del_lnk--> Paramyxoviridae</i>, <!--del_lnk--> order <i><!--del_lnk--> Mononegavirales</i> containing two members, <b>Hendravirus</b> and <b>Nipahvirus</b>. The henipaviruses are naturally harboured by <!--del_lnk--> Pteropid <!--del_lnk--> fruit bats (flying foxes) and are characterised by a large <!--del_lnk--> genome, a wide host range and their recent emergence as <!--del_lnk--> zoonotic pathogens capable of causing illness and death in <!--del_lnk--> domestic animals and <a href="../../wp/h/Human.htm" title="Human">humans</a>.<p>
       <script type="text/javascript">
//<![CDATA[
 if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } 
//]]>
</script><a id="Virus_structure" name="Virus_structure"></a><h2> <span class="mw-headline">Virus structure</span></h2>
      <div class="thumb tright">
       <div style="width:352px;"><a class="internal" href="../../images/187/18791.png.htm" title="Structure of henipaviruses"><img alt="Structure of henipaviruses" height="267" longdesc="/wiki/Image:Henipavirus_structure.png" src="../../images/187/18791.png" width="350" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/187/18791.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Structure of henipaviruses</div>
       </div>
      </div>
      <div class="thumb tright">
       <div style="width:327px;"><a class="internal" href="../../images/187/18792.png.htm" title="The henipavirus genome (3&rsquo; to 5&rsquo; orientation) and products of the P gene"><img alt="The henipavirus genome (3&rsquo; to 5&rsquo; orientation) and products of the P gene" height="142" longdesc="/wiki/Image:Henipavirus_genome.png" src="../../images/187/18792.png" width="325" /></a><div class="thumbcaption">The henipavirus genome (3&rsquo; to 5&rsquo; orientation) and products of the P gene</div>
       </div>
      </div>
      <p>Henipaviruses are <i><b>pleomorphic</b></i> (variably shaped), ranging in size from 40 to 600 nm in diameter . They possess a <a href="../../wp/l/Lipid.htm" title="Lipid">lipid</a> membrane overlying a shell of viral <!--del_lnk--> matrix protein. At the core is a single helical strand of genomic <!--del_lnk--> RNA tightly bound to N (nucleocapsid) protein and associated with the L (large) and P (phosphoprotein) proteins which provide <!--del_lnk--> RNA polymerase activity during replication.<p>Embedded within the lipid membrane are spikes of F (fusion) protein trimers and G (attachment) protein tetramers. The function of the G protein is to attach the virus to the surface of a host cell via <!--del_lnk--> ephrin B2, a highly conserved protein present in many mammals . The F protein fuses the viral membrane with the host cell membrane, releasing the virion contents into the cell. It also causes infected cells to fuse with neighbouring cells to form large, multinucleated <!--del_lnk--> syncytia.<p><a id="Genome_structure" name="Genome_structure"></a><h2> <span class="mw-headline">Genome structure</span></h2>
      <p>As with all viruses in the <i>Mononegavirales</i> order, the Hendra virus and Nipah virus genomes are non-segmented, single-stranded <!--del_lnk--> negative-sense RNA. Both genomes are 18.2 kb in size and contain six <!--del_lnk--> genes corresponding to six structural proteins .<p>In common with other members of the <i>Paramyxovirinae</i> subfamily, the number of <!--del_lnk--> nucleotides in the henipavirus genome is a multiple of six, known as the &#39;rule of six&#39;. Deviation from the rule of six, through mutation or incomplete genome synthesis, leads to inefficient viral replication, probably due to structural constraints imposed by the binding between the RNA and the N protein.<p>Henipaviruses employ an unusual process called <!--del_lnk--> RNA editing to generate mulitple proteins from a single gene. The process involves the insertion of extra <!--del_lnk--> guanosine residues into the P gene <!--del_lnk--> mRNA prior to <!--del_lnk--> translation. The number of residues added determines whether the P, V or W proteins are synthesised. The functions of the V and W proteins are unknown, but they may be involved in disrupting host antiviral mechanisms.<p><a id="Hendra_virus" name="Hendra_virus"></a><h2> <span class="mw-headline">Hendra virus</span></h2>
      <p><a id="Emergence" name="Emergence"></a><h3> <span class="mw-headline">Emergence</span></h3>
      <p>Hendra virus (originally <i>Equine <!--del_lnk--> morbillivirus</i>) was discovered in September <!--del_lnk--> 1994 when it caused the deaths of thirteen <a href="../../wp/h/Horse.htm" title="Horse">horses</a>, and a trainer at a training complex in <!--del_lnk--> Hendra, a suburb of <a href="../../wp/b/Brisbane.htm" title="Brisbane">Brisbane</a> in <!--del_lnk--> Queensland, <a href="../../wp/a/Australia.htm" title="Australia">Australia</a>.<p>The index case, a mare, was housed with 23 other horses after falling ill and died two days later. Subsequently, 19 of the remaining horses succumbed with 12 dying. Both the trainer and a stable hand were involved in nursing the index case and both fell ill within one week of the horse&rsquo;s death with an influenza-like illness. The stable hand recovered while the trainer died of respiratory and renal failure. The source of virus was most likely frothy nasal discharge from the index case.<p>A second <!--del_lnk--> outbreak occurred in August 1994 (chronologically preceding the first outbreak) in <!--del_lnk--> Mackay 1000km north of Brisbane resulting in the deaths of two horses and their owner . The owner assisted in <!--del_lnk--> autopsies of the horses and within three weeks was admitted to hospital suffering from <a href="../../wp/m/Meningitis.htm" title="Meningitis">meningitis</a>. He recovered, but 14 months later developed neurologic signs and died. This outbreak was diagnosed retrospectively by the presence of Hendra virus in the brain of the patient.<p>A survey of wildlife in the outbreak areas was conducted and identified pteropid fruit bats as the most likely source of Hendra virus with a <!--del_lnk--> seroprevalence of 47%. All of the other 46 species sampled were negative. Virus isolations from the reproductive tract and urine of wild bats indicated that transmission to horses may have occurred via exposure to bat urine or birthing fluids .<p><a id="Outbreaks" name="Outbreaks"></a><h3> <span class="mw-headline">Outbreaks</span></h3>
      <p>Four more incidents, in <!--del_lnk--> Cairns in January <!--del_lnk--> 1999 and October 2004, in <!--del_lnk--> Townsville in December 2004 and on the <!--del_lnk--> Sunshine Coast in June 2006 each resulted in the death of one horse. A vet involved in autopsy of the horse from the 2004 Townsville incident developed a Hendra-related illness soon after and recovered.<p>The distribution of black and spectacled flying foxes covers Townsville and Cairns, and the timing of incidents indicates a seasonal pattern of outbreaks possibly related to the seasonality of fruit bat birthing. As there is no evidence of transmission to humans directly from bats, it is thought that human infection only occurs via an intermediate host.<p><a id="Pathology" name="Pathology"></a><h3> <span class="mw-headline">Pathology</span></h3>
      <p>Flying foxes are unaffected by Hendra virus infection. Symptoms of Hendra virus infection of humans may be respiratory, including <!--del_lnk--> haemorrhage and <!--del_lnk--> oedema of the lungs, or <!--del_lnk--> encephalitic resulting in <a href="../../wp/m/Meningitis.htm" title="Meningitis">meningitis</a>. In horses, infection usually causes pulmonary oedema and congestion.<p><a id="Nipah_virus" name="Nipah_virus"></a><h2> <span class="mw-headline">Nipah virus</span></h2>
      <p><a id="Emergence_2" name="Emergence_2"></a><h3> <span class="mw-headline">Emergence</span></h3>
      <div class="thumb tright">
       <div style="width:182px;"><a class="internal" href="../../images/187/18793.jpg.htm" title="Pteropus vampyrus (Malayan flying fox), one of the natural reservoirs of Nipah virus"><img alt="Pteropus vampyrus (Malayan flying fox), one of the natural reservoirs of Nipah virus" height="262" longdesc="/wiki/Image:Pteropus_vampyrus1.jpg" src="../../images/187/18793.jpg" width="180" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/187/18793.jpg.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div><i>Pteropus vampyrus</i> (Malayan flying fox), one of the natural reservoirs of Nipah virus</div>
       </div>
      </div>
      <p>Nipah virus was identified in 1999 when it caused an outbreak of neurological and respiratory disease on pig farms in peninsular <a href="../../wp/m/Malaysia.htm" title="Malaysia">Malaysia</a>, resulting in 105 human deaths and the culling of one million pigs . In <a href="../../wp/s/Singapore.htm" title="Singapore">Singapore</a>, 11 cases including one death occurred in abattoir workers exposed to pigs imported from the affected Malaysian farms.<p>Symptoms of infection from the Malaysian outbreak were primarily encephalitic in humans and respiratory in pigs. Later outbreaks have caused respiratory illness in humans, increasing the likelihood of human-to-human transmission and indicating the existence of more dangerous strains of the virus.<p>Based on seroprevalence data and virus isolations, the primary reservoir for Nipah virus was identified as Pteropid fruit bats including <i>Pteropus vampyrus</i> (Malayan flying fox) and <i>Pteropus hypomelanus</i> (Island flying fox), both of which occur in Malaysia.<p>The transmission of Nipah virus from flying foxes to pigs is thought to be due to an increasing overlap between bat habitats and piggeries in peninsular Malaysia. At the index farm, fruit orchards were in close proximity to the piggery, allowing the spillage of urine, faeces and partially eaten fruit onto the pigs . Retrospective studies demonstrate that viral spillover into pigs may have been occurring in Malaysia since 1996 without detection . During 1998, viral spread was aided by the transfer of infected pigs to other farms where new outbreaks occurred.<p><a id="Outbreaks_2" name="Outbreaks_2"></a><h3> <span class="mw-headline">Outbreaks</span></h3>
      <p>Six more outbreaks of Nipah virus have occurred since <!--del_lnk--> 1998, one in <a href="../../wp/i/India.htm" title="India">India</a> and five in <a href="../../wp/b/Bangladesh.htm" title="Bangladesh">Bangladesh</a>. All outbreak sites lie within the range of Pteropus species (<i>Pteropus giganteus</i>). As with Hendra virus, the timing of the outbreaks indicates a seasonal effect.<div class="thumb tright">
       <div style="width:352px;"><a class="internal" href="../../images/187/18794.png.htm" title="Locations of henipavirus outbreaks (red stars&ndash;Hendra virus; blue stars&ndash;Nipah virus) and distribution of henipavirus flying fox reservoirs (red shading&ndash;Hendra virus&nbsp;; blue shading&ndash;Nipah virus)"><img alt="Locations of henipavirus outbreaks (red stars&ndash;Hendra virus; blue stars&ndash;Nipah virus) and distribution of henipavirus flying fox reservoirs (red shading&ndash;Hendra virus&nbsp;; blue shading&ndash;Nipah virus)" height="230" longdesc="/wiki/Image:Flying_fox_distribution.png" src="../../images/187/18794.png" width="350" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/187/18794.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Locations of henipavirus outbreaks (red stars&ndash;Hendra virus; blue stars&ndash;Nipah virus) and distribution of henipavirus flying fox reservoirs (red shading&ndash;Hendra virus&nbsp;; blue shading&ndash;Nipah virus)</div>
       </div>
      </div>
      <ul>
       <li>2001 January 31 &ndash; February 23, <!--del_lnk--> Siliguri, India: 66 cases with a 74% mortality rate . 75% of patients were either hospital staff or had visited one of the other patients in hospital, indicating person-to-person transmission.</ul>
      <ul>
       <li>2001 April &ndash; May, <!--del_lnk--> Meherpur district, Bangladesh: 13 cases with nine fatalities (69% mortality) .</ul>
      <ul>
       <li>2003 January, <!--del_lnk--> Naogaon district, Bangladesh: 12 cases with eight fatalities (67% mortality) .</ul>
      <ul>
       <li>2004 January &ndash; February, <!--del_lnk--> Manikganj and <!--del_lnk--> Rajbari provinces, Bangladesh: 42 cases with 14 fatalities (33% mortality).</ul>
      <ul>
       <li>2004 <!--del_lnk--> 19 February &ndash; <!--del_lnk--> 16 April, <!--del_lnk--> Faridpur district, Bangladesh: 36 cases with 27 fatalities (75% mortality). Epidemiological evidence strongly suggests that this outbreak involved person-to-person transmission of Nipah virus, which had not previously been confirmed . 92% of cases involved close contact with at least one other person infected with Nipah virus. Two cases involved a single short exposure to an ill patient, including a rickshaw driver who transported a patient to hospital. In addition, at least six cases involved acute respiratory distress syndrome which has not been reported previously for Nipah virus illness in humans. This symptom is likely to have assisted human-to-human transmission through large droplet dispersal.</ul>
      <ul>
       <li>2005 January, <!--del_lnk--> Tangail district, Bangladesh: 32 cases with 12 fatalities (38% mortality). The virus was probably contracted from drinking date palm juice contaminated by fruit bat droppings or saliva .</ul>
      <p>Eleven isolated cases of Nipah virus encephalitis have also been documented in Bangladesh since 2001.<p>Nipah virus has been isolated from Lyle&#39;s flying fox (<i>Pteropus lylei</i>) in <a href="../../wp/c/Cambodia.htm" title="Cambodia">Cambodia</a>  and viral RNA found in urine and saliva from <i>P. lylei</i> and Horsfield&#39;s roundleaf bat (<i>Hipposideros larvartus</i>) from <a href="../../wp/t/Thailand.htm" title="Thailand">Thailand</a> . The Cambodian strain shows 98% identity with the virus causing the 1998 outbreak. No infection of humans or other species have been observed in Cambodia or Thailand.<p><a id="Pathology_2" name="Pathology_2"></a><h3> <span class="mw-headline">Pathology</span></h3>
      <p>In humans, the infection presents as fever, headache and drowsiness. Cough, abdominal pain, nausea, vomiting, weakness, problems with swallowing and blurred vision are relatively common. About a quarter of the patients have seizures and about 60% become comatose and might need mechanical ventilation. In patients with severe disease, their conscious state may deteriorate and they may develop severe hypertension, fast heart rate, and very high temperature.<p>Nipah virus is also known to cause relapse encephalitis. In the initial Malaysian outbreak, a patient presented with relapse encephalitis some 53 months after his initial infection. There is no definitive treatment for Nipah encephalitis, apart from supportive measures, such as mechanical ventilation and prevention of secondary infection. <!--del_lnk--> Ribavirin, an antiviral drug, was tested in the Malaysian outbreak and the results were encouraging, though further studies are still needed.<p>In animals, especially in pigs, the virus causes porcine respiratory and neurologic syndrome also known as barking pig syndrome or one mile cough.<p><a id="Causes_of_Emergence" name="Causes_of_Emergence"></a><h2> <span class="mw-headline">Causes of Emergence</span></h2>
      <p>The emergence of henipaviruses parallels the emergence of other zoonotic viruses in recent decades. <!--del_lnk--> SARS coronavirus, <!--del_lnk--> Australian bat lyssavirus, <!--del_lnk--> Menangle virus and probably <!--del_lnk--> Ebola virus and <!--del_lnk--> Marburg virus are also harboured by bats and are capable of infecting a variety of other species. The emergence of each of these viruses has been linked to an increase in contact between bats and humans, sometimes involving an intermediate domestic animal host. The increased contact is driven both by human encroachment into the bats&rsquo; territory (in the case of Nipah, specifically pigpens in said territory) and by movement of bats towards human populations due to changes in food distribution and loss of habitat.<p>There is evidence of habitat loss for flying foxes both in South Asia and Australia (particularly along the east coast) as well as encroachment of human dwellings and agriculture into the remaining habitats, creating greater overlap of human and flying fox distributions.<div class="printfooter"> Retrieved from &quot;<!--del_lnk--> http://en.wikipedia.org/wiki/Henipavirus&quot;</div>
      <!-- end content -->
      <div class="visualClear">
      </div>
     </div>
    </div>
   </div>
   <!-- end of the left (by default at least) column -->
   <div class="visualClear">
   </div>
   <div id="footer">
    <div class="center"> This reference article is mainly selected from the English Wikipedia with only minor checks and changes (see www.wikipedia.org for details of authors and sources) and is available under the <nobr><a href="../../wp/w/Wikipedia_Text_of_the_GNU_Free_Documentation_License.htm">GNU Free Documentation License</a></nobr>. See also our <b><a href="../../disclaimer.htm">Disclaimer</a></b>. </div>
   </div>
   <script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
  </div>
  
 </body>
</html>
