<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html dir="ltr" lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
 <head>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
  <meta content="Bioinformatics,Ecology,Cell biology,Botany,Developmental biology,Evolutionary biology,Marine biology,Genomics,Genetics,Biochemistry,Astrobiology" name="keywords" />
  <link href="../../favicon.ico" rel="shortcut icon" />
  <link href="../../wp/w/Wikipedia_Text_of_the_GNU_Free_Documentation_License.htm" rel="copyright" />
  <title>Bioinformatics</title>
  <style media="screen,projection" type="text/css">/*<![CDATA[*/ @import "../../css/wp-monobook-main.css"; /*]]>*/</style>
  <link href="../../css/wp-commonPrint.css" media="print" rel="stylesheet" type="text/css" />
  <!--[if lt IE 5.5000]><style type="text/css">@import "../../css/IE50Fixes.css";</style><![endif]-->
  <!--[if IE 5.5000]><style type="text/css">@import "../../css/IE55Fixes.css";</style><![endif]-->
  <!--[if IE 6]><style type="text/css">@import "../../css/IE60Fixes.css";</style><![endif]-->
  <!--[if IE 7]><style type="text/css">@import "../../css/IE70Fixes.css";</style><![endif]-->
  <!--[if lt IE 7]><script type="text/javascript" src="../../js/IEFixes.js"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->
  <script type="text/javascript">
			var skin = "monobook";
			var stylepath = "/skins-1.5";

			var wgArticlePath = "/wiki/$1";
			var wgScriptPath = "/w";
			var wgServer = "http://en.wikipedia.org";
                        
			var wgCanonicalNamespace = "";
			var wgNamespaceNumber = 0;
			var wgPageName = "Bioinformatics";
			var wgTitle = "Bioinformatics";
			var wgArticleId = 4214;
			var wgIsArticle = true;
                        
			var wgUserName = null;
			var wgUserLanguage = "en";
			var wgContentLanguage = "en";
		</script>
  <script src="../../js/wikibits.js" type="text/javascript"><!-- wikibits js --></script>
  <script src="../../js/wp.js" type="text/javascript"><!-- site js --></script>
  <style type="text/css">/*<![CDATA[*/
@import "../../css/wp-common.css";
@import "../../css/wp-monobook.css";
@import "../../css/wp.css";
/*]]>*/</style>
  <!-- Head Scripts -->
 </head>
 <body class="mediawiki ns-0 ltr page-Bioinformatics">
  <div id="globalWrapper">
   <div id="column-content">
    <div id="content"><a id="top" name="top"></a><h1 class="firstHeading">Bioinformatics</h1>
     <div id="bodyContent">
      <h3 id="siteSub"><a href="../../index.htm">2007 Schools Wikipedia Selection</a>. Related subjects: <a href="../index/subject.Science.Biology.General_Biology.htm">General Biology</a>&#59; <a href="../index/subject.IT.Software.htm">Software</a></h3>
      <!-- start content -->
      <p><b>Bioinformatics</b> and <b>computational biology</b> involve the use of techniques including <a href="../../wp/a/Applied_mathematics.htm" title="Applied mathematics">applied mathematics</a>, <!--del_lnk--> informatics, <a href="../../wp/s/Statistics.htm" title="Statistics">statistics</a>, <a href="../../wp/c/Computer_science.htm" title="Computer science">computer science</a>, <!--del_lnk--> artificial intelligence, <a href="../../wp/c/Chemistry.htm" title="Chemistry">chemistry</a> and <!--del_lnk--> biochemistry to solve <a href="../../wp/b/Biology.htm" title="Biology">biological</a> problems usually on the <!--del_lnk--> molecular level. Research in computational biology often overlaps with <!--del_lnk--> systems biology. Major research efforts in the field include <a href="../../wp/s/Sequence_alignment.htm" title="Sequence alignment">sequence alignment</a>, <!--del_lnk--> gene finding, <!--del_lnk--> genome assembly, <!--del_lnk--> protein structure alignment, <!--del_lnk--> protein structure prediction, prediction of <!--del_lnk--> gene expression and <!--del_lnk--> protein-protein interactions, and the modeling of <a href="../../wp/e/Evolution.htm" title="Evolution">evolution</a>.<p>The terms <i>bioinformatics</i> and <i>computational biology</i> are often used interchangeably. However <i>bioinformatics</i> more properly refers to the creation and advancement of algorithms, computational and statistical techniques, and theory to solve formal and practical problems posed by or inspired from the management and analysis of biological data. <i>Computational biology,</i> on the other hand, refers to hypothesis-driven investigation of a specific biological problem using computers, carried out with experimental and simulated data, with the primary goal of discovery and the advancement of biological knowledge. A similar distinction is made by <!--del_lnk--> National Institutes of Health in their <!--del_lnk--> working definitions of Bioinformatics and Computational Biology, where it is further emphasized that there is a tight coupling of developments and knowledge between the more hypothesis-driven research in computational biology and technique-driven research in bioinformatics. Computational biology also includes lesser known but equally important subdisciplines such as computational <!--del_lnk--> biochemistry and computational <!--del_lnk--> biophysics.<p>A common thread in projects in bioinformatics and computational biology is the use of mathematical tools to extract useful information from data produced by high-throughput biological techniques such as <!--del_lnk--> genome sequencing. A representative problem in bioinformatics is the assembly of high-quality genome sequences from fragmentary &quot;shotgun&quot; DNA <!--del_lnk--> sequencing. Other common problems include the study of <!--del_lnk--> gene regulation using data from <!--del_lnk--> microarrays or <!--del_lnk--> mass spectrometry.<p>
       <script type="text/javascript">
//<![CDATA[
 if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } 
//]]>
</script><a id="Major_Research_Areas" name="Major_Research_Areas"></a><h2> <span class="mw-headline">Major Research Areas</span></h2>
      <p><a id="Sequence_analysis" name="Sequence_analysis"></a><h3> <span class="mw-headline">Sequence analysis</span></h3>
      <dl>
       <dd>
      </dl>
      <p>Since the <!--del_lnk--> Phage &Phi;-X174 was <!--del_lnk--> sequenced in 1977, the <!--del_lnk--> DNA sequences of hundreds of organisms have been decoded and stored in databases. These data are analyzed to determine genes that code for <a href="../../wp/p/Protein.htm" title="Protein">proteins</a>, as well as regulatory sequences. A comparison of genes within a <!--del_lnk--> species or between different species can show similarities between protein functions, or relations between species (the use of <!--del_lnk--> molecular systematics to construct <!--del_lnk--> phylogenetic trees). With the growing amount of data, it long ago became impractical to analyze DNA sequences manually. Today, <!--del_lnk--> computer programs are used to search the <!--del_lnk--> genome of thousands of organisms, containing billions of <!--del_lnk--> nucleotides. These programs can compensate for mutations (exchanged, deleted or inserted bases) in the DNA sequence, in order to identify sequences that are related, but not identical. A variant of this <a href="../../wp/s/Sequence_alignment.htm" title="Sequence alignment">sequence alignment</a> is used in the sequencing process itself. The so-called <!--del_lnk--> shotgun sequencing technique (which was used, for example, by <!--del_lnk--> The Institute for Genomic Research to sequence the first bacterial genome, <i>Haemophilus influenza</i>) does not give a sequential list of nucleotides, but instead the sequences of thousands of small DNA fragments (each about 600-800 nucleotides long). The ends of these fragments overlap and, when aligned in the right way, make up the complete genome. Shotgun sequencing yields sequence data quickly, but the task of assembling the fragments can be quite complicated for larger genomes. In the case of the <!--del_lnk--> Human Genome Project, it took several months of CPU time (on a circa-2000 vintage DEC Alpha computer) to assemble the fragments. Shotgun sequencing is the method of choice for virtually all genomes sequenced today, and genome assembly algorithms are a critical area of bioinformatics research.<p>Another aspect of bioinformatics in sequence analysis is the automatic <!--del_lnk--> search for genes and regulatory sequences within a genome. Not all of the nucleotides within a genome are genes. Within the genome of higher organisms, large parts of the DNA do not serve any obvious purpose. This so-called <!--del_lnk--> junk DNA may, however, contain unrecognized functional elements. Bioinformatics helps to bridge the gap between genome and <!--del_lnk--> proteome projects--for example, in the use of DNA sequences for protein identification.<p><i>See also:</i> <!--del_lnk--> sequence analysis, <!--del_lnk--> sequence profiling tool, <!--del_lnk--> sequence motif.<p><a id="Genome_annotation" name="Genome_annotation"></a><h4> <span class="mw-headline">Genome annotation</span></h4>
      <dl>
       <dd>
      </dl>
      <p>In the context of genomics, <b>annotation</b> is the process of marking the genes and other biological features in a DNA sequence. The first genome annotation software system was designed in 1995 by Owen White, who was part of the team that sequenced and analyzed the first genome of a free-living organism to be decoded, the bacterium <!--del_lnk--> Haemophilus influenzae. Dr. White built a software system to find the genes (places in the DNA sequence that encode a protein), the transfer RNA, and other features, and to make initial assignments of function to those genes. Most current genome annotation systems work similarly, but the programs available for analysis of genomic DNA are constantly changing and improving.<p><a id="Computational_evolutionary_biology" name="Computational_evolutionary_biology"></a><h3> <span class="mw-headline">Computational evolutionary biology</span></h3>
      <p><!--del_lnk--> Evolutionary biology is the study of the origin and descent of <!--del_lnk--> species, as well as their change over time. Informatics has assisted evolutionary biologists in several key ways; it has enabled researchers to:<ul>
       <li>trace the evolution of a large number of organisms by measuring changes in their <a href="../../wp/d/DNA.htm" title="DNA">DNA</a>, rather than through <!--del_lnk--> physical taxonomy or physiological observations alone,<li>more recently, compare entire <!--del_lnk--> genomes, which permits the study of more complex evolutionary events, such as <!--del_lnk--> gene duplication, <!--del_lnk--> lateral gene transfer, and the prediction of bacterial <!--del_lnk--> speciation factors,<li>build complex computational models of populations to predict the outcome of the system over time<li>track and share information on an increasingly large number of species and organisms</ul>
      <p>Future work endeavours to reconstruct the now more complex <!--del_lnk--> tree of life.<p>The area of research within <a href="../../wp/c/Computer_science.htm" title="Computer science">computer science</a> that uses <!--del_lnk--> genetic algorithms is sometimes confused with <!--del_lnk--> computational evolutionary biology, but the two areas are unrelated.<p><a id="Measuring_biodiversity" name="Measuring_biodiversity"></a><h3> <span class="mw-headline">Measuring biodiversity</span></h3>
      <p><a href="../../wp/b/Biodiversity.htm" title="Biodiversity">Biodiversity</a> of an ecosystem might be defined as the total genomic complement of a particular environment, from all of the species present, whether it is a biofilm in an abandoned mine, a drop of sea water, a scoop of soil, or the entire <a href="../../wp/b/Biosphere.htm" title="Biosphere">biosphere</a> of the planet <a href="../../wp/e/Earth.htm" title="Earth">Earth</a>. Databases are used to collect the <!--del_lnk--> species names, descriptions, distributions, genetic information, status and size of <!--del_lnk--> populations, <!--del_lnk--> habitat needs, and how each organism interacts with other species. Specialized <!--del_lnk--> software programs are used to find, visualize, and analyze the information, and most importantly, communicate it to other people. Computer simulations model such things as population dynamics, or calculate the cumulative genetic health of a breeding pool (in <a href="../../wp/a/Agriculture.htm" title="Agriculture">agriculture</a>) or endangered population (in <!--del_lnk--> conservation). One very exciting potential of this field is that entire <a href="../../wp/d/DNA.htm" title="DNA">DNA</a> sequences, or <!--del_lnk--> genomes of <!--del_lnk--> endangered species can be preserved, allowing the results of Nature&#39;s genetic experiment to be remembered <i><!--del_lnk--> in silico</i>, and possibly reused in the future, even if that species is eventually lost.<p><i>Important Projects:</i> <!--del_lnk--> Species 2000 project; <!--del_lnk--> uBio Project.<p><a id="Analysis_of_gene_expression" name="Analysis_of_gene_expression"></a><h3> <span class="mw-headline">Analysis of gene expression</span></h3>
      <p>The <!--del_lnk--> expression of many genes can be determined by measuring <!--del_lnk--> mRNA levels with multiple techniques including <!--del_lnk--> microarrays, <!--del_lnk--> expressed cDNA sequence tag (EST) sequencing, <!--del_lnk--> serial analysis of gene expression (SAGE) tag sequencing, <!--del_lnk--> massively parallel signature sequencing (MPSS), or various applications of multiplexed in-situ hybridization. All of these techniques are extremely noise-prone and/or subject to bias in the biological measurement, and a major research area in computational biology involves developing statistical tools to separate <!--del_lnk--> signal from <!--del_lnk--> noise in high-throughput gene expression studies. Such studies are often used to determine the genes implicated in a disorder: one might compare microarray data from cancerous <!--del_lnk--> epithelial cells to data from non-cancerous cells to determine the transcripts that are up-regulated and down-regulated in a particular population of cancer cells.<p><a id="Analysis_of_regulation" name="Analysis_of_regulation"></a><h3> <span class="mw-headline">Analysis of regulation</span></h3>
      <p>Regulation is the complex orchestration of events starting with an <!--del_lnk--> extra-cellular signal and ultimately leading to an increase or decrease in the <!--del_lnk--> activity of one or more protein molecules. Bioinformatics techniques have been applied to explore various steps in this process. For example, <!--del_lnk--> promoter analysis involves the elucidation and study of <!--del_lnk--> sequence motifs in the genomic region surrounding the coding region of a gene. These motifs influence the extent to which that region is transcribed into mRNA. Expression data can be used to infer gene regulation: one might compare <!--del_lnk--> microarray data from a wide variety of states of an organism to form hypotheses about the genes involved in each state. In a single-cell organism, one might compare stages of the <!--del_lnk--> cell cycle, along with various stress conditions (heat shock, starvation, etc.). One can then apply <!--del_lnk--> clustering algorithms to that expression data to determine which genes are co-expressed. For example, the upstream regions (promoters) of co-expressed genes can be searched for over-represented <!--del_lnk--> regulatory elements.<p><a id="Analysis_of_protein_expression" name="Analysis_of_protein_expression"></a><h3> <span class="mw-headline">Analysis of protein expression</span></h3>
      <p>Protein <!--del_lnk--> microarrays and high throughput (HT) <!--del_lnk--> mass spectrometry (MS) can provide a snapshot of the proteins present in a biological sample. Bioinformatics is very much involved in making sense of protein microarray and HT MS data; the former approach faces similar problems as with microarrays targeted at mRNA, the latter involves the problem of matching large amounts of mass data against predicted masses from protein sequence databases, and the complicated statistical analysis of samples where multiple, but incomplete peptides from each protein are detected.<p><a id="Analysis_of_mutations_in_cancer" name="Analysis_of_mutations_in_cancer"></a><h3> <span class="mw-headline">Analysis of mutations in cancer</span></h3>
      <p>Massive sequencing efforts are currently underway to identify <!--del_lnk--> point mutations in a variety of <!--del_lnk--> genes in <a href="../../wp/c/Cancer.htm" title="Cancer">cancer</a>. The sheer volume of data produced requires automated systems to read sequence data, and to compare the sequencing results to the known sequence of the <!--del_lnk--> human genome, including known <!--del_lnk--> germline polymorphisms.<p><!--del_lnk--> Oligonucleotide microarrays, including <!--del_lnk--> comparative genomic hybridization and <!--del_lnk--> single nucleotide polymorphism arrays, able to probe simultaneously up to several hundred thousand sites throughout the genome are being used to identify chromosomal gains and losses in cancer. <!--del_lnk--> Hidden Markov model and <!--del_lnk--> change-point analysis methods are being developed to infer real copy number changes from often noisy data. Further informatics approaches are being developed to understand the implications of lesions found to be recurrent across many tumors.<p>Some modern tools (e.g. <!--del_lnk--> Quantum 3.1 ) provide tool for changing the protein sequence at specific sites through alterations to its amino acids and predict changes in the bioactivity after mutations.<p><a id="Prediction_of_protein_structure" name="Prediction_of_protein_structure"></a><h3> <span class="mw-headline">Prediction of protein structure</span></h3>
      <dl>
       <dd>
      </dl>
      <p>Protein structure prediction is another important application of bioinformatics. The <!--del_lnk--> amino acid sequence of a protein, the so-called <i>primary structure</i>, can be easily determined from the sequence on the gene that codes for it. In the vast majority of cases, this primary structure uniquely determines a structure in its native environment. (Of course, there are exceptions, such as the <!--del_lnk--> bovine spongiform encephalopathy - aka <!--del_lnk--> Mad Cow Disease - <a href="../../wp/p/Prion.htm" title="Prion">prion</a>.) Knowledge of this structure is vital in understanding the function of the protein. For lack of better terms, structural information is usually classified as one of <i><!--del_lnk--> secondary</i>, <i><!--del_lnk--> tertiary</i> and <i><!--del_lnk--> quaternary</i> structure. A viable general solution to such predictions remains an open problem. As of now, most efforts have been directed towards heuristics that work most of the time.<p>One of the key ideas in bioinformatics is the notion of <!--del_lnk--> homology. In the genomic branch of bioinformatics, homology is used to predict the function of a gene: if the sequence of gene <i>A</i>, whose function is known, is homologous to the sequence of gene <i>B,</i> whose function is unknown, one could infer that B may share A&#39;s function. In the structural branch of bioinformatics, homology is used to determine which parts of a protein are important in structure formation and interaction with other proteins. In a technique called homology modelling, this information is used to predict the structure of a protein once the structure of a homologous protein is known. This currently remains the only way to predict protein structures reliably.<p>One example of this is the similar protein homology between haemoglobin in humans and the haemoglobin in legumes (<!--del_lnk--> leghemoglobin). Both serve the same purpose of transporting oxygen in the organism. Though both of these proteins have completely different amino acid sequences, their protein structures are virtually identical, which reflects their near identical purposes.<p>Other techniques for predicting protein structure include protein threading and <i>de novo</i> (from scratch) physics-based modeling.<p>See also <!--del_lnk--> structural motif and <!--del_lnk--> structural domain.<p><a id="Comparative_genomics" name="Comparative_genomics"></a><h3> <span class="mw-headline">Comparative genomics</span></h3>
      <p>The core of comparative genome analysis is the establishment of the correspondence between <!--del_lnk--> genes (orthology analysis) or other genomic features in different organisms. It is these intergenomic maps that make it possible to trace the evolutionary processes responsible for the divergence of two genomes. A multitude of evolutionary events acting at various organizational levels shape genome evolution. At the lowest level, point mutations affect individual nucleotides. At a higher level, large chromosomal segments undergo duplication, lateral transfer, inversion, transposition, deletion and insertion. Ultimately, whole genomes are involved in processes of hybridization, polyploidization and <!--del_lnk--> endosymbiosis, often leading to rapid speciation. The complexity of genome evolution poses many exciting challenges to developers of mathematical models and algorithms, who have recourse to a spectra of algorithmic, statistical and mathematical techniques, ranging from exact, <!--del_lnk--> heuristics, fixed parameter and <!--del_lnk--> approximation algorithms for problems based on parsimony models to <!--del_lnk--> Markov Chain Monte Carlo algorithms for <!--del_lnk--> Bayesian analysis of problems based on probabilistic models.<p>Many of these studies are based on the homology detection and protein families computation.<p>See also <!--del_lnk--> comparative genomics, <!--del_lnk--> bayesian network and <!--del_lnk--> protein family.<p><a id="Modeling_biological_systems" name="Modeling_biological_systems"></a><h3> <span class="mw-headline">Modeling biological systems</span></h3>
      <dl>
       <dd>
      </dl>
      <p>Systems biology involves the use of <!--del_lnk--> computer simulations of <a href="../../wp/c/Cell_%2528biology%2529.htm" title="Cell (biology)">cellular</a> subsystems (such as the <!--del_lnk--> networks of metabolites and <!--del_lnk--> enzymes which comprise <!--del_lnk--> metabolism, <!--del_lnk--> signal transduction pathways and <!--del_lnk--> gene regulatory networks) to both analyze and visualize the complex connections of these cellular processes. <!--del_lnk--> Artificial life or virtual evolution attempts to understand evolutionary processes via the computer simulation of simple (artificial) life forms.<p><a id="High-throughput_image_analysis" name="High-throughput_image_analysis"></a><h3> <span class="mw-headline">High-throughput image analysis</span></h3>
      <p>Computational technologies are used to accelerate or fully automate the processing, quantification and analysis of large amounts of high-information-content <!--del_lnk--> biomedical imagery. Modern image analysis systems augment an observer&#39;s ability to make measurements from a large or complex set of images, by improving <!--del_lnk--> accuracy, <!--del_lnk--> objectivity, or speed. A fully developed analysis system may completely replace the observer. Although these systems are not unique to biomedical imagery, biomedical imaging is becoming more important for both <!--del_lnk--> diagnostics and research. Some examples are:<ul>
       <li>high-throughput and high-fidelity quantification and sub-cellular localization (<!--del_lnk--> high-content screening, <!--del_lnk--> cytohistopathology)<li><!--del_lnk--> morphometrics<li>clinical image analysis and visualization<li>determining the real-time air-flow patterns in breathing lungs of living animals<li>quantifying occlusion size in real-time imagery from the development of and recovery during arterial injury<li>making behavioural observations from extended video recordings of laboratory animals<li>infrared measurements for metabolic activity determination</ul>
      <p><a id="Software_tools" name="Software_tools"></a><h2> <span class="mw-headline">Software tools</span></h2>
      <p>First generation bioinformatics tools consisted of applications, usually with a text-based interface, which performed a specific task well. The computational biology tool best-known among biologists is probably <!--del_lnk--> BLAST, an algorithm for searching large databases of protein or DNA sequences. The <!--del_lnk--> NCBI provides a popular web-based implementation that searches their massive sequence databases. Also fairly early on, due to the amassing of sequence and annotation data, keyword search engines which were able to resolve gene and protein synonyms were important. In these early days, computer scripting languages such as <a href="../../wp/p/Perl.htm" title="Perl">Perl</a> and <!--del_lnk--> Python were often used to interface with <!--del_lnk--> biological databases and <!--del_lnk--> parse output from bioinformatics programs written in languages such as <!--del_lnk--> C or <a href="../../wp/c/C%252B%252B.htm" title="C++">C++</a>, and much legacy <!--del_lnk--> code is still in use today. Today, many other languages are used to author excellent software, and communities of bioinformatics programmers have set up <!--del_lnk--> free open source bioinformatics projects to develop and distribute the tools and modules they produce.<p>As the data sources expanded and diversified, both in content and geography, bioinformatic meta search engines, such as <!--del_lnk--> Sequence profiling tools, emerged to help find relevant information from several databases. These meta search engines might index data from a local server or even from a panel of third party services.<p>More recently, SOAP-based interfaces have been developed for a wide variety of bioinformatics applications allowing an application running on one computer in one part of the world to use algorithms, data and computing resources on servers in other parts of the world. A large availability of these <!--del_lnk--> SOAP-based bioinformatics web services, along with the open source bioinformatics collections, lead to the next generation of bioinformatics tools: the <!--del_lnk--> integrated bioinformatics platform. These tools range from a collection of standalone tools with a common data format under a single, slick standalone or web-based interface, to integrative and extensible <!--del_lnk--> bioinformatics workflow development environments.<p>An interesting novel direction for bioinformatics applications is illustrated by Q-Pharm&#39;s Quantum 3.1, an example of the bioinformatics post-<!--del_lnk--> QSAR technology applying quantum and molecular physics instead of statistical methods.<p><a id="See_also" name="See_also"></a><div class="printfooter"> Retrieved from &quot;<!--del_lnk--> http://en.wikipedia.org/wiki/Bioinformatics&quot;</div>
      <!-- end content -->
      <div class="visualClear">
      </div>
     </div>
    </div>
   </div>
   <!-- end of the left (by default at least) column -->
   <div class="visualClear">
   </div>
   <div id="footer">
    <div class="center"> This reference article is mainly selected from the English Wikipedia with only minor checks and changes (see www.wikipedia.org for details of authors and sources) and is available under the <nobr><a href="../../wp/w/Wikipedia_Text_of_the_GNU_Free_Documentation_License.htm">GNU Free Documentation License</a></nobr>. See also our <b><a href="../../disclaimer.htm">Disclaimer</a></b>. </div>
   </div>
   <script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
  </div>
  
 </body>
</html>
