<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html dir="ltr" lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
 <head>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
  <meta content="Enzyme kinetics,Nonlinear regression,Lineweaver-Burk plot,Reciprocal,Extrapolation,Eadie-Hofstee diagram,Absorbance,Active site,Adenylate cyclase,Aspartate aminotransferase,Aspartate transcarbamoylase" name="keywords" />
  <link href="../../favicon.ico" rel="shortcut icon" />
  <link href="../../wp/w/Wikipedia_Text_of_the_GNU_Free_Documentation_License.htm" rel="copyright" />
  <title>Enzyme kinetics</title>
  <style media="screen,projection" type="text/css">/*<![CDATA[*/ @import "../../css/wp-monobook-main.css"; /*]]>*/</style>
  <link href="../../css/wp-commonPrint.css" media="print" rel="stylesheet" type="text/css" />
  <!--[if lt IE 5.5000]><style type="text/css">@import "../../css/IE50Fixes.css";</style><![endif]-->
  <!--[if IE 5.5000]><style type="text/css">@import "../../css/IE55Fixes.css";</style><![endif]-->
  <!--[if IE 6]><style type="text/css">@import "../../css/IE60Fixes.css";</style><![endif]-->
  <!--[if IE 7]><style type="text/css">@import "../../css/IE70Fixes.css";</style><![endif]-->
  <!--[if lt IE 7]><script type="text/javascript" src="../../js/IEFixes.js"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->
  <script type="text/javascript">
			var skin = "monobook";
			var stylepath = "/skins-1.5";

			var wgArticlePath = "/wiki/$1";
			var wgScriptPath = "/w";
			var wgServer = "http://en.wikipedia.org";

			var wgCanonicalNamespace = "";
			var wgNamespaceNumber = 0;
			var wgPageName = "Enzyme_kinetics";
			var wgTitle = "Enzyme kinetics";
			var wgArticleId = 3043886;
			var wgCurRevisionId = 92339586;
			var wgIsArticle = true;
		
			var wgUserName = null;
			var wgUserLanguage = "en";
			var wgContentLanguage = "en";
		</script>
  <script src="../../js/wikibits.js" type="text/javascript"><!-- wikibits js --></script>
  <script src="../../js/wp.js" type="text/javascript"><!-- site js --></script>
  <style type="text/css">/*<![CDATA[*/
@import "../../css/wp-common.css";
@import "../../css/wp-monobook.css";
@import "../../css/wp.css";
/*]]>*/</style>
  <!-- Head Scripts -->
 </head>
 <body class="mediawiki ns-0 ltr page-Enzyme_kinetics">
  <div id="globalWrapper">
   <div id="column-content">
    <div id="content"><a id="top" name="top"></a><h1 class="firstHeading">Enzyme kinetics</h1>
     <div id="bodyContent">
      <h3 id="siteSub"><a href="../../index.htm">2007 Schools Wikipedia Selection</a>. Related subjects: <a href="../index/subject.Science.Chemistry.General_Chemistry.htm">General Chemistry</a></h3>
      <!-- start content -->
      <div class="thumb tright">
       <div style="width:302px;"><a class="internal" href="../../images/180/18094.png.htm" title="Dihydrofolate reductase from E. coli with its two substrates, dihydrofolate (right) and NADPH (left), bound in the active site. The protein is shown as a ribbon diagram, with alpha helices in red, beta sheets in yellow and loops in blue. Generated from 7DFR."><img alt="Dihydrofolate reductase from E. coli with its two substrates, dihydrofolate (right) and NADPH (left), bound in the active site. The protein is shown as a ribbon diagram, with alpha helices in red, beta sheets in yellow and loops in blue. Generated from 7DFR." height="400" longdesc="/wiki/Image:EcDHFR_raytraced.png" src="../../images/180/18094.png" width="300" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/180/18094.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div><!--del_lnk--> Dihydrofolate reductase from <i>E. coli</i> with its two substrates, dihydrofolate (right) and NADPH (left), bound in the active site. The protein is shown as a ribbon diagram, with alpha helices in red, beta sheets in yellow and loops in blue. Generated from <!--del_lnk--> 7DFR.</div>
       </div>
      </div>
      <p><b>Enzyme kinetics</b> is the study of the <!--del_lnk--> rates of <!--del_lnk--> chemical reactions that are <!--del_lnk--> catalysed by <!--del_lnk--> enzymes. The study of an enzyme&#39;s <!--del_lnk--> kinetics provides insights into the catalytic mechanism of this enzyme, its role in <!--del_lnk--> metabolism, how its activity is controlled in the <a href="../../wp/c/Cell_%2528biology%2529.htm" title="Cell (biology)">cell</a> and how <!--del_lnk--> drugs and <!--del_lnk--> poisons can <!--del_lnk--> inhibit its activity.<p>Enzymes are <!--del_lnk--> molecules that manipulate other molecules &mdash; the enzymes&#39; <!--del_lnk--> substrates. These target molecules bind to an enzyme&#39;s <!--del_lnk--> active site and are transformed into products through a series of steps known as the <!--del_lnk--> enzymatic mechanism. Some enzymes bind multiple substrates and/or release multiple products, such as a <!--del_lnk--> protease cleaving one protein substrate into two polypeptide products. Others join substrates together, such as <!--del_lnk--> DNA polymerase linking a <!--del_lnk--> nucleotide to <a href="../../wp/d/DNA.htm" title="DNA">DNA</a>. Although these mechanisms are often a complex series of steps, there is typically one <i>rate-determining step</i> that determines the overall kinetics. This rate-determining step may be a chemical reaction or a conformational change of the enzyme or substrates, such as those involved in the release of product(s) from the enzyme.<p>Knowledge of the <!--del_lnk--> enzyme&#39;s structure is helpful in visualizing the kinetic data. For example, the structure can suggest how substrates and products bind during catalysis; what changes occur during the reaction; and even the role of particular amino acid residues in the mechanism. Some enzymes change shape significantly during the mechanism; in such cases, it is helpful to determine the enzyme structure with and without bound substrate analogs that do not undergo the enzymatic reaction.<p>Enzyme mechanisms can be divided into single-substrate and multiple-substrate mechanisms. Kinetic studies on enzymes that only bind one substrate, such as <!--del_lnk--> triosephosphate isomerase, aim to measure the <!--del_lnk--> affinity with which the enzyme binds this substrate and the turnover rate. When enzymes bind multiple substrates, such as <!--del_lnk--> dihydrofolate reductase (shown right), enzyme kinetics can also show the sequence in which these substrates bind and the sequence in which products are released.<p>Not all biological catalysts are protein enzymes; <!--del_lnk--> RNA-based catalysts such as <!--del_lnk--> ribozymes and <!--del_lnk--> ribosomes are essential to many cellular functions, such as <!--del_lnk--> RNA splicing and <!--del_lnk--> translation. The main difference between ribozymes and enzymes is that the RNA catalysts perform a more limited set of reactions, although their <!--del_lnk--> reaction mechanisms and kinetics can be analysed and classified by the same methods.<p>
       <script type="text/javascript">
//<![CDATA[
 if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } 
//]]>
</script><a id="General_principles" name="General_principles"></a><h2> <span class="mw-headline">General principles</span></h2>
      <div class="thumb tright">
       <div style="width:452px;"><a class="internal" href="../../images/180/18095.png.htm" title="The rate of reaction will increase as substrate concentration increases, eventually becoming saturated at very high concentrations of substrate."><img alt="The rate of reaction will increase as substrate concentration increases, eventually becoming saturated at very high concentrations of substrate." height="126" longdesc="/wiki/Image:Enzyme_saturation.png" src="../../images/180/18095.png" width="450" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/180/18095.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> The rate of reaction will increase as substrate concentration increases, eventually becoming saturated at very high concentrations of substrate.</div>
       </div>
      </div>
      <p>The reaction catalysed by an enzyme uses exactly the same reactants and produces exactly the same products as the uncatalysed reaction. Like other <!--del_lnk--> catalysts, enzymes do not alter the position of <!--del_lnk--> equilibrium between substrates and products. However, unlike normal chemical reactions, enzymes are saturable. This means as more substrate is added, the reaction rate will increase, because more active sites become occupied. This can continue until all the enzyme becomes saturated with substrate and the rate reaches a maximum.<p>The two most important kinetic properties of an enzyme are how quickly the enzyme becomes saturated with a particular substrate, and the maximum rate it can achieve. Knowing these properties suggests what an enzyme might do in the environment of the cell and can show how the enzyme will respond to changes in these conditions.<p><a id="Enzyme_assays" name="Enzyme_assays"></a><h2> <span class="mw-headline">Enzyme assays</span></h2>
      <dl>
       <dd>
      </dl>
      <div class="thumb tleft">
       <div style="width:252px;"><a class="internal" href="../../images/180/18096.png.htm" title="Progress curve for an enzyme reaction. The slope in the initial rate period represents the initial rate of reaction v.  Equations such as the Michaelis-Menten equation describe how this slope varies with the substrate and enzyme concentrations."><img alt="Progress curve for an enzyme reaction. The slope in the initial rate period represents the initial rate of reaction v.  Equations such as the Michaelis-Menten equation describe how this slope varies with the substrate and enzyme concentrations." height="195" longdesc="/wiki/Image:Enzyme_progress_curve.svg" src="../../images/180/18096.png" width="250" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/180/18096.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Progress curve for an enzyme reaction. The slope in the initial rate period represents the <b>initial rate of reaction</b> <i>v</i>. Equations such as the <!--del_lnk--> Michaelis-Menten equation describe how this slope varies with the substrate and enzyme concentrations.</div>
       </div>
      </div>
      <p><!--del_lnk--> Enzyme assays are laboratory procedures that measure the rate of enzyme reactions. Because enzymes are not consumed by the reactions they catalyse, enzyme assays usually follow changes in the concentration of either substrates or products to measure the rate of reaction. There are many methods of measurement. <!--del_lnk--> Spectrophotometric assays observe change in the <!--del_lnk--> absorbance of light between products and reactants; radiometric assays involve the incorporation or release of <!--del_lnk--> radioactivity to measure the amount of product made over time. Spectrophotometric assays are most convenient since they allow the rate of the reaction to be measured continuously. Although radiometric assays require the removal and counting of samples (i.e., they are discontinuous assays) they are usually extremely sensitive and can measure very low levels of enzyme activity. An analogous approach is to use <!--del_lnk--> mass spectrometry to monitor the incorporation or release of <!--del_lnk--> stable isotopes as substrate is converted into product.<p>The most sensitive enzyme assays use <!--del_lnk--> lasers focused through a <a href="../../wp/m/Microscope.htm" title="Microscope">microscope</a> to observe changes in single enzyme molecules as they catalyse their reactions. These measurements either use changes in the <!--del_lnk--> fluorescence of <!--del_lnk--> cofactors during an enzyme&#39;s reaction mechanism, or <!--del_lnk--> fluorescent dyes added onto specific sites of the <a href="../../wp/p/Protein.htm" title="Protein">protein</a> that report movements that occur during catalysis. These studies are providing a new view of the kinetics and dynamics of single molecules, as opposed to traditional enzyme kinetics, which observes the average behaviour of populations of millions of enzyme molecules.<p>On the left is shown a typical progress curve for an enzyme assay. The enzyme produces product at a linear initial rate at the start of the reaction. Later in this progress curve, the rate slows down as substrate is used up or products accumulate. The length of the initial rate period depends on the assay conditions and can range from milliseconds to hours. Enzyme assays are usually set up to produce an initial rate lasting over a minute, to make measurements easier. However, equipment for rapidly mixing liquids allows fast kinetic measurements on initial rates of less than one second. These very rapid assays are essential for measuring pre-steady-state kinetics, which are discussed below.<p>Most enzyme kinetics studies concentrate on this initial, linear part of enzyme reactions. However, it is also possible to measure the complete reaction curve and fit this data to a non-linear rate equation. This way of measuring enzyme reactions is called progress-curve analysis. This approach is useful as an alternative to rapid kinetics when the initial rate is too fast to measure accurately.<p><a id="Single-substrate_reactions" name="Single-substrate_reactions"></a><h2> <span class="mw-headline">Single-substrate reactions</span></h2>
      <p>Enzymes with single-substrate mechanisms include <!--del_lnk--> isomerases such as <!--del_lnk--> triosephosphateisomerase or <!--del_lnk--> bisphosphoglycerate mutase, intramolecular <!--del_lnk--> lyases such as <!--del_lnk--> adenylate cyclase and the <!--del_lnk--> hammerhead ribozyme, a RNA lyase. However, some enzymes that only have a single substrate do not fall into this category of mechanisms. <!--del_lnk--> Catalase is an example of this, as the enzyme reacts with a first molecule of <a href="../../wp/h/Hydrogen_peroxide.htm" title="Hydrogen peroxide">hydrogen peroxide</a> substrate, becomes oxidised and is then reduced by a second molecule of substrate. Although a single substrate is involved, the existence of a modified enzyme intermediate means that the mechanism of catalase is actually a ping&ndash;pong mechanism, a type of mechanism that is discussed in the <i>Multi-substrate reactions</i> section below.<p><a id="Michaelis.E2.80.93Menten_kinetics" name="Michaelis.E2.80.93Menten_kinetics"></a><h3> <span class="mw-headline">Michaelis&ndash;Menten kinetics</span></h3>
      <div class="thumb tleft">
       <div style="width:302px;"><a class="internal" href="../../images/180/18097.png.htm" title="Saturation curve for an enzyme showing the relation between the concentration of substrate and rate."><img alt="Saturation curve for an enzyme showing the relation between the concentration of substrate and rate." height="229" longdesc="/wiki/Image:MM_curve_v2.png" src="../../images/180/18097.png" width="300" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/180/18097.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Saturation curve for an enzyme showing the relation between the concentration of substrate and rate.</div>
       </div>
      </div>
      <p>As enzyme-catalysed reactions are saturable, their rate of catalysis does not show a linear response to increasing substrate. If the initial rate of the reaction is measured over a range of substrate concentrations (denoted as [S]), the reaction rate (<i>v</i>) increases as [S] increases, as shown on the left. However, as [S] gets higher, the enzyme becomes saturated with substrate and the rate reaches <b><i>V</i><sub>max</sub></b>, the enzyme&#39;s maximum rate.<div class="thumb tright">
       <div style="width:302px;"><a class="internal" href="../../images/180/18098.png.htm" title="Single-substrate mechanism for an enzyme reaction. k1, k-1 and k2 are the rate constants for the individual steps."><img alt="Single-substrate mechanism for an enzyme reaction. k1, k-1 and k2 are the rate constants for the individual steps." height="101" longdesc="/wiki/Image:Mechanism_plus_rates.svg" src="../../images/180/18098.png" width="300" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/180/18098.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Single-substrate mechanism for an enzyme reaction. <i>k</i><sub>1</sub>, <i>k</i><sub>-1</sub> and <i>k</i><sub>2</sub> are the rate constants for the individual steps.</div>
       </div>
      </div>
      <p>The <!--del_lnk--> Michaelis-Menten kinetic model of a single-substrate reaction is shown on the right. There is an initial <!--del_lnk--> bimolecular reaction between the enzyme E and substrate S to form the enzyme&ndash;substrate complex ES. Although the enzymatic mechanism for the <!--del_lnk--> unimolecular reaction <img alt="ES \rightarrow E + P" class="tex" src="../../images/468/46870.png" /> reaction can be quite complex, there is typically one rate-determining enzymatic step that allows the mechanism to be modeled as a single kinetic step of rate constant <i>k</i><sub>2</sub>.<dl>
       <dd><img alt="\begin{matrix}v = k_2 [\mbox{ES}]\end{matrix}" class="tex" src="../../images/468/46871.png" />&nbsp;&nbsp;&nbsp;&nbsp;<i>(Equation 1).</i></dl>
      <p><i>k</i><sub>2</sub> is also called <i>k</i><sub>cat</sub> or the <b>turnover number</b>, the maximum number of enzymatic reactions catalyzed per second.<p>At low concentrations of substrate [S], the enzyme exists in an equilibrium between both the free form E and the enzyme&ndash;substrate complex ES; increasing [S] likewises increases [ES] at the expense of [E], shifting the binding equilibrium to the right. Since the rate of the reaction depends on the concentration [ES], the rate is sensitive to small changes in [S]. However, at very high [S], the enzyme is entirely saturated with substrate, and exists only in the ES form. Under these conditions, the rate (<i>v</i>&asymp;<i>k</i><sub>2</sub>[E]<sub>tot</sub>=<i>V</i><sub>max</sub>) is insensitive to small changes in [S]; here, [E]<sub>tot</sub> is the total enzyme concentration<dl>
       <dd><img alt="[\mbox{E}]_{tot} \ \stackrel{\mathrm{def}}{=}\  [\mbox{E}] + [\mbox{ES}]" class="tex" src="../../images/468/46872.png" /></dl>
      <p>which is approximately equal to the concentration [ES] under saturating conditions.<p>The <!--del_lnk--> Michaelis&ndash;Menten equation describes how the reaction rate <i>v</i> depends on the position of the substrate-binding <!--del_lnk--> equilibrium and the rate constant <i>k</i><sub>2</sub>. <!--del_lnk--> Michaelis and <!--del_lnk--> Menten showed when <i>k</i><sub>2</sub> is much less than <i>k</i><sub>-1</sub> (the <b>equilibrium approximation</b>) they could derive the following equation:<dl>
       <dd><img alt="v = \frac{V_{max}[\mbox{S}]}{K_m + [\mbox{S}]}" class="tex" src="../../images/468/46873.png" />&nbsp;&nbsp;&nbsp;&nbsp;<i>(Equation 2)</i></dl>
      <p>This Michaelis-Menten equation is the basis for most single-substrate enzyme kinetics.<p><b>The Michaelis constant</b> <i>K</i><sub>m</sub> is defined as the concentration at which the rate of the enzyme reaction is half <i>V</i><sub>max</sub>. This may be verified by substituting [S] = <i>K</i><sub>m</sub> into the Michaelis-Menten equation. If the rate-determining enzymatic step is slow compared to substrate dissociation (<i>k</i><sub>2</sub> &lt;&lt; <i>k</i><sub>-1</sub>), the Michaelis constant <i>K</i><sub>m</sub> is roughly the <!--del_lnk--> dissociation constant of the ES complex, although this situation is relatively rare.<p>The more normal situation where <i>k</i><sub>2</sub> &gt; <i>k</i><sub>-1</sub> is sometimes called Briggs-<!--del_lnk--> Haldane kinetics. The Michaelis&ndash;Menten equation still holds under these more general conditions, as may be derived from the <b><!--del_lnk--> steady-state approximation</b>. During the initial-rate period, the reaction rate <i>v</i> is roughly constant, indicating that [ES] is similarly constant (cf. equation 1):<dl>
       <dd><img alt="\frac{d}{dt}[\mbox{ES}] = k_{1} [\mbox{E}][\mbox{S}] - k_{2}[\mbox{ES}] - k_{-1}[\mbox{ES}] \approx 0" class="tex" src="../../images/468/46874.png" /></dl>
      <p>Therefore, the concentration [ES] is given by the formula<dl>
       <dd><img alt="[\mbox{ES}] \approx \frac{[\mbox{E}]_{tot} [\mbox{S}]}{[\mbox{S}] + K_{m}}" class="tex" src="../../images/468/46875.png" /></dl>
      <p>where the Michaelis constant <i>K</i><sub>m</sub> is defined<dl>
       <dd><img alt="K_{m} \ \stackrel{\mathrm{def}}{=}\  \frac{k_{2} + k_{-1}}{k_{1}} \approx \frac{[\mbox{E}][\mbox{S}]}{[\mbox{ES}]}" class="tex" src="../../images/468/46876.png" /></dl>
      <p>([E] is the concentration of <i>free</i> enzyme). Taken together, the general formula for the reaction rate <i>v</i> is again the Michaelis-Menten equation:<dl>
       <dd><img alt="v = k_{2} [\mathrm{ES}] = \frac{k_{2} [\mbox{E}]_{tot} [\mbox{S}]}{[\mbox{S}] + K_{m}} = \frac{V_{max} [\mbox{S}]}{[\mbox{S}] + K_{m}}" class="tex" src="../../images/468/46877.png" /></dl>
      <p><b>The specificity constant</b> <span class="texhtml"><i>k</i><sub><i>c</i><i>a</i><i>t</i></sub> / <i>K</i><sub><i>m</i></sub></span> is a measure of how efficiently an enzyme converts a substrate into product. Using the definition of the Michaelis constant <span class="texhtml"><i>K</i><sub><i>m</i></sub></span>, the Michaelis-Menten equation may be written in the form<dl>
       <dd><img alt="v = k_{2} [\mathrm{ES}] = \frac{k_{2}}{K_{m}} [\mbox{E}] [\mbox{S}]" class="tex" src="../../images/468/46878.png" /></dl>
      <p>where [E] is the concentration of free enzyme. Thus, the specificity constant is an effective bimolecular rate constant for free enzyme to react with free substrate to form product. The specificity constant is limited by the frequency with which the substrate and enzyme encounter each other in solution, roughly 10<sup>10</sup> <!--del_lnk--> M<sup>-1</sup> <!--del_lnk--> s<sup>-1</sup> at 25&deg;C. Remarkably, this maximum does not depend on the size of either the substrate or the enzyme. The ratio of the specificity constants for two substrates is a quantitative comparison of how efficient the enzyme is in converting those substrates. The slope of the Michaelis-Menten equation at low substrate concentration [S] (when [S] &lt;&lt; <i>K</i><sub>m</sub>) also yields the specificity constant.<p><a id="Linear_plots_of_the_Michaelis-Menten_equation" name="Linear_plots_of_the_Michaelis-Menten_equation"></a><h3> <span class="mw-headline">Linear plots of the Michaelis-Menten equation</span></h3>
      <div class="thumb tright">
       <div style="width:252px;"><a class="internal" href="../../images/180/18099.png.htm" title="Lineweaver&ndash;Burk or double-reciprocal plot of kinetic data, showing the significance of the axis intercepts and gradient."><img alt="Lineweaver&ndash;Burk or double-reciprocal plot of kinetic data, showing the significance of the axis intercepts and gradient." height="200" longdesc="/wiki/Image:Lineweaver-Burke_plot.PNG" src="../../images/180/18099.png" width="250" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/180/18099.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Lineweaver&ndash;Burk or double-reciprocal plot of kinetic data, showing the significance of the axis intercepts and gradient.</div>
       </div>
      </div>
      <p>Using an interactive Michaelis&ndash;Menten kinetics tutorial at the University of Virginia, the effects on the behaviour of an enzyme of varying kinetic constants can be explored.<p>The plot of <i>v</i> versus [S] above is not linear; although initially linear at low [S], it bends over to saturate at high [S]. Before the modern era of <!--del_lnk--> nonlinear curve-fitting on computers, this nonlinearity could make it difficult to estimate <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub> accurately. Therefore, several researchers developed linearizations of the Michaelis-Menten equation, such as the <!--del_lnk--> Lineweaver-Burk plot and the <!--del_lnk--> Eadie-Hofstee diagram.<p>The <!--del_lnk--> Lineweaver-Burk plot or <b>double reciprocal</b> plot is common way of illustrating kinetic data. This is produced by taking the <!--del_lnk--> reciprocal of both sides of the Michaelis&ndash;Menten equation. As shown on the right, this is a linear form of the Michaelis&ndash;Menten equation and produces a straight line with the equation <i>y</i> = m<i>x</i> + c with a <i>y</i>-intercept equivalent to 1/<i>V</i><sub>max</sub> and an <i>x</i>-intercept of the graph representing -1/<i>K</i><sub>m</sub>.<dl>
       <dd><img alt="\frac{1}{v} = \frac{K_{m}}{V_{max} [\mbox{S}]} + \frac{1}{V_{max}}" class="tex" src="../../images/468/46880.png" /></dl>
      <p>Naturally, no experimental values can be taken at negative 1/[S]; the lower limiting value 1/[S] = 0 (the <i>y</i>-intercept) corresponds to an infinite substrate concentration, where <i>1/v=1/V<sub>max</sub></i> as shown at the right; thus, the <i>x</i>-intercept is an <!--del_lnk--> extrapolation of the experimental data taken at positive concentrations. More generally, the Lineweaver-Burk plot skews the importance of measurements taken at low substrate concentrations and, thus, can yield inaccurate estimates of <i>V</i><sub>max</sub> and <i>K</i><sub>m</sub>. A more accurate linear plotting method is the <!--del_lnk--> Eadie-Hofstee plot although, in modern research, all such linearizations have been superseded by more reliable <!--del_lnk--> nonlinear regression methods.<p><a id="Practical_significance_of_kinetic_constants" name="Practical_significance_of_kinetic_constants"></a><h3> <span class="mw-headline">Practical significance of kinetic constants</span></h3>
      <p>The study of enzyme kinetics is important for two basic reasons. Firstly, it helps explain how enzymes work, and secondly, it helps predict how enzymes behave in living organisms. The kinetic constants defined above, <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub>, are critical to attempts to understand how enzymes work together to control <!--del_lnk--> metabolism.<p>Making these predictions is not trivial, even for simple systems. For example, <!--del_lnk--> oxaloacetate is formed by <!--del_lnk--> malate dehydrogenase within the <a href="../../wp/m/Mitochondrion.htm" title="Mitochondrion">mitochondrion</a>. Oxaloacetate can then be consumed by <!--del_lnk--> citrate synthase, <!--del_lnk--> phosphoenolpyruvate carboxykinase or <!--del_lnk--> aspartate aminotransferase, feeding into the <!--del_lnk--> citric acid cycle, <!--del_lnk--> gluconeogenesis or <!--del_lnk--> aspartic acid biosynthesis, respectively. Being able to predict how much oxaloacetate goes into which pathway requires knowledge of the concentration of oxaloacetate as well as the concentration and kinetics of each of these enzymes. This aim of predicting the behaviour of metabolic pathways reaches its most complex expression in the synthesis of huge amounts of kinetic and <!--del_lnk--> gene expression data into mathematical models of entire organisms. Although this goal is far in the future for any <a href="../../wp/e/Eukaryote.htm" title="Eukaryote">eukaryote</a>, attempts are now being made to achieve this in <a href="../../wp/b/Bacteria.htm" title="Bacteria">bacteria</a>, with models of <i>Escherichia coli</i> metabolism now being produced and tested.<p><a id="Multi-substrate_reactions" name="Multi-substrate_reactions"></a><h2> <span class="mw-headline">Multi-substrate reactions</span></h2>
      <p>Multi-substrate reactions follow complex rate equations that describe how the substrates bind and in what sequence. The analysis of these reactions is much simpler if the concentration of substrate A is kept constant and substrate B varied. Under these conditions, the enzyme behaves just like a single-substrate enzyme and a plot of <i>v</i> by [S] gives apparent <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub> constants for substrate B. If a set of these measurements is performed at different fixed concentrations of A, these data can be used to work out what the mechanism of the reaction is. For an enzyme that takes two substrates A and B and turns them into two products P and Q, there are two types of mechanism: ternary complex and ping&ndash;pong.<div class="thumb tright">
       <div style="width:302px;"><a class="internal" href="../../images/181/18100.png.htm" title="Random-order ternary-complex mechanism for an enzyme reaction. The reaction path is shown as a line and enzyme intermediates containing substrates A and B or products P and Q are written below the line."><img alt="Random-order ternary-complex mechanism for an enzyme reaction. The reaction path is shown as a line and enzyme intermediates containing substrates A and B or products P and Q are written below the line." height="145" longdesc="/wiki/Image:Random_order_ternary_mechanism.svg" src="../../images/181/18100.png" width="300" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/181/18100.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Random-order ternary-complex mechanism for an enzyme reaction. The reaction path is shown as a line and enzyme intermediates containing substrates A and B or products P and Q are written below the line.</div>
       </div>
      </div>
      <p><a id="Ternary-complex_mechanisms" name="Ternary-complex_mechanisms"></a><h3> <span class="mw-headline">Ternary-complex mechanisms</span></h3>
      <p>In these enzymes, both substrates bind to the enzyme at the same time to produce an EAB ternary complex. The order of binding can either be random (in a random mechanism) or substrates have to bind in a particular sequence (in an ordered mechanism). When a set of <i>v</i> by [S] curves (fixed A, varying B) from an enzyme with a ternary-complex mechanism are plotted in a <!--del_lnk--> Lineweaver&ndash;Burk plot, the set of lines produced will intersect.<p>Enzymes with ternary-complex mechanisms include <!--del_lnk--> glutathione <i>S</i>-transferase, <!--del_lnk--> dihydrofolate reductase and <!--del_lnk--> DNA polymerase. The following links show short animations of the ternary-complex mechanisms of the enzymes dihydrofolate reductase and DNA polymerase.<p><a id="Ping.E2.80.93pong_mechanisms" name="Ping.E2.80.93pong_mechanisms"></a><h3> <span class="mw-headline">Ping&ndash;pong mechanisms</span></h3>
      <div class="thumb tright">
       <div style="width:302px;"><a class="internal" href="../../images/181/18101.png.htm" title="Ping&ndash;pong mechanism for an enzyme reaction. Enzyme intermediates contain substrates A and B or products P and Q."><img alt="Ping&ndash;pong mechanism for an enzyme reaction. Enzyme intermediates contain substrates A and B or products P and Q." height="44" longdesc="/wiki/Image:Ping_pong.svg" src="../../images/181/18101.png" width="300" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/181/18101.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Ping&ndash;pong mechanism for an enzyme reaction. Enzyme intermediates contain substrates A and B or products P and Q.</div>
       </div>
      </div>
      <p>As shown on the right, enzymes with a ping-pong mechanism can exist in two states, E and a chemically modified form of the enzyme E*; this modified enzyme is known as an <!--del_lnk--> intermediate. In such mechanisms, substrate A binds, changes the enzyme to E* by, for example, transferring a chemical group to the active site, and is then released. Only after the first substrate is released can substrate B bind and react with the modified enzyme, regenerating the unmodified E form. When a set of <i>v</i> by [S] curves (fixed A, varying B) from an enzyme with a ping&ndash;pong mechanism are plotted in a Lineweaver&ndash;Burk plot, a set of parallel lines will be produced.<p>Enzymes with ping&ndash;pong mechanisms include some <!--del_lnk--> oxidoreductases such as <!--del_lnk--> thioredoxin peroxidase and <!--del_lnk--> serine proteases such as <!--del_lnk--> trypsin and <!--del_lnk--> chymotrypsin. Serine proteases are a very common and diverse family of enzymes, including <!--del_lnk--> digestive enzymes (trypsin, chymotrypsin, and elastase), several enzymes of the <!--del_lnk--> blood clotting cascade and many others. In these serine proteases, the E* intermediate is an acyl-enzyme species formed by the attack of an active site <!--del_lnk--> serine residue on a <!--del_lnk--> peptide bond in a protein substrate. A short animation showing the mechanism of chymotrypsin is linked here.<p><a id="Non-Michaelis.E2.80.93Menten_kinetics" name="Non-Michaelis.E2.80.93Menten_kinetics"></a><h2> <span class="mw-headline">Non-Michaelis&ndash;Menten kinetics</span></h2>
      <div class="thumb tleft">
       <div style="width:302px;"><a class="internal" href="../../images/181/18102.png.htm" title="Saturation curve for an enzyme reaction showing sigmoid kinetics."><img alt="Saturation curve for an enzyme reaction showing sigmoid kinetics." height="231" longdesc="/wiki/Image:Allosteric_v_by_S_curve.svg" src="../../images/181/18102.png" width="300" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/181/18102.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Saturation curve for an enzyme reaction showing sigmoid kinetics.</div>
       </div>
      </div>
      <p>Some enzymes produce a <!--del_lnk--> sigmoid <i>v</i> by [S] plot, which often indicates <!--del_lnk--> cooperative binding of substrate to the active site. This means that the binding of one substrate molecule affects the binding of subsequent substrate molecules. This behaviour is most common in <!--del_lnk--> multimeric enzymes with several interacting active sites. Here, the mechanism of cooperation is similar to that of <!--del_lnk--> haemoglobin, with binding of substrate to one active site altering the affinity of the other active sites for substrate molecules. Positive cooperativity occurs when binding of the first substrate molecule <i>increases</i> the affinity of the other active sites for substrate. Negative cooperativity occurs when binding of the first substrate <i>decreases</i> the affinity of the enzyme for other substrate molecules.<p>Allosteric enzymes include mammalian tyrosyl tRNA-synthetase, which shows negative cooperativity, and bacterial <!--del_lnk--> aspartate transcarbamoylase and <!--del_lnk--> phosphofructokinase, which show positive cooperativity.<p>Cooperativity is surprisingly common and can help regulate the responses of enzymes to changes in the concentrations of their substrates. Positive cooperativity makes enzymes much more sensitive to [S] and their activities can show large changes over a narrow range of substrate concentration. Conversely, negative cooperativity makes enzymes insensitive to small changes in [S].<p>The <!--del_lnk--> Hill equation is ofetn used to characterize the degree of cooperativity quantitatively for non-Michaelis&ndash;Menten enzymes. The derived Hill coefficient <i>n</i> measures how much the binding of substrate to one active site affects the binding of substrate to the other active sites. A Hill coefficient of &lt;1 indicates negative cooperativity and a coefficient of &gt;1 indicates positive cooperativity.<p><a id="Pre-steady-state_kinetics" name="Pre-steady-state_kinetics"></a><h2> <span class="mw-headline">Pre-steady-state kinetics</span></h2>
      <div class="thumb tright">
       <div style="width:302px;"><a class="internal" href="../../images/181/18103.png.htm" title="Pre-steady state progress curve, showing the burst phase of an enzyme reaction."><img alt="Pre-steady state progress curve, showing the burst phase of an enzyme reaction." height="206" longdesc="/wiki/Image:Burst_phase.png" src="../../images/181/18103.png" width="300" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/181/18103.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Pre-steady state progress curve, showing the burst phase of an enzyme reaction.</div>
       </div>
      </div>
      <p>In the first moment after an enzyme is mixed with substrate, no product has been formed and no <!--del_lnk--> intermediates exist. The study of the next few milliseconds of the reaction is called pre-steady-state kinetics. Pre-steady-state kinetics is therefore concerned with the formation and consumption of enzyme&ndash;substrate intermediates (such as ES or E*) until their <!--del_lnk--> steady-state concentrations are reached.<p>This approach was first applied to the hydrolysis reaction catalysed by <!--del_lnk--> chymotrypsin. Often, the detection of an intermediate is a vital piece of evidence in investigations of what mechanism an enzyme follows. For example, in the ping&ndash;pong mechanisms that are shown above, rapid kinetic measurements can follow the release of product P and measure the formation of the modified enzyme intermediate E*. In the case of chymotrypsin, this intermediate is formed by the attack of the substrate by the <!--del_lnk--> nucleophilic serine in the active site and the formation of the acyl-enzyme intermediate.<p>In the figure to the right, the enzyme produces E* rapidly in the first few seconds of the reaction. The rate then slows as steady state is reached. This rapid burst phase of the reaction measures a single turnover of the enzyme. Consequently, the amount of product released in this burst, shown as the intercept on the <i>y</i>-axis of the graph, also gives the amount of functional enzyme which is present in the assay.<p><a id="Chemical_mechanism" name="Chemical_mechanism"></a><h2> <span class="mw-headline">Chemical mechanism</span></h2>
      <p>An important goal of measuring enzyme kinetics is to determine the chemical mechanism of an enzyme reaction, i.e., the sequence of chemical steps that transform substrate into product. The kinetic approaches discussed above will show at what rates <!--del_lnk--> intermediates are formed and inter-converted, but they cannot identify exactly what these intermediates are.<p>Kinetic measurements taken under various solution conditions or on slightly modified enzymes or substrates often shed light on this chemical mechanism, as they reveal the rate-determining step or intermediates in the reaction. For example, the breaking of a <!--del_lnk--> covalent bond to a <a href="../../wp/h/Hydrogen.htm" title="Hydrogen">hydrogen</a> <a href="../../wp/a/Atom.htm" title="Atom">atom</a> is a common rate-determining step. Which of the possible hydrogen transfers is rate determining can be shown by measuring the kinetic effects of substituting each hydrogen by <!--del_lnk--> deuterium, its stable <!--del_lnk--> isotope. The rate will change when the critical hydrogen is replaced, due to a primary <!--del_lnk--> kinetic isotope effect, which occurs because bonds to deuterium are harder to break then bonds to hydrogen. It is also possible to measure similar effects with other isotope substitutions, such as <sup>13</sup>C/<sup>12</sup>C and <sup>18</sup>O/<sup>16</sup>O, but these effects are more subtle.<p>Isotopes can also be used to reveal the fate of various parts of the substrate molecules in the final products. For example, it is sometimes difficult to discern the origin of an <a href="../../wp/o/Oxygen.htm" title="Oxygen">oxygen</a> atom in the final product; since it may have come from water or from part of the substrate. This may be determined by systematically substituting oxygen&#39;s stable isotope <sup>18</sup>O into the various molecules that participate in the reaction and checking for the isotope in the product. The chemical mechanism can also be elucidated by examining the kinetics and isotope effects under different pH conditions, by altering the metal ions or other bound <!--del_lnk--> cofactors, by <!--del_lnk--> site-directed mutagenesis of conserved amino acid residues, or by studying the behaviour of the enzyme in the presence of analogues of the substrate(s).<p><a id="Enzyme_inhibition" name="Enzyme_inhibition"></a><h2> <span class="mw-headline">Enzyme inhibition</span></h2>
      <div class="thumb tright">
       <div style="width:302px;"><a class="internal" href="../../images/181/18104.png.htm" title="Kinetic scheme for reversible enzyme inhibitors."><img alt="Kinetic scheme for reversible enzyme inhibitors." height="128" longdesc="/wiki/Image:Reversible_inhibition.svg" src="../../images/181/18104.png" width="300" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/181/18104.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Kinetic scheme for reversible enzyme inhibitors.</div>
       </div>
      </div>
      <dl>
       <dd>
      </dl>
      <p>Enzyme inhibitors are molecules that reduce or abolish enzyme activity. These are either <i>reversible</i> (i.e., removal of the inhibitor restores enzyme activity) or <i>irreversible</i> (i.e., the inhibitor permanently inactivates the enzyme).<p><a id="Reversible_inhibitors" name="Reversible_inhibitors"></a><h3> <span class="mw-headline">Reversible inhibitors</span></h3>
      <p>Reversible enzyme inhibitors can be classified as competitive, uncompetitive, non-competitive or mixed, according to their effects on <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub>. These different effects result from the inhibitor binding to the enzyme E, to the enzyme&ndash;substrate complex ES, or to both, as shown in the figure to the right and the table below. The particular type of an inhibitor can be discerned by studying the enzyme kinetics as a function of the inhibitor concentration. The four types of inhibition produce <!--del_lnk--> Lineweaver&ndash;Burke and <!--del_lnk--> Eadie&ndash;Hofstee plots that vary in distinctive ways with inhibitor concentration. For brevity, two symbols are used:<dl>
       <dd><img alt="\alpha = 1 + \frac{[\mbox{I}]}{K_{i}}" class="tex" src="../../images/468/46886.png" />&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;and&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<img alt="\alpha^{\prime} = 1 + \frac{[\mbox{I}]}{K_{i}^{\prime}}" class="tex" src="../../images/468/46887.png" /></dl>
      <p>where <i>K</i><sub>i</sub> and <i>K&#39;</i><sub>i</sub> are the <!--del_lnk--> dissociation constants for binding to the enzyme and to the enzyme&ndash;substrate complex, respectively. In the presence of the reversible inhibitor, the enzyme&#39;s apparent <i>K</i><sub>m</sub> and <i>V</i><sub>max</sub> become (&alpha;/&alpha;&#39;)<i>K</i><sub>m</sub> and (1/&alpha;&#39;)<i>V</i><sub>max</sub>, respectively, as shown below for common cases.<table align="center" border="0" width="75%">
       <tr>
        <td align="center" width="10%">
        </td>
        <td align="center" width="10%">
        </td>
        <td align="center" bgcolor="#FFC3C3" width="20%"><b>Type of inhibition</b></td>
        <td align="center" bgcolor="#FFC3C3" width="20%"><b><i>K</i><sub>m</sub> apparent</b></td>
        <td align="center" bgcolor="#FFC3C3" width="20%"><b><i>V</i><sub>max</sub> apparent</b></td>
       </tr>
       <tr>
        <td align="center" bgcolor="#DDDDFF"><i>K</i><sub>i</sub> only</td>
        <td align="center">(<img alt="\alpha^{\prime}=1" class="tex" src="../../images/468/46888.png" />)</td>
        <td align="center" bgcolor="#DDFFDD">competitive</td>
        <td align="center"><img alt="K_m \alpha~" class="tex" src="../../images/468/46889.png" /></td>
        <td align="center"><img alt="~V_{max}~" class="tex" src="../../images/468/46890.png" /></td>
       </tr>
       <tr>
        <td align="center" bgcolor="#DDDDFF"><i>K</i><sub>i</sub>&#39; only</td>
        <td align="center">(<img alt="\alpha=1~" class="tex" src="../../images/468/46891.png" />)</td>
        <td align="center" bgcolor="#DDFFDD">uncompetitive</td>
        <td align="center"><img alt="\frac{K_m}{\alpha^{\prime}}" class="tex" src="../../images/468/46892.png" /></td>
        <td align="center"><img alt="\frac{V_{max}}{\alpha^{\prime}}" class="tex" src="../../images/468/46893.png" /></td>
       </tr>
       <tr>
        <td align="center" bgcolor="#DDDDFF"><i>K</i><sub>i</sub> = <i>K</i><sub>i</sub>&#39;</td>
        <td align="center">(<img alt="\alpha = \alpha^{\prime}" class="tex" src="../../images/468/46894.png" />)</td>
        <td align="center" bgcolor="#DDFFDD">non-competitive</td>
        <td align="center"><img alt="~K_m~" class="tex" src="../../images/468/46895.png" /></td>
        <td align="center"><img alt="\frac{V_{max}}{\alpha^{\prime}}" class="tex" src="../../images/468/46893.png" /></td>
       </tr>
       <tr>
        <td align="center" bgcolor="#DDDDFF"><i>K</i><sub>i</sub> &ne; <i>K</i><sub>i</sub>&#39;</td>
        <td align="center">(<img alt="\alpha \neq \alpha^{\prime}" class="tex" src="../../images/468/46896.png" />)</td>
        <td align="center" bgcolor="#DDFFDD">mixed</td>
        <td align="center"><img alt="\frac{K_m\alpha}{\alpha^{\prime}}" class="tex" src="../../images/468/46897.png" /></td>
        <td align="center"><img alt="\frac{V_{max}}{\alpha^{\prime}}" class="tex" src="../../images/468/46893.png" /></td>
       </tr>
      </table>
      <p><!--del_lnk--> Non-linear regression fits of the enzyme kinetics data to the rate equations above can yield accurate estimates of the dissociation constants <i>K</i><sub>i</sub> and <i>K&#39;</i><sub>i</sub>.<p><a id="Irreversible_inhibitors" name="Irreversible_inhibitors"></a><h3> <span class="mw-headline">Irreversible inhibitors</span></h3>
      <p>Enzyme inhibitors can also irreversibly inactivate enzymes, usually by covalently modifying active site residues. These reactions follow <!--del_lnk--> exponential decay functions and are usually saturable. Below saturation, they follow <!--del_lnk--> first order kinetics with respect to inhibitor.<p><a id="Mechanisms_of_catalysis" name="Mechanisms_of_catalysis"></a><h2> <span class="mw-headline">Mechanisms of catalysis</span></h2>
      <div class="thumb tright">
       <div style="width:302px;"><a class="internal" href="../../images/137/13719.png.htm" title="The energy variation as a function of reaction coordinate shows the stabilisation of the transition state by an enzyme."><img alt="The energy variation as a function of reaction coordinate shows the stabilisation of the transition state by an enzyme." height="235" longdesc="/wiki/Image:Activation2_updated.svg" src="../../images/137/13719.png" width="300" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/137/13719.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> The energy variation as a function of <!--del_lnk--> reaction coordinate shows the stabilisation of the transition state by an enzyme.</div>
       </div>
      </div>
      <dl>
       <dd>
      </dl>
      <p>The favoured model for the enzyme&ndash;substrate interaction is the induced fit model. This model proposes that the initial interaction between enzyme and substrate is relatively weak, but that these weak interactions rapidly induce conformational changes in the enzyme that strengthen binding. These <!--del_lnk--> conformational changes also bring catalytic residues in the active site close to the chemical bonds in the substrate that will be altered in the reaction. After binding takes place, one or more mechanisms of catalysis lower the energy of the reaction&#39;s <!--del_lnk--> transition state, by providing an alternative chemical pathway for the reaction. Mechanisms of catalysis include catalysis by bond strain: by proximity and orientation: by active-site proton donors or acceptors: covalent catalysis and quantum tunnelling.<p>Enzyme kinetics cannot prove which modes of catalysis are used by an enzyme. However, some kinetic data can suggest possibilities to be examined by other techniques. For example, a ping&ndash;pong mechanism with burst-phase pre-steady-state kinetics would suggest covalent catalysis might be important in this enzyme&#39;s mechanism. Alternatively, the observation of a strong pH effect on <i>V</i><sub>max</sub> but not <i>K</i><sub>m</sub> might indicate that a residue in the active site needs to be in a particular <!--del_lnk--> ionisation state for catalysis to occur.<p><a id="See_also" name="See_also"></a><div class="printfooter"> Retrieved from &quot;<!--del_lnk--> http://en.wikipedia.org/wiki/Enzyme_kinetics&quot;</div>
      <!-- end content -->
      <div class="visualClear">
      </div>
     </div>
    </div>
   </div>
   <!-- end of the left (by default at least) column -->
   <div class="visualClear">
   </div>
   <div id="footer">
    <div class="center"> This reference article is mainly selected from the English Wikipedia with only minor checks and changes (see www.wikipedia.org for details of authors and sources) and is available under the <nobr><a href="../../wp/w/Wikipedia_Text_of_the_GNU_Free_Documentation_License.htm">GNU Free Documentation License</a></nobr>. See also our <b><a href="../../disclaimer.htm">Disclaimer</a></b>. </div>
   </div>
   <script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
  </div>
  
 </body>
</html>
