<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html dir="ltr" lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
 <head>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
  <meta content="Genetic code,Reading frames,Pyrimidine,Purine,Cysteine,Asparagine,Nucleotide,Amino acid,DNA,RNA,Peptide" name="keywords" />
  <link href="../../favicon.ico" rel="shortcut icon" />
  <link href="../../wp/w/Wikipedia_Text_of_the_GNU_Free_Documentation_License.htm" rel="copyright" />
  <title>Genetic code</title>
  <style media="screen,projection" type="text/css">/*<![CDATA[*/ @import "../../css/wp-monobook-main.css"; /*]]>*/</style>
  <link href="../../css/wp-commonPrint.css" media="print" rel="stylesheet" type="text/css" />
  <!--[if lt IE 5.5000]><style type="text/css">@import "../../css/IE50Fixes.css";</style><![endif]-->
  <!--[if IE 5.5000]><style type="text/css">@import "../../css/IE55Fixes.css";</style><![endif]-->
  <!--[if IE 6]><style type="text/css">@import "../../css/IE60Fixes.css";</style><![endif]-->
  <!--[if IE 7]><style type="text/css">@import "../../css/IE70Fixes.css";</style><![endif]-->
  <!--[if lt IE 7]><script type="text/javascript" src="../../js/IEFixes.js"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->
  <script type="text/javascript">
			var skin = "monobook";
			var stylepath = "/skins-1.5";

			var wgArticlePath = "/wiki/$1";
			var wgScriptPath = "/w";
			var wgServer = "http://en.wikipedia.org";
                        
			var wgCanonicalNamespace = "";
			var wgNamespaceNumber = 0;
			var wgPageName = "Genetic_code";
			var wgTitle = "Genetic code";
			var wgArticleId = 12385;
			var wgIsArticle = true;
                        
			var wgUserName = null;
			var wgUserLanguage = "en";
			var wgContentLanguage = "en";
		</script>
  <script src="../../js/wikibits.js" type="text/javascript"><!-- wikibits js --></script>
  <script src="../../js/wp.js" type="text/javascript"><!-- site js --></script>
  <style type="text/css">/*<![CDATA[*/
@import "../../css/wp-common.css";
@import "../../css/wp-monobook.css";
@import "../../css/wp.css";
/*]]>*/</style>
  <!-- Head Scripts -->
 </head>
 <body class="mediawiki ns-0 ltr page-Genetic_code">
  <div id="globalWrapper">
   <div id="column-content">
    <div id="content"><a id="top" name="top"></a><h1 class="firstHeading">Genetic code</h1>
     <div id="bodyContent">
      <h3 id="siteSub"><a href="../../index.htm">2007 Schools Wikipedia Selection</a>. Related subjects: <a href="../index/subject.Science.Biology.General_Biology.htm">General Biology</a></h3>
      <!-- start content -->
      <div class="thumb tright">
       <div style="width:182px;"><a class="internal" href="../../images/184/18485.png.htm" title="A series of codons in an short RNA molecule. Each codon consists of three nucleotides, representing a single amino acid."><img alt="A series of codons in an short RNA molecule. Each codon consists of three nucleotides, representing a single amino acid." height="306" longdesc="/wiki/Image:RNA-codon.png" src="../../images/184/18485.png" width="180" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/184/18485.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> A series of codons in an short <!--del_lnk--> RNA molecule. Each codon consists of three <!--del_lnk--> nucleotides, representing a single <!--del_lnk--> amino acid.</div>
       </div>
      </div>
      <p>The <b>genetic code</b> is the set of rules by which information encoded in genetic material (<a href="../../wp/d/DNA.htm" title="DNA">DNA</a> or <!--del_lnk--> RNA sequences) is translated into <a href="../../wp/p/Protein.htm" title="Protein">proteins</a> (<!--del_lnk--> amino acid sequences) by living <a href="../../wp/c/Cell_%2528biology%2529.htm" title="Cell (biology)">cells</a>. Specifically, the code defines a mapping between tri-<!--del_lnk--> nucleotide sequences called <b>codons</b> and amino acids; every triplet of nucleotides in a nucleic acid sequence specifies a single amino acid. Most organisms use a nearly universal code that is referred to as the <b>standard genetic code</b>. Even <a href="../../wp/v/Virus.htm" title="Virus">viruses</a>, which are not cellular and do not synthesize proteins themselves, have proteins made using this standard code. For a time, therefore, the code was thought to be universal. However, there are notable exceptions. It is also possible for a single organism to translate different parts of the genome in different ways. For example, in humans, protein synthesis in <a href="../../wp/m/Mitochondrion.htm" title="Mitochondrion">mitochondria</a> relies on a modified genetic code that varies from the standard one.<p>
       <script type="text/javascript">
//<![CDATA[
 if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } 
//]]>
</script><a id="Cracking_the_genetic_code" name="Cracking_the_genetic_code"></a><h2> <span class="mw-headline">Cracking the genetic code</span></h2>
      <p>After the structure of DNA was deciphered by <!--del_lnk--> James Watson, <a href="../../wp/f/Francis_Crick.htm" title="Francis Crick">Francis Crick</a> and <!--del_lnk--> Rosalind Franklin, serious efforts to understand the nature of the encoding of proteins began. <!--del_lnk--> George Gamov postulated that a three-letter code must be employed to encode the 20 different <!--del_lnk--> amino acids used by living cells to encode proteins. The first elucidation of a codon was done by <!--del_lnk--> Marshall Nirenberg and <!--del_lnk--> Heinrich J. Matthaei in 1961 at the <!--del_lnk--> National Institutes of Health. They used a cell-free system to <!--del_lnk--> translate a poly-uracil RNA sequence (or UUUUU... in biochemical terms) and discovered that the <!--del_lnk--> polypeptide they had synthesized consisted of only the amino acid <!--del_lnk--> phenylalanine. They, thereby deduced from this poly-phenylalanine that the codon UUU specified the amino-acid <!--del_lnk--> phenylalanine. Extending this work, Nirenberg and his coworkers were able to determine the nucleotide makeup of each codon. In order to determine the order of the sequence, trinucleotides were bound to ribosomes and radioactivaly labeled aminoacyl-tRNA was used to determine, which amino acid corresponded to the codon. Nirenberg&#39;s group was able to determine the the sequences of 54 out of 64 codons. Subsequent work by <!--del_lnk--> Har Gobind Khorana identified the rest of the code, and shortly thereafter <!--del_lnk--> Robert W. Holley determined the structure <!--del_lnk--> transfer RNA, the adapter molecule that facilitates translation. In 1968, Khorana, Holley and Nirenberg shared the <!--del_lnk--> Nobel Prize in Physiology or Medicine for their work.<p><a id="Transfer_of_information_via_the_genetic_code" name="Transfer_of_information_via_the_genetic_code"></a><h2> <span class="mw-headline">Transfer of information via the genetic code</span></h2>
      <p>The genetic information carried by an organism, its <!--del_lnk--> genome, is inscribed in one or more <a href="../../wp/d/DNA.htm" title="DNA">DNA</a>, or in some cases <!--del_lnk--> RNA, molecules. Each functional portion of a DNA or RNA molecule is referred to as a <!--del_lnk--> gene. The gene sequence inscribed in DNA, and in RNA, is composed of tri-nucleotide units called <b>codons</b>, each coding for a single amino acid. Each nucleotide sub-unit consists of a <!--del_lnk--> phosphate, <!--del_lnk--> deoxyribose sugar and one of the 4 nitrogenous <!--del_lnk--> nucleotide bases grouped into 2 categories, <!--del_lnk--> purine and <!--del_lnk--> pyrimidine. The purine bases <!--del_lnk--> adenine (<b>A</b>) and <!--del_lnk--> guanine (<b>G</b>) are larger and consist of two aromatic rings. The pyrimidine bases <!--del_lnk--> cytosine (<b>C</b>) and <!--del_lnk--> thymine (<b>T</b>) are smaller and consist of only one aromatic ring. The percentages of adenine and thymine in the molecule are always the same, as well as percentages of cytosine and guanine. This is due to the fact that hydogen bonds can only form between these nucleotides- this is known as base pairing. In RNA, however, thymine (<b>T</b>) is substituted by <!--del_lnk--> uracil (<b>U</b>), and the deoxyribose is substituted by <!--del_lnk--> ribose.<p>Each protein-coding gene is <!--del_lnk--> transcribed into a short template molecule of the related polymer RNA, known as <!--del_lnk--> messenger RNA or mRNA. This in turn is <!--del_lnk--> translated on the <!--del_lnk--> ribosome into an <!--del_lnk--> amino acid chain or <!--del_lnk--> polypeptide, which will then fold, resulting in secondary and tertiary structures. The process of translation requires <!--del_lnk--> transfer RNAs specific for individual amino acids with the amino acids <!--del_lnk--> covalently attached to them, <!--del_lnk--> guanosine triphosphate as an energy source, and a number of translation factors. tRNAs have anticodons complementary to the codons in mRNA and can be &quot;charged&quot; covalently with amino acids at their 3&#39; terminal CCA ends. Individual tRNAs are charged with specific amino acids by enzymes known as <!--del_lnk--> aminoacyl tRNA synthetases which have high specificity for both their cognate amino acids and tRNAs. The high specificity of these enzymes is a major reasons why the fidelity of protein translation is maintained.<p>Theoretically, there are 4&sup3; = 64 different codon combinations possible with a triplet codon of three nucleotides. In reality, all 64 codons of the standard genetic code are assigned for either amino acids or stop signals during translation. If, for example, an RNA sequence, UUUAAACCC is considered and the <b><!--del_lnk--> reading-frame</b> starts with the first U (by convention, <!--del_lnk--> 5&#39; to 3&#39;), there are three codons, namely, UUU, AAA and CCC, each of which specifies one amino acid. This RNA sequence will be translated into an amino acid sequence, three amino acids long.<p>The standard genetic code is shown in the following tables. Table 1 shows what amino acid each of the 64 codons specifies. Table 2 shows what codons specify each of the 20 standard amino acids involved in translation. These are called forward and reverse codon tables, respectively. For example, the codon AAU represents the amino acid <!--del_lnk--> asparagine, and UGU and UGC represent <!--del_lnk--> cysteine {standard three-letter designations, Asn and Cys respectively).<p><a id="Table_1:_RNA_codon_table" name="Table_1:_RNA_codon_table"></a><h2> <span class="mw-headline">Table 1: RNA codon table</span></h2>
      <table border="1" cellpadding="3" cellspacing="0" id="Table_1">
       <caption>This table shows the 64 codons and the amino acid each codon codes for. The direction is <!--del_lnk--> 5&#39; to 3&#39;.</caption>
       <tr>
        <td colspan="2" rowspan="2">
        </td>
        <th colspan="4">2nd base</th>
       </tr>
       <tr>
        <th>U</th>
        <th>C</th>
        <th>A</th>
        <th>G</th>
       </tr>
       <tr>
        <th rowspan="4">1st<br /> base</th>
        <th>U</th>
        <td>
         <p>UUU&nbsp;(Phe/F)<!--del_lnk--> Phenylalanine<br /> UUC&nbsp;(Phe/F)Phenylalanine<br /> UUA&nbsp;(Leu/L)<!--del_lnk--> Leucine<br /> UUG&nbsp;(Leu/L)Leucine<br />
        </td>
        <td>
         <p>UCU&nbsp;(Ser/S)<!--del_lnk--> Serine<br /> UCC&nbsp;(Ser/S)Serine<br /> UCA&nbsp;(Ser/S)Serine<br /> UCG&nbsp;(Ser/S)Serine<br />
        </td>
        <td>
         <p>UAU&nbsp;(Tyr/Y)<!--del_lnk--> Tyrosine<br /> UAC&nbsp;(Tyr/Y)Tyrosine<br /> UAA&nbsp;Ochre&nbsp;(<i>Stop</i>)<br /> UAG&nbsp;Amber&nbsp;(<i>Stop</i>)<br />
        </td>
        <td>
         <p>UGU&nbsp;(Cys/C)<!--del_lnk--> Cysteine<br /> UGC&nbsp;(Cys/C)Cysteine<br /> UGA&nbsp;Opal&nbsp;(<i>Stop</i>)<br /> UGG&nbsp;(Trp/W)<!--del_lnk--> Tryptophan<br />
        </td>
       </tr>
       <tr>
        <th>C</th>
        <td>
         <p>CUU&nbsp;(Leu/L)Leucine<br /> CUC&nbsp;(Leu/L)Leucine<br /> CUA&nbsp;(Leu/L)Leucine<br /> CUG&nbsp;(Leu/L)Leucine<br />
        </td>
        <td>
         <p>CCU&nbsp;(Pro/P)<!--del_lnk--> Proline<br /> CCC&nbsp;(Pro/P)Proline<br /> CCA&nbsp;(Pro/P)Proline<br /> CCG&nbsp;(Pro/P)Proline<br />
        </td>
        <td>
         <p>CAU&nbsp;(His/H)<!--del_lnk--> Histidine<br /> CAC&nbsp;(His/H)Histidine<br /> CAA&nbsp;(Gln/Q)<!--del_lnk--> Glutamine<br /> CAG&nbsp;(Gln/Q)Glutamine<br />
        </td>
        <td>
         <p>CGU&nbsp;(Arg/R)<!--del_lnk--> Arginine<br /> CGC&nbsp;(Arg/R)Arginine<br /> CGA&nbsp;(Arg/R)Arginine<br /> CGG&nbsp;(Arg/R)Arginine<br />
        </td>
       </tr>
       <tr>
        <th>A</th>
        <td>
         <p>AUU&nbsp;(Ile/I)<!--del_lnk--> Isoleucine<br /> AUC&nbsp;(Ile/I)Isoleucine<br /> AUA&nbsp;(Ile/I)Isoleucine<br /> AUG&nbsp;(Met/M)<!--del_lnk--> Methionine,&nbsp;<i>Start</i><br />
        </td>
        <td>
         <p>ACU&nbsp;(Thr/T)<!--del_lnk--> Threonine<br /> ACC&nbsp;(Thr/T)Threonine<br /> ACA&nbsp;(Thr/T)Threonine<br /> ACG&nbsp;(Thr/T)Threonine<br />
        </td>
        <td>
         <p>AAU&nbsp;(Asn/N)<!--del_lnk--> Asparagine<br /> AAC&nbsp;(Asn/N)Asparagine<br /> AAA&nbsp;(Lys/K)<!--del_lnk--> Lysine<br /> AAG&nbsp;(Lys/K)Lysine<br />
        </td>
        <td>
         <p>AGU&nbsp;(Ser/S)<!--del_lnk--> Serine<br /> AGC&nbsp;(Ser/S)Serine<br /> AGA&nbsp;(Arg/R)<!--del_lnk--> Arginine<br /> AGG&nbsp;(Arg/R)Arginine<br />
        </td>
       </tr>
       <tr>
        <th>G</th>
        <td>
         <p>GUU&nbsp;(Val/V)<!--del_lnk--> Valine<br /> GUC&nbsp;(Val/V)Valine<br /> GUA&nbsp;(Val/V)Valine<br /> GUG&nbsp;(Val/V)Valine<br />
        </td>
        <td>
         <p>GCU&nbsp;(Ala/A)<!--del_lnk--> Alanine<br /> GCC&nbsp;(Ala/A)Alanine<br /> GCA&nbsp;(Ala/A)Alanine<br /> GCG&nbsp;(Ala/A)Alanine<br />
        </td>
        <td>
         <p>GAU&nbsp;(Asp/D)<!--del_lnk--> Aspartic&nbsp;acid<br /> GAC&nbsp;(Asp/D)Aspartic&nbsp;acid<br /> GAA&nbsp;(Glu/E)<!--del_lnk--> Glutamic&nbsp;acid<br /> GAG&nbsp;(Glu/E)Glutamic&nbsp;acid<br />
        </td>
        <td>
         <p>GGU&nbsp;(Gly/G)<!--del_lnk--> Glycine<br /> GGC&nbsp;(Gly/G)Glycine<br /> GGA&nbsp;(Gly/G)Glycine<br /> GGG&nbsp;(Gly/G)Glycine<br />
        </td>
       </tr>
      </table>
      <p><a id="Table_2:_Reverse_codon_table" name="Table_2:_Reverse_codon_table"></a><h2> <span class="mw-headline">Table 2: Reverse codon table</span></h2>
      <p>This table shows the 20 standard amino acids used in proteins, and the codons that code for each amino acid.<table class="prettytable">
       <tr>
        <td align="center" valign="top"><b>Ala</b></td>
        <td valign="top">A</td>
        <td valign="top">GCU, GCC, GCA, GCG</td>
        <td align="center" valign="top"><b>Leu</b></td>
        <td valign="top">L</td>
        <td valign="top">UUA, UUG, CUU, CUC, CUA, CUG</td>
       </tr>
       <tr>
        <td align="center" valign="top"><b>Arg</b></td>
        <td valign="top">R</td>
        <td valign="top">CGU, CGC, CGA, CGG, AGA, AGG</td>
        <td align="center" valign="top"><b>Lys</b></td>
        <td valign="top">K</td>
        <td valign="top">AAA, AAG</td>
       </tr>
       <tr>
        <td align="center" valign="top"><b>Asn</b></td>
        <td valign="top">N</td>
        <td valign="top">AAU, AAC</td>
        <td align="center" valign="top"><b>Met</b></td>
        <td valign="top">M</td>
        <td valign="top">AUG</td>
       </tr>
       <tr>
        <td align="center" valign="top"><b>Asp</b></td>
        <td valign="top">D</td>
        <td valign="top">GAU, GAC</td>
        <td align="center" valign="top"><b>Phe</b></td>
        <td valign="top">F</td>
        <td valign="top">UUU, UUC</td>
       </tr>
       <tr>
        <td align="center" valign="top"><b>Cys</b></td>
        <td valign="top">C</td>
        <td valign="top">UGU, UGC</td>
        <td align="center" valign="top"><b>Pro</b></td>
        <td valign="top">P</td>
        <td valign="top">CCU, CCC, CCA, CCG</td>
       </tr>
       <tr>
        <td align="center" valign="top"><b>Gln</b></td>
        <td valign="top">Q</td>
        <td valign="top">CAA, CAG</td>
        <td align="center" valign="top"><b>Ser</b></td>
        <td valign="top">S</td>
        <td valign="top">UCU, UCC, UCA, UCG, AGU,AGC</td>
       </tr>
       <tr>
        <td align="center" valign="top"><b>Glu</b></td>
        <td valign="top">E</td>
        <td valign="top">GAA, GAG</td>
        <td align="center" valign="top"><b>Thr</b></td>
        <td valign="top">T</td>
        <td valign="top">ACU, ACC, ACA, ACG</td>
       </tr>
       <tr>
        <td align="center" valign="top"><b>Gly</b></td>
        <td valign="top">G</td>
        <td valign="top">GGU, GGC, GGA, GGG</td>
        <td align="center" valign="top"><b>Trp</b></td>
        <td valign="top">W</td>
        <td valign="top">UGG</td>
       </tr>
       <tr>
        <td align="center" valign="top"><b>His</b></td>
        <td valign="top">H</td>
        <td valign="top">CAU, CAC</td>
        <td align="center" valign="top"><b>Tyr</b></td>
        <td valign="top">Y</td>
        <td valign="top">UAU, UAC</td>
       </tr>
       <tr>
        <td align="center" valign="top"><b>Ile</b></td>
        <td valign="top">I</td>
        <td valign="top">AUU, AUC, AUA</td>
        <td align="center" valign="top"><b>Val</b></td>
        <td valign="top">V</td>
        <td valign="top">GUU, GUC, GUA, GUG</td>
       </tr>
       <tr>
        <td align="left" colspan="2" valign="top"><i><b>Start</b></i></td>
        <td valign="top">AUG</td>
        <td align="left" colspan="2" valign="top"><i><b>Stop</b></i></td>
        <td valign="top">UAG, UGA, UAA</td>
       </tr>
      </table>
      <p><a id="Salient_features" name="Salient_features"></a><h2> <span class="mw-headline">Salient features</span></h2>
      <p><a id="Reading_frame_of_a_sequence" name="Reading_frame_of_a_sequence"></a><h3> <span class="mw-headline">Reading frame of a sequence</span></h3>
      <p>Note that a <a href="../../wp/g/Genetic_code.htm" title="Codon">codon</a> is defined by the inital nucleotide from which translation starts. For example, the string GGGAAACCC, if read from the first position, contains the codons GGG, AAA and CCC;and if read from the second position, it contains the codons GGA and AAC; if read starting from the third position, GAA and ACC. Partial codons have been ignored in this example. Every sequence can thus be read in three <b><!--del_lnk--> reading frames</b>, each of which will produce a different amino acid sequence (in the given example, Gly-Lys-Pro, Gly-Asp, or Glu-Thr, respectively). With double-stranded DNA there are six possible <!--del_lnk--> reading frames, three in the forward orientation on one strand and three reverse, or on the opposite strand.<p>The actual frame a protein sequence is translated in is defined by a <b><!--del_lnk--> start codon</b>, usually the first AUG codon in the mRNA sequence. Mutations that disrupt the reading frame by insertions or deletions of one or two nucleotide bases are known as <!--del_lnk--> frameshift mutations. These mutations may impair the function of the resulting protein, if it is formed, and are thus rare in <i><!--del_lnk--> in vivo</i> protein-coding sequences. Often such misformed proteins are targeted for proteolytic degradation. One reason for the rareness of frame-shifted mutations being inherited is that if the protein being translated is essential for growth under the selective pressures the organism faces, absence of a functional protein may cause lethality before the organism is viable.<p><a id="Start.2Fstop_codons" name="Start.2Fstop_codons"></a><h3> <span class="mw-headline">Start/stop codons</span></h3>
      <p>Translation starts with a chain <!--del_lnk--> initiation codon (<!--del_lnk--> start codon). Unlike stop codons, the codon alone is not sufficient to begin the process. Nearby sequences and <!--del_lnk--> initiation factors are also required to start translation. The most common start codon is AUG, which also codes for methionine, but other <!--del_lnk--> start codons are also used.<p>The three stop codons have been given names: UAG is <i>amber</i>, UGA is <i>opal</i> (sometimes also called <i>umber</i>), and UAA is <i>ochre</i>. &quot;Amber&quot; was named after its discoverer <!--del_lnk--> Harris Bernstein, whose last name means &quot;amber&quot; in German. The other two stop codons were named &#39;ochre&quot; and &quot;opal&quot; in order to keep the &quot;colour names&quot; theme. Stop codons are also called termination codons and they signal release of the nascent polypeptide from the ribosome due to binding of <!--del_lnk--> release factors in the absence of cognate tRNAs with anticodons complementary to these stop signals.<p><a id="Degeneracy_of_the_genetic_code" name="Degeneracy_of_the_genetic_code"></a><h3> <span class="mw-headline">Degeneracy of the genetic code</span></h3>
      <p>Many codons are <i>redundant</i>, meaning that two or more codons can code for the same amino acid. Degenerate codons may differ in their third positions; e.g., both GAA and GAG code for the amino acid <!--del_lnk--> glutamic acid. A codon is said to be <b>four-fold degenerate</b> if any nucleotide at its third position specifies the same amino acid; it is said to be <b>two-fold degenerate</b> if only two of four possible nucleotides at its third position specify the same amino acid. In two-fold degenerate codons, the equivalent third position nucleotides are always either two <!--del_lnk--> purines (A/G) or two <!--del_lnk--> pyrimidines (C/T). Only two amino acids are specified by a single codon; one of these is the amino-acid <!--del_lnk--> methionine, specified by the codon AUG, which also specifies the start of translation; the other is <!--del_lnk--> tryptophan, specified by the codon UGG. The degeneracy of the genetic code is what accounts for the existence of <!--del_lnk--> silent mutations.<p>Degeneracy results because a triplet code designates 20 amino acids and a stop codon. Because there are four bases, triplet codons are required to produce at least 21 different codes. For example, if there were two bases per codon, then only 16 amino acids could be coded for (4&sup2;=16). Because at least 21 codes are required, then 4&sup3; gives 64 possible codons, meaning that some degeneracy must exist.<p>These properties of the genetic code make it more fault-tolerant for <!--del_lnk--> point mutations. For example, in theory, four-fold degenerate codons can tolerate any point mutation at the third position, although <!--del_lnk--> codon usage bias restricts this in practice in many organisms; two-fold degenerate codons can tolerate one out of the three possible point mutations at the third position. Since transition mutations (purine to purine or pyrimidine to pyrimidine mutations) are more likely than transversion (purine to pyrimidine or vice-versa) mutations, the equivalence of purines or that of pyrimidines at two-fold degenerate sites adds a further fault-tolerance.<p>A practical consequence of redundancy is that some errors in the genetic code only cause a silent mutation or an error that would not affect the protein because the <!--del_lnk--> hydrophilicity or <!--del_lnk--> hydrophobicity is maintained by equivalent substitution of amino acids; for example, a codon of NUN (where N = any nucleotide) tends to code for hydrophobic amino acids. Even so, it is a single point mutation that causes a modified <!--del_lnk--> hemoglobin molecule in <a href="../../wp/s/Sickle-cell_disease.htm" title="Sickle-cell disease">sickle-cell disease</a>. The hydrophilic <!--del_lnk--> glutamate (Glu) is substituted by the hydrophobic <!--del_lnk--> valine (Val), which reduces the solubility of &szlig;-globin. In this case, this mutation causes <!--del_lnk--> hemoglobin to form linear polymers linked by the hydrophobic interaction between the valine groups causing sickle-cell deformation of erythrocytes. Sickle-cell disease is generally not caused by a <i><!--del_lnk--> de novo</i> mutation. Rather it is selected for in <a href="../../wp/m/Malaria.htm" title="Malaria">malarial</a> regions (in a similar way to <a href="../../wp/t/Thalassemia.htm" title="Thalassemia">thalassemia</a>), as <!--del_lnk--> heterozygous people have some resistance to the malarial <i><!--del_lnk--> Plasmodium</i> parasite (<!--del_lnk--> heterozygote advantage).<p>These variable codes for amino acids are allowed because of modified bases in the first base of the <!--del_lnk--> anticodon of the tRNA, and the base-pair formed is called a <!--del_lnk--> wobble base pair. The modified bases include <!--del_lnk--> inosine and the Non-Watson-Crick U-G basepair.<p><a id="Variations" name="Variations"></a><h2> <span class="mw-headline">Variations</span></h2>
      <p>Numerous variations of the standard genetic code are found in <a href="../../wp/m/Mitochondrion.htm" title="Mitochondrion">mitochondria</a>, which are energy-producing organelles that are found inside eukaryotic cells. <i><!--del_lnk--> Mycoplasma</i> translate the codon UGA as tryptophan. <!--del_lnk--> Ciliate <!--del_lnk--> protozoa also have some variation in the genetic code: UAG and often UAA code for glutamine (a variant also found in some <!--del_lnk--> green algae), or UGA codes for cysteine. Another variant is found in some species of the <a href="../../wp/y/Yeast.htm" title="Yeast">yeast</a>, <i><!--del_lnk--> Candida</i>, where CUG codes for <!--del_lnk--> serine. In addition in some rare cases certain proteins may also use alternate initiation (start) codons.<p>In certain proteins, non-standard amino acids are substituted for standard stop codons, depending upon associated signal sequences in the messenger RNA: UGA can code for <!--del_lnk--> selenocysteine and UAG can code for <!--del_lnk--> pyrrolysine as discussed in the relevant articles. A detailed description of variations in the genetic code can be found at the <!--del_lnk--> NCBI web site. However, there may be other non-standard interpretations that are not yet known. Sequencing of genomes may reveal unique genetic codes that allow the incorporation of other novel amino acids into proteins.<p><a id="Origin_of_the_genetic_code" name="Origin_of_the_genetic_code"></a><h2> <span class="mw-headline">Origin of the genetic code</span></h2>
      <p>Despite the variations that exist, the genetic codes used by all known forms of life on Earth are very similar. Since there are many possible genetic codes that are thought to have similar utility to the one used by Earth life, the theory of <a href="../../wp/e/Evolution.htm" title="Evolution">evolution</a> suggests that the genetic code was established very early in the history of life.<p>One can ask the question: is the genetic code completely random, just one set of codon-amino acid correspondences that happened to establish itself and be &quot;frozen in&quot; early in evolution, although <i>functionally</i> any of the many other possible transcription tables would have done just as well? Already a cursory look at the table shows patterns that suggest that this is not the case.<p>There are three themes running through the many theories that seek to explain the evolution of the genetic code (and hence the origin of these patterns). One is illustrated by recent <!--del_lnk--> aptamer experiments which show that some amino acids have a selective chemical affinity for the base triplets that code for them. This suggests that the current, complex transcription mechanism involving <!--del_lnk--> tRNA and associated enzymes may be a later development, and that originally, protein sequences were directly templated on base sequences. Another is that the standard genetic code that we see today grew from a simpler, earlier code through a process of &quot;biosynthetic expansion&quot;. Here the idea is that primordial life &#39;invented&#39; new amino acids (e.g. as by-products of metabolism) and later back-incorporated some of these into the machinery of genetic coding. Although much circumstantial evidence has been found to indicate that originally the number of different amino acids used may have been considerably smaller than today, precise and detailed hypotheses about exactly which amino acids entered the code in exactly what order has proved far more controversial. A third is that <a href="../../wp/n/Natural_selection.htm" title="Natural selection">natural selection</a> organized the codon assignments of the genetic code to minimize the effects of genetic errors (<!--del_lnk--> mutations).<div class="printfooter"> Retrieved from &quot;<!--del_lnk--> http://en.wikipedia.org/wiki/Genetic_code&quot;</div>
      <!-- end content -->
      <div class="visualClear">
      </div>
     </div>
    </div>
   </div>
   <!-- end of the left (by default at least) column -->
   <div class="visualClear">
   </div>
   <div id="footer">
    <div class="center"> This reference article is mainly selected from the English Wikipedia with only minor checks and changes (see www.wikipedia.org for details of authors and sources) and is available under the <nobr><a href="../../wp/w/Wikipedia_Text_of_the_GNU_Free_Documentation_License.htm">GNU Free Documentation License</a></nobr>. See also our <b><a href="../../disclaimer.htm">Disclaimer</a></b>. </div>
   </div>
   <script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
  </div>
  
 </body>
</html>
