<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html dir="ltr" lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
 <head>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
  <meta content="DNA,1953,1997,2006,3&#39; end,5&#39; end,Abstract machine,Adaptation (biology),Adenine,Adenosine,Adenosine diphosphate" name="keywords" />
  <link href="../../favicon.ico" rel="shortcut icon" />
  <link href="../../wp/w/Wikipedia_Text_of_the_GNU_Free_Documentation_License.htm" rel="copyright" />
  <title>DNA</title>
  <style media="screen,projection" type="text/css">/*<![CDATA[*/ @import "../../css/wp-monobook-main.css"; /*]]>*/</style>
  <link href="../../css/wp-commonPrint.css" media="print" rel="stylesheet" type="text/css" />
  <!--[if lt IE 5.5000]><style type="text/css">@import "../../css/IE50Fixes.css";</style><![endif]-->
  <!--[if IE 5.5000]><style type="text/css">@import "../../css/IE55Fixes.css";</style><![endif]-->
  <!--[if IE 6]><style type="text/css">@import "../../css/IE60Fixes.css";</style><![endif]-->
  <!--[if IE 7]><style type="text/css">@import "../../css/IE70Fixes.css";</style><![endif]-->
  <!--[if lt IE 7]><script type="text/javascript" src="../../js/IEFixes.js"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->
  <script type="text/javascript">
			var skin = "monobook";
			var stylepath = "/skins-1.5";

			var wgArticlePath = "/wiki/$1";
			var wgScriptPath = "/w";
			var wgServer = "http://en.wikipedia.org";
                        
			var wgCanonicalNamespace = "";
			var wgNamespaceNumber = 0;
			var wgPageName = "DNA";
			var wgTitle = "DNA";
			var wgArticleId = 7955;
			var wgIsArticle = true;
                        
			var wgUserName = null;
			var wgUserLanguage = "en";
			var wgContentLanguage = "en";
		</script>
  <script src="../../js/wikibits.js" type="text/javascript"><!-- wikibits js --></script>
  <script src="../../js/wp.js" type="text/javascript"><!-- site js --></script>
  <style type="text/css">/*<![CDATA[*/
@import "../../css/wp-common.css";
@import "../../css/wp-monobook.css";
@import "../../css/wp.css";
/*]]>*/</style>
  <!-- Head Scripts -->
 </head>
 <body class="mediawiki ns-0 ltr page-DNA">
  <div id="globalWrapper">
   <div id="column-content">
    <div id="content"><a id="top" name="top"></a><h1 class="firstHeading">DNA</h1>
     <div id="bodyContent">
      <h3 id="siteSub"><a href="../../index.htm">2007 Schools Wikipedia Selection</a>. Related subjects: <a href="../index/subject.Science.Biology.General_Biology.htm">General Biology</a></h3>
      <!-- start content -->
      <div class="thumb tright">
       <div style="width:222px;"><a class="internal" href="../../images/176/17604.png.htm" title="The general structure of a section of DNA"><img alt="The general structure of a section of DNA" height="551" longdesc="/wiki/Image:DNA_Overview.png" src="../../images/176/17604.png" width="220" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/176/17604.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> The general structure of a section of DNA</div>
       </div>
      </div>
      <p><b>Deoxyribonucleic acid</b> (<b>DNA</b>) is a <!--del_lnk--> nucleic acid that contains the <a href="../../wp/g/Genetics.htm" title="Genetics">genetic</a> instructions for the <!--del_lnk--> biological development of a <a href="../../wp/c/Cell_%2528biology%2529.htm" title="Cell (biology)">cellular</a> form of <a href="../../wp/l/Life.htm" title="Life">life</a> or a <a href="../../wp/v/Virus.htm" title="Virus">virus</a>. All known cellular life and some viruses have DNA. DNA is a long <!--del_lnk--> polymer of <!--del_lnk--> nucleotides (a polynucleotide) that encodes the sequence of <!--del_lnk--> amino acid residues in <a href="../../wp/p/Protein.htm" title="Protein">proteins</a>, using the <a href="../../wp/g/Genetic_code.htm" title="Genetic code">genetic code</a>.<p>
       <script type="text/javascript">
//<![CDATA[
 if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } 
//]]>
</script><a id="Inheritance_of_DNA" name="Inheritance_of_DNA"></a><h2> <span class="mw-headline">Inheritance of DNA</span></h2>
      <p>DNA is responsible for the genetic propagation of most <!--del_lnk--> inherited <!--del_lnk--> traits. In humans, these traits range from hair colour to <!--del_lnk--> disease susceptibility. The genetic information encoded by an organism&#39;s DNA is called its <!--del_lnk--> genome. During <!--del_lnk--> cell division, DNA is <!--del_lnk--> replicated, and during <!--del_lnk--> reproduction is transmitted to <!--del_lnk--> offspring.<p>In <a href="../../wp/e/Eukaryote.htm" title="Eukaryote">eukaryotic</a> <a href="../../wp/c/Cell_%2528biology%2529.htm" title="Cell (biology)">cells</a>, such as those of <a href="../../wp/p/Plant.htm" title="Plant">plants</a>, <a href="../../wp/a/Animal.htm" title="Animal">animals</a>, <a href="../../wp/f/Fungus.htm" title="Fungus">fungi</a> and <!--del_lnk--> protists, most of the DNA is located in the <!--del_lnk--> cell nucleus, and each DNA molecule is usually packed into a <!--del_lnk--> chromosome that are passed to daughter cells during <!--del_lnk--> cell division. By contrast, in simpler cells called <!--del_lnk--> prokaryotes, including the <a href="../../wp/b/Bacteria.htm" title="Bacterium">eubacteria</a> and <!--del_lnk--> archaea, DNA is found directly in the <!--del_lnk--> cytoplasm (not separated by a <!--del_lnk--> nuclear envelope) and is circular. The cellular <!--del_lnk--> organelles known as <!--del_lnk--> chloroplasts and <!--del_lnk--> mitochondria also carry DNA. DNA is thought to have originated approximately 3.5 to 4.6 billion years ago.<p>In humans, the mother&#39;s <!--del_lnk--> mitochondrial DNA together with 23 <!--del_lnk--> chromosomes from each parent combine to form the genome of a <!--del_lnk--> zygote, the <!--del_lnk--> fertilized <!--del_lnk--> egg. As a result, with certain exceptions such as <!--del_lnk--> red blood cells, most human cells contain 23 pairs of chromosomes, together with mitochondrial DNA inherited from the mother. <!--del_lnk--> Lineage studies can be done because <!--del_lnk--> mitochondrial DNA only comes from the mother, and the <!--del_lnk--> Y chromosome only comes from the father.<div class="thumb tright">
       <div style="width:183px;"><a class="internal" href="../../images/176/17605.gif.htm" title="Animation of a section of DNA rotating."><img alt="Animation of a section of DNA rotating." height="313" longdesc="/wiki/Image:ADN_animation.gif" src="../../images/176/17605.gif" width="181" /></a><div class="thumbcaption">Animation of a section of DNA rotating.</div>
       </div>
      </div>
      <div class="thumb tright">
       <div style="width:302px;"><a class="internal" href="../../images/176/17606.gif.htm" title="DNA base pairing"><img alt="DNA base pairing" height="257" longdesc="/wiki/Image:Dna_pairing_aa.gif" src="../../images/176/17606.gif" width="300" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/176/17606.gif.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> DNA base pairing</div>
       </div>
      </div>
      <p><a id="Replication" name="Replication"></a><h3> <span class="mw-headline">Replication</span></h3>
      <dl>
       <dd>
      </dl>
      <div class="thumb tright">
       <div style="width:204px;"><a class="internal" href="../../images/189/18948.png.htm" title="DNA replication"><img alt="DNA replication" height="397" longdesc="/wiki/Image:Dna-split.png" src="../../images/176/17607.png" width="202" /></a><div class="thumbcaption">DNA replication</div>
       </div>
      </div>
      <p>The double-stranded structure of DNA provides a mechanism for <!--del_lnk--> DNA replication: the two strands are separated, and then each strand&#39;s complement is recreated by exposing the strand to a mixture of the four bases. An <!--del_lnk--> enzyme makes the complement strand by finding the correct base in the mixture and bonding it with the original strand. In this way, the base on the old strand dictates which base appears on the new strand, and the cell ends up with an extra copy of its DNA.<p><a id="Physical_and_chemical_properties" name="Physical_and_chemical_properties"></a><h2> <span class="mw-headline">Physical and chemical properties</span></h2>
      <div class="thumb tright">
       <div style="width:302px;"><a class="internal" href="../../images/176/17608.png.htm" title="Comparisons between DNA and single stranded RNA with the diagram of the bases showing."><img alt="Comparisons between DNA and single stranded RNA with the diagram of the bases showing." height="305" longdesc="/wiki/Image:NA-comparedto-DNA_thymineAndUracilCorrected.png" src="../../images/176/17608.png" width="300" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/176/17608.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Comparisons between DNA and single stranded RNA with the diagram of the bases showing.</div>
       </div>
      </div>
      <p><a id="Molecular_structure" name="Molecular_structure"></a><h3> <span class="mw-headline">Molecular structure</span></h3>
      <p>Although sometimes called &quot;the molecule of heredity&quot;, DNA macromolecules as people typically think of them are not single molecules. Rather, they are pairs of molecules, which entwine like vines, in the shape of a <b>double <!--del_lnk--> helix</b> (see the illustration at the right).<p>DNA consists of a pair of <!--del_lnk--> molecules, organized as strands running start-to-end and joined by <!--del_lnk--> hydrogen bonds along their lengths. Each strand is a chain of chemical &quot;building blocks&quot;, called <!--del_lnk--> nucleotides, of which there are four types: <!--del_lnk--> adenine (abbreviated A), <!--del_lnk--> cytosine (C), <!--del_lnk--> guanine (G) and <!--del_lnk--> thymine (T). (Thymine should not be confused with <a href="../../wp/t/Thiamine.htm" title="Thiamine">thiamine</a>, which is vitamin B<sub>1</sub>.) The DNA of some organisms, most notably of the PBS1 <!--del_lnk--> phage, have <!--del_lnk--> Uracil (U) instead of T.<p>Each strand of DNA is a covalently linked chain of <!--del_lnk--> nucleotides, with alternating <a href="../../wp/s/Sugar.htm" title="Sugar">sugar</a> (<!--del_lnk--> deoxyribose)-<!--del_lnk--> phosphates forming the &quot;backbone&quot; for the <!--del_lnk--> nucleobases (&quot;bases&quot;). The negatively-charged phosphate groups between each deoxyribose make DNA an acid in solution and allow DNA molecules of different sizes to be separated by <!--del_lnk--> electrophoresis. Because DNA strands are composed of these nucleotide subunits, they are <!--del_lnk--> polymers. The major difference between DNA and RNA is the sugar, 2-deoxyribose in DNA and ribose in RNA.<p><a id="Base_pairing" name="Base_pairing"></a><h3> <span class="mw-headline">Base pairing</span></h3>
      <dl>
       <dd>
      </dl>
      <p>DNA is composed of 4 bases: <!--del_lnk--> adenine (A), <!--del_lnk--> thymine (T), <!--del_lnk--> cytosine (C), and <!--del_lnk--> guanine (G). <!--del_lnk--> Uracil (U), is rarely found in DNA except as a result of chemical degradation of Cytosine, but the DNA of some viruses (notably PBS1 phage DNA) and <!--del_lnk--> RNA (Ribonucleic Acid), has Uracil instead of Thymine.<p>Each base on one strand forms a bond with just one kind of base on another strand, called a &quot;complementary&quot; base: A bonds with T, and C bonds with G. Therefore, the whole double-strand sequence can be described by the sequence on one of the strands, chosen by convention. Two nucleotides paired together are called a <!--del_lnk--> base pair.<p>In a DNA double helix, two polynucleotide strands can associate through the <!--del_lnk--> hydrophobic effect and <!--del_lnk--> pi stacking. Which strands associate depends on <!--del_lnk--> complementary pairing. Each base forms <!--del_lnk--> hydrogen bonds readily to only one other base, A to T forming two hydrogen bonds, and C to G forming three hydrogen bonds. The GC content and length of each DNA molcule dictates the strength of the association; the more complementary bases exist, the stronger and longer-lasting the association, characterised by the temperature required to break the hydrogen bond, its <!--del_lnk--> melting temperature (also called <i>T<sub>m</sub></i> value)).<div class="thumb tright">
       <div style="width:202px;"><a class="internal" href="../../images/52/5268.png.htm" title="The chemical structure of DNA"><img alt="The chemical structure of DNA" height="215" longdesc="/wiki/Image:DNA_chemical_structure.png" src="../../images/176/17609.png" width="200" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/52/5268.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> The chemical structure of DNA</div>
       </div>
      </div>
      <p><a id="Strand_direction" name="Strand_direction"></a><h3> <span class="mw-headline">Strand direction</span></h3>
      <p>The asymmetric shape and linkage of nucleotides means that a DNA strand always has a discernible orientation or directionality. Inspection of a double helix reveals that the direction of the nucleotides in one strand is opposite to their direction in the other strand. This arrangement of the strands is called <b>antiparallel</b>.<dl>
       <dt>Chemical nomenclature (<!--del_lnk--> 5&#39; and <!--del_lnk--> 3&#39;)</dl>
      <p>The assymetric &quot;ends&quot; of the DNA bases are referred to as <!--del_lnk--> 5&#39; (<i>five prime</i>) and <!--del_lnk--> 3&#39; (<i>three prime</i>). Within the nucleus, the enzymes that perform <!--del_lnk--> replication and <!--del_lnk--> transcription read the DNA template in the &quot;<b>3&#39; to 5&#39; direction</b>&quot;, although this directional reading should not be assumed in other cases. In a vertically oriented double helix, the <!--del_lnk--> 3&#39; strand is said to be ascending while the <!--del_lnk--> 5&#39; strand is said to be descending.<dl>
       <dt>Sense and antisense<dd>
      </dl>
      <p>As a result of their antiparallel arrangement and the sequence-reading preferences of enzymes, even if both strands carried identical instead of complementary sequences, cells could properly translate only one of them. The other strand a cell can only read backwards. <!--del_lnk--> Molecular biologists call a sequence &quot;<b>sense</b>&quot; if it is translated or translatable, and they call its complement &quot;<b>antisense</b>&quot;. It follows then, somewhat paradoxically, that the template for transcription is the <i>antisense</i> strand. The resulting transcript is an RNA replica of the <i>sense</i> strand and is itself <i>sense.</i><p>A small proportion of genes in <!--del_lnk--> prokaryotes, and more in <!--del_lnk--> plasmids and <a href="../../wp/v/Virus.htm" title="Viruses">viruses</a>, blur the distinction made above between sense and antisense strands. Certain sequences of their <!--del_lnk--> genomes do double duty, encoding one protein when read 5&#39; to 3&#39; along one strand, and a second protein when read in the opposite direction (still 5&#39; to 3&#39;) along the other strand. As a result, the genomes of these viruses are unusually compact for the number of genes they contain, which biologists view as an <!--del_lnk--> adaptation. This merely confirms that there is no biological distinction between the two strands of the double helix. Typically each strand of a DNA double helix will act as sense and antisense in different regions.<p><a id="Single-stranded_DNA" name="Single-stranded_DNA"></a><h3> <span class="mw-headline">Single-stranded DNA</span></h3>
      <p>In some <a href="../../wp/v/Virus.htm" title="Virus">viruses</a> DNA appears in a non-helical, single-stranded form. Because many of the <a href="../../wp/d/DNA_repair.htm" title="DNA repair">DNA repair</a> mechanisms of cells work only on paired bases, viruses that carry single-stranded DNA <!--del_lnk--> genomes <!--del_lnk--> mutate more frequently than they would otherwise. As a result, such species may adapt more rapidly to avoid extinction. The result would not be so favorable in more complicated and more slowly replicating organisms, however, which may explain why only viruses carry single-stranded DNA. These viruses presumably also benefit from the lower cost of replicating one strand versus two.<p>For further discussion of the physical structure of DNA see <!--del_lnk--> Mechanical properties of DNA.<p><a id="DNA_sequence" name="DNA_sequence"></a><h2> <span class="mw-headline">DNA sequence</span></h2>
      <p>DNA contains the genetic <a href="../../wp/i/Information.htm" title="Information">information</a>, that is inherited by the offspring of an organism. This information is determined by the <!--del_lnk--> sequence of base pairs along its length. A strand of DNA contains <!--del_lnk--> genes, areas that <!--del_lnk--> regulate genes, and areas that either have no function, or a function <!--del_lnk--> yet unknown. Genes are the units of heredity and can be loosely viewed as the organism&#39;s &quot;cookbook&quot; or &quot;blueprint&quot;. DNA is often referred to as the molecule of <!--del_lnk--> heredity.<p><a id="The_genetic_code" name="The_genetic_code"></a><h3> <span class="mw-headline">The genetic code</span></h3>
      <dl>
       <dd>
      </dl>
      <p>Within a gene, the sequence of <!--del_lnk--> nucleotides along a DNA strand defines a messenger RNA sequence which then defines a <a href="../../wp/p/Protein.htm" title="Protein">protein</a>, that an <a href="../../wp/o/Organism.htm" title="Organism">organism</a> is liable to manufacture or &quot;<!--del_lnk--> express&quot; at one or several points in its life using the information of the sequence. The relationship between the nucleotide sequence and the <!--del_lnk--> amino-acid sequence of the protein is determined by simple cellular rules of <!--del_lnk--> translation, known collectively as the <a href="../../wp/g/Genetic_code.htm" title="Genetic code">genetic code</a>. The genetic code consists of three-letter &#39;words&#39; (termed a codon) formed from a sequence of three nucleotides (e.g. ACT, CAG, TTT). These codons can then be translated with <!--del_lnk--> messenger RNA and then <!--del_lnk--> transfer RNA, with a codon corresponding to a particular amino acid. There are 64 possible codons (4 bases in 3 places <span class="texhtml">4<sup>3</sup></span>) that encode 20 amino acids. Most amino acids, therefore, have more than one possible codon. There are also three &#39;stop&#39; or &#39;nonsense&#39; codons signifying the end of the coding region, namely the UAA, UGA and UAG codons.<p><a id="Non-coding_DNA" name="Non-coding_DNA"></a><h3> <span class="mw-headline">Non-coding DNA</span></h3>
      <dl>
       <dd>
      </dl>
      <p>In many <!--del_lnk--> species, only a small fraction of the total sequence of the <!--del_lnk--> genome appears to encode protein. For example, only about 1.5% of the <!--del_lnk--> human genome consists of protein-coding <!--del_lnk--> exons. The function of the rest is a matter of speculation. It is known that certain nucleotide sequences specify affinity for <!--del_lnk--> DNA binding proteins, which play a wide variety of vital roles, in particular through control of replication and transcription. These sequences are frequently called <!--del_lnk--> regulatory sequences, and researchers assume that so far they have identified only a tiny fraction of the total that exist. &quot;<!--del_lnk--> Junk DNA&quot; represents sequences that do not yet appear to contain genes or to have a function. The reasons for the presence of so much <!--del_lnk--> non-coding DNA in <!--del_lnk--> eukaryotic genomes and the extraordinary differences in <!--del_lnk--> genome size (&quot;<!--del_lnk--> C-value&quot;) among species represent a long-standing puzzle in DNA research known as the &quot;<!--del_lnk--> C-value enigma&quot;.<p>Some DNA sequences play structural roles in chromosomes. <!--del_lnk--> Telomeres and <!--del_lnk--> centromeres typically contain few (if any) protein-coding genes, but are important for the function and stability of chromosomes. Some genes code for &quot;RNA genes&quot; (see <!--del_lnk--> tRNA and <!--del_lnk--> rRNA). Some RNA genes code for transcripts that function as regulatory RNAs (see <!--del_lnk--> siRNA) that influence the function of other RNA molecules. The intron-exon structure of some genes (such as immunoglobin and protocadeherin genes) is important for allowing alternative splicing of pre-mRNA which allows several different proteins to be made from the same gene. Indeed, the 34,000 human genes encode some 100,000 proteins. Some non-coding DNA represents <!--del_lnk--> pseudogenes, which have been hypothesized to serve as raw genetic material for the creation of new genes through the process of <!--del_lnk--> gene duplication and <!--del_lnk--> divergence. Some non-coding DNA provided hot-spots for duplication of short DNA regions; such sequence duplication has been the major form of genetic change in the human lineage (see evidence from the <!--del_lnk--> Chimpanzee Genome Project).<p>Sequence also determines a DNA segment&#39;s susceptibility to cleavage by <!--del_lnk--> restriction enzymes, an important tool in <!--del_lnk--> genetic engineering. The position of cleavage sites throughout an individual&#39;s genome determines one kind of an individual&#39;s &quot;<!--del_lnk--> DNA fingerprint&quot;.<p><a id="Mutation" name="Mutation"></a><h3> <span class="mw-headline">Mutation</span></h3>
      <dl>
       <dd>
      </dl>
      <p>A cell&#39;s machinery separates the DNA double helix, and uses each DNA strand as a template for synthesizing a new strand which is nearly identical to the previous strand. Errors that occur in the synthesis are called <!--del_lnk--> mutations. <!--del_lnk--> Mutations are the results of the cells&#39; attempts to repair chemical imperfections in this process, where a base is accidentally skipped, inserted, or incorrectly copied, or the chain is trimmed, or added to. On rare occasions, wrong pairing can happen, when <!--del_lnk--> thymine goes into its <!--del_lnk--> enol form or <!--del_lnk--> cytosine goes into its <!--del_lnk--> imino form. Mutations can also occur after chemical damage (through <!--del_lnk--> mutagens), light (<a href="../../wp/u/Ultraviolet.htm" title="Ultraviolet">UV</a> damage), or through other more complicated gene swapping events. This process of replication is mimiced <!--del_lnk--> in vitro by a process called <a href="../../wp/p/Polymerase_chain_reaction.htm" title="Polymerase chain reaction">Polymerase chain reaction</a> (PCR).<p><a id="The_study_of_DNA" name="The_study_of_DNA"></a><h2> <span class="mw-headline">The study of DNA</span></h2>
      <p><a id="First_isolation_of_DNA" name="First_isolation_of_DNA"></a><h3> <span class="mw-headline">First isolation of DNA</span></h3>
      <p>Working in the 19th century, biochemists initially isolated DNA and RNA (mixed together) from cell nuclei. They were relatively quick to appreciate the polymeric nature of their &quot;nucleic acid&quot; isolates, but realized only later that nucleotides were of two types--one containing <!--del_lnk--> ribose and the other <!--del_lnk--> deoxyribose. It was this subsequent discovery that led to the identification and naming of DNA as a substance distinct from RNA.<p><!--del_lnk--> Friedrich Miescher (1844-1895) discovered a substance he called &quot;nuclein&quot; in 1869. Somewhat later, he isolated a pure sample of the material now known as DNA from the sperm of salmon, and in 1889 his pupil, <!--del_lnk--> Richard Altmann, named it &quot;nucleic acid&quot;. This substance was found to exist only in the chromosomes.<p>In 1929 <!--del_lnk--> Phoebus Levene at the <!--del_lnk--> Rockefeller Institute identified the components (the four bases, the sugar and the phosphate chain) and he showed that the components of DNA were linked in the order phosphate-sugar-base. He called each of these units a <!--del_lnk--> nucleotide and suggested the DNA molecule consisted of a string of nucleotide units linked together through the phosphate groups, which are the &#39;backbone&#39; of the molecule. However Levene thought the chain was short and that the bases repeated in the same fixed order. <!--del_lnk--> Torbjorn Caspersson and <!--del_lnk--> Einar Hammersten showed that DNA was a polymer.<p><a id="Chromosomes_and_inherited_traits" name="Chromosomes_and_inherited_traits"></a><h3> <span class="mw-headline">Chromosomes and inherited traits</span></h3>
      <p><!--del_lnk--> Max Delbr&uuml;ck, <!--del_lnk--> Nikolai V. Timofeeff-Ressovsky, and <!--del_lnk--> Karl G. Zimmer published results in 1935 suggesting that chromosomes are very large molecules the structure of which can be changed by treatment with <!--del_lnk--> X-rays, and that by so changing their structure it was possible to change the heritable characteristics governed by those chromosomes. In 1937 <!--del_lnk--> William Astbury produced the first <!--del_lnk--> X-ray diffraction patterns from DNA. He was not able to propose the correct structure but the patterns showed that DNA had a regular structure and therefore it might be possible to deduce what this structure was.<p>In 1943, <!--del_lnk--> Oswald Theodore Avery and a team of scientists discovered that traits proper to the &quot;smooth&quot; form of the <i>Pneumococcus</i> could be transferred to the &quot;rough&quot; form of the same bacteria merely by making the killed &quot;smooth&quot; (S) form available to the live &quot;rough&quot; (R) form. Quite unexpectedly, the living R <i>Pneumococcus</i> bacteria were transformed into a new strain of the S form, and the transferred S characteristics turned out to be heritable. Avery called the medium of transfer of traits the <!--del_lnk--> transforming principle; he identified DNA as the transforming principle, and not <a href="../../wp/p/Protein.htm" title="Protein">protein</a> as previously thought. He essentially redid <!--del_lnk--> Frederick Griffith&#39;s experiment. In 1953, <!--del_lnk--> Alfred Hershey and <!--del_lnk--> Martha Chase did an experiment (<!--del_lnk--> Hershey-Chase experiment) that showed, in <!--del_lnk--> T2 phage, that DNA is the <!--del_lnk--> genetic material (Hershey shared the Nobel prize with Luria).<p><a id="Discovery_of_the_structure_of_DNA" name="Discovery_of_the_structure_of_DNA"></a><h3> <span class="mw-headline">Discovery of the structure of DNA</span></h3>
      <p>In the 1950s, three groups made it their goal to determine the structure of DNA. The first group to start was at <!--del_lnk--> King&#39;s College London and was led by <!--del_lnk--> Maurice Wilkins and was later joined by <!--del_lnk--> Rosalind Franklin. Another group consisting of <a href="../../wp/f/Francis_Crick.htm" title="Francis Crick">Francis Crick</a> and <a href="../../wp/j/James_D._Watson.htm" title="James D. Watson">James D. Watson</a> was at <a href="../../wp/u/University_of_Cambridge.htm" title="University of Cambridge">Cambridge</a>. A third group was at <!--del_lnk--> Caltech and was led by <a href="../../wp/l/Linus_Pauling.htm" title="Linus Pauling">Linus Pauling</a>. Crick and Watson built physical models using metal rods and balls, in which they incorporated the known chemical structures of the nucleotides, as well as the known position of the linkages joining one nucleotide to the next along the polymer. At King&#39;s College Maurice Wilkins and Rosalind Franklin examined <!--del_lnk--> X-ray diffraction patterns of DNA fibers. Of the three groups, only the London group was able to produce good quality diffraction patterns and thus produce sufficient quantitative data about the structure.<p><a id="Helix_structure" name="Helix_structure"></a><h4> <span class="mw-headline">Helix structure</span></h4>
      <p>In 1948 Pauling discovered that many proteins included helical (see <!--del_lnk--> alpha helix) shapes. Pauling had deduced this structure from X-ray patterns and from attempts to physically model the structures. (Pauling was also later to suggest an incorrect three chain helical structure based on Astbury&#39;s data.) Even in the initial diffraction data from DNA by Maurice Wilkins, it was evident that the structure involved helices. But this insight was only a beginning. There remained the questions of how many strands came together, whether this number was the same for every helix, whether the bases pointed toward the helical axis or away, and ultimately what were the explicit angles and coordinates of all the bonds and atoms. Such questions motivated the modeling efforts of Watson and Crick.<p><a id="Complementary_nucleotides" name="Complementary_nucleotides"></a><h4> <span class="mw-headline">Complementary nucleotides</span></h4>
      <p>In their modeling, Watson and Crick restricted themselves to what they saw as chemically and biologically reasonable. Still, the breadth of possibilities was very wide. A breakthrough occurred in 1952, when <!--del_lnk--> Erwin Chargaff visited Cambridge and inspired Crick with a description of experiments Chargaff had published in 1947. Chargaff had observed that the proportions of the four nucleotides vary between one DNA sample and the next, but that for particular pairs of nucleotides &mdash; adenine and thymine, guanine and cytosine &mdash; the two nucleotides are always present in equal proportions.<p><a id="Watson_and_Crick.27s_model" name="Watson_and_Crick.27s_model"></a><h4> <span class="mw-headline">Watson and Crick&#39;s model</span></h4>
      <div class="thumb tright">
       <div style="width:202px;"><a class="internal" href="../../images/176/17612.jpg.htm" title="Crick and Watson DNA model built in 1953, was reconstructed largely from its original pieces in 1973 and donated to the National Science Museum in London."><img alt="Crick and Watson DNA model built in 1953, was reconstructed largely from its original pieces in 1973 and donated to the National Science Museum in London." height="267" longdesc="/wiki/Image:DNA_Model_Crick-Watson.jpg" src="../../images/176/17612.jpg" width="200" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/176/17612.jpg.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Crick and Watson DNA model built in 1953, was <!--del_lnk--> reconstructed largely from its original pieces in 1973 and donated to the <!--del_lnk--> National Science Museum in London.</div>
       </div>
      </div>
      <p>The discovery that DNA was the carrier of genetic information was a process that required many earlier discoveries. The existence of DNA was discovered in the mid 19th century. However, it was only in the early 20th century that researchers began suggesting that it might store genetic information. This gained almost universal acceptance after the structure of DNA was elucidated by <a href="../../wp/j/James_D._Watson.htm" title="James D. Watson">James D. Watson</a> and <a href="../../wp/f/Francis_Crick.htm" title="Francis Crick">Francis Crick</a> in their 1953 <!--del_lnk--> <i>Nature</i> publication. Watson and Crick proposed the <!--del_lnk--> central dogma of molecular biology in 1957, describing the process whereby proteins are produced from <!--del_lnk--> nucleic DNA. In 1962 Watson, Crick, and <!--del_lnk--> Maurice Wilkins jointly received the <!--del_lnk--> Nobel Prize for their determination of the structure of DNA.<p>In spite of all this, the prize presented to Watson and Crick was indeed very controversial. In 1951, <!--del_lnk--> Rosalind Franklin, a physical chemist working in Paris, was researching DNA&#39;s structure at King&#39;s College and gave a department lecture on her work at the time on DNA. Watson attended this lecture and initially learned of Franklin&#39;s data, but he did not take notes. This led to an initial structure proposed by Watson and Crick, which Franklin refuted when she revealed information that Watson had neglected to write from attending her lecture.<p><a href="../../wp/j/James_D._Watson.htm" title="James D. Watson">Watson</a> and <a href="../../wp/f/Francis_Crick.htm" title="Francis Crick">Crick</a> had begun to contemplate double helical arrangements, but they lacked information about the amount of twist (pitch) and the distance between the two strands. <!--del_lnk--> Rosalind Franklin had to disclose some of her findings for the <!--del_lnk--> Medical Research Council and Crick saw this material through <!--del_lnk--> Max Perutz&#39;s links to the MRC. Franklin&#39;s work confirmed that the phosphate &quot;backbone&quot; was on the outside of the molecule and also gave an insight into its symmetry, in particular that the two helical strands ran in opposite directions. In the end, however, it turned out that much of Franklin&#39;s data from this MRC report had been presented in that open seminar where Watson had neglected to take notes.<p>Watson and Crick were again greatly assisted by more of Franklin&#39;s data. This is controversial because Franklin&#39;s critical X-ray pattern was shown to Watson and Crick without Franklin&#39;s knowledge or permission. Wilkins showed the famous Photo 51 of the much simpler <i>B</i> type of DNA to Watson at his lab immediately after Watson had been unsuccessful in asking Franklin to collaborate to beat Pauling in finding the structure.<p>From the data in photograph 51 Watson and Crick were able to discern that not only was the distance between the two strands constant, but also to measure its exact value of 2 nanometres. The same photograph also gave them the 3.4 nanometre-per-10 bp &quot;pitch&quot; of the helix.<p>The final insight came when Crick and Watson saw that a complementary pairing of the bases could provide an explanation for Chargaff&#39;s puzzling finding. However the structure of the bases had been incorrectly guessed in the textbooks as the <!--del_lnk--> enol <!--del_lnk--> tautomer when they were more likely to be in the <!--del_lnk--> keto form. When <!--del_lnk--> Jerry Donohue pointed this fallacy out to Watson, Watson quickly realised that the pairs of adenine and thymine, and guanine and cytosine were almost identical in shape and so would provide equally sized &#39;rungs&#39; between the two strands. Watson and Crick worked to develop a physical model of the double-helical structure out of wire which they used to confirm that the distances between the molecules were permissible. With the base-pairing, the Watson and Crick quickly converged upon a model, which they announced before Franklin herself had published any of her work.<p>The disclosure of Franklin&#39;s data to Watson has angered some people who believe Franklin did not receive due credit at the time and that she might have discovered the structure on her own before Crick and Watson. In Crick and Watson&#39;s famous paper in Nature in 1953, they said that their work had been stimulated by the work of Wilkins and Franklin, whereas it had been the basis of their work. However they had agreed with Wilkins and Franklin that they all should publish papers in the same issue of <i>Nature</i> in support of the proposed structure. Additionally, in his autobiography, <i><!--del_lnk--> The Double Helix</i>, Watson describes Franklin in very unflattering terms (commenting derisively on her lack of &quot;feminine&quot; traits) and all but implies that her work actually impaired that of Wilkins.<p>Franklin died in 1958 and four years later, Watson, Crick and Wilkins won the Nobel Prize for their work on the structure of DNA. Because the Nobel Prize is not awarded posthumously, Franklin could not share in it.<p><a name=".22Central_Dogma.22"></a><h4> <span class="mw-headline">&quot;Central Dogma&quot;</span></h4>
      <p>Watson and Crick&#39;s model attracted great interest immediately upon its presentation. Arriving at their conclusion on <!--del_lnk--> February 21, <!--del_lnk--> 1953, Watson and Crick made their first announcement on <!--del_lnk--> February 28. Their paper, <i>A Structure for Deoxyribose Nucleic Acid</i>, was published on April 25. In an influential presentation in 1957, Crick laid out the &quot;<!--del_lnk--> Central Dogma&quot;, which foretold the relationship between DNA, RNA, and proteins, and articulated the &quot;sequence hypothesis.&quot; A critical confirmation of the replication mechanism that was implied by the double-helical structure followed in 1958 in the form of the <!--del_lnk--> Meselson-Stahl experiment. Work by Crick and coworkers showed that the genetic code was based on non-overlapping triplets of bases, called codons, and <!--del_lnk--> Har Gobind Khorana and others deciphered the <a href="../../wp/g/Genetic_code.htm" title="Genetic code">genetic code</a> not long afterward. These findings represent the birth of <!--del_lnk--> molecular biology.<p><a href="../../wp/j/James_D._Watson.htm" title="James D. Watson">Watson</a>, <a href="../../wp/f/Francis_Crick.htm" title="Francis Crick">Crick</a>, and <!--del_lnk--> Wilkins were awarded the 1962 <!--del_lnk--> Nobel Prize for Physiology or Medicine for discovering the molecular structure of DNA, by which time <!--del_lnk--> Franklin had died from cancer at 37. Nobel prizes are not awarded posthumously; had she lived, the difficult decision over whom to jointly award the prize would have been complicated as the prize can only be shared between a maximum of three; but because their work could be considered to be chemistry, it is conceivable that <!--del_lnk--> Wilkins and <!--del_lnk--> Franklin could have been awarded the <!--del_lnk--> Nobel Prize for Chemistry instead; see Graeme Hunter&#39;s biography of Sir Lawrence Bragg for more information on how scientists were nominated for Nobel Prizes.<p><a id="Forensics" name="Forensics"></a><h3> <span class="mw-headline">Forensics</span></h3>
      <dl>
       <dd>
      </dl>
      <p><!--del_lnk--> Forensic scientists can use DNA located in <!--del_lnk--> blood, <!--del_lnk--> semen, <!--del_lnk--> skin, <!--del_lnk--> saliva or <!--del_lnk--> hair left at the scene of a crime to identify a possible suspect, a process called <!--del_lnk--> genetic fingerprinting or DNA profiling. In DNA profiling the relative lengths of sections of repetitive DNA, such as <!--del_lnk--> short tandem repeats and <!--del_lnk--> minisatellites, are compared. DNA profiling was developed in 1984 by British geneticist Sir <!--del_lnk--> Alec Jeffreys of the <!--del_lnk--> University of Leicester, and was first used to convict Colin Pitchfork in 1988 in the <!--del_lnk--> Enderby murders case in <!--del_lnk--> Leicestershire, <a href="../../wp/u/United_Kingdom.htm" title="United Kingdom">United Kingdom</a>. Many jurisdictions require convicts of certain types of crimes to provide a sample of DNA for inclusion in a computerized database. This has helped investigators solve old cases where the perpetrator was unknown and only a DNA sample was obtained from the scene (particularly in <!--del_lnk--> rape cases between strangers). This method is one of the most reliable techniques for identifying a criminal, but is not always perfect, for example if no DNA can be retrieved, or if the scene is contaminated with the DNA of several possible suspects.<p><a id="DNA_and_computation" name="DNA_and_computation"></a><h3> <span class="mw-headline">DNA and computation</span></h3>
      <dl>
       <dd>
      </dl>
      <p>DNA plays an important role in <a href="../../wp/c/Computer_science.htm" title="Computer science">computer science</a>, <a href="../../wp/b/Bioinformatics.htm" title="Bioinformatics">bioinformatics and computational biology</a>, both as a motivating research problem and as a method of computation in itself. A <!--del_lnk--> Sequence profiling tool like <!--del_lnk--> Sequerome assists researchers working on sequence data by linking the entire <a href="../../wp/s/Sequence_alignment.htm" title="Sequence alignment">Sequence alignment</a> report (<!--del_lnk--> BLAST) to many third party servers/sites that provide highly specific services in sequence manipulations such as <!--del_lnk--> restriction enzyme maps, <!--del_lnk--> open reading frame analyses for <!--del_lnk--> nucleotide sequences, and <!--del_lnk--> secondary structure prediction.<p>Research on <!--del_lnk--> string searching algorithms, which find an occurrence of a sequence of letters inside a larger sequence of letters, was motivated in part by DNA research, where it is used to find specific sequences of nucleotides in a large sequence. In other applications such as <!--del_lnk--> text editors, even simple algorithms for this problem usually suffice, but DNA sequences cause these algorithms to exhibit near-worst-case behaviour due to their small number of distinct characters.<p><!--del_lnk--> Database theory has been influenced by DNA research, which poses special problems for storing and manipulating DNA sequences. Databases specialized for DNA research are called <!--del_lnk--> genomic databases, and must address a number of unique technical challenges associated with the operations of approximate matching, sequence comparison, finding repeating patterns, and homology searching.<p>In 1994, <!--del_lnk--> Leonard Adleman of the <!--del_lnk--> University of Southern California made headlines when he discovered a way of solving the directed <!--del_lnk--> Hamiltonian path problem, an <!--del_lnk--> NP-complete problem, using tools from molecular biology, in particular DNA. The new approach, dubbed <!--del_lnk--> DNA computing, has practical advantages over traditional computers in power use, space use, and efficiency, due to its ability to highly parallelize the computation (see <!--del_lnk--> parallel computing), although there is labor worth mentioning involved in retrieving the answers. A number of other problems, including simulation of various <!--del_lnk--> abstract machines, the <!--del_lnk--> boolean satisfiability problem, and the bounded version of the <!--del_lnk--> Post correspondence problem, have since been analyzed using DNA computing.<p>Due to its compactness, DNA also has a theoretical role in <a href="../../wp/c/Cryptography.htm" title="Cryptography">cryptography</a>, where in particular it allows unbreakable <!--del_lnk--> one-time pads to be efficiently constructed and used.<p><a id="History_and_anthropology" name="History_and_anthropology"></a><h3> <span class="mw-headline">History and anthropology</span></h3>
      <p>Because DNA collects mutations over time, which are then passed down from parent to offspring, it contains information about processes that have occurred in the past, becoming in time <!--del_lnk--> ancient DNA. By comparing different DNA sequences, geneticists can attempt to infer the history of organisms.<p>If DNA sequences from different <!--del_lnk--> species are compared, then the resulting family tree, or <!--del_lnk--> phylogeny can be used to study the <a href="../../wp/e/Evolution.htm" title="Evolution">evolution</a> of these species. This field of <!--del_lnk--> phylogenetics is a powerful tool in <!--del_lnk--> evolutionary biology. If DNA sequences within a species are compared, <!--del_lnk--> population geneticists can glean information on the history of particular populations. This can be used in studies ranging from <!--del_lnk--> ecological genetics to <a href="../../wp/a/Anthropology.htm" title="Anthropology">anthropology</a> (for example, DNA evidence is also being used to try to identify the <!--del_lnk--> Ten Lost Tribes of Israel).<p>DNA has also been used to look at fairly recent issues of family relationships, such as establishing some manner of familial relationship between the descendants of <!--del_lnk--> Sally Hemings and the family of <a href="../../wp/t/Thomas_Jefferson.htm" title="Thomas Jefferson">Thomas Jefferson</a>. This usage is closely related to the use of DNA in criminal investigations detailed above. Indeed, some criminal investigations have been solved when DNA from crime scenes has fortuitously matched relatives of the guilty individual.<p><a id="Global_variation_in_copy_number_in_the_human_genome" name="Global_variation_in_copy_number_in_the_human_genome"></a><h3> <span class="mw-headline">Global variation in copy number in the human genome</span></h3>
      <p>In a report published in 2006 in <a href="../../wp/n/Nature.htm" title="Nature">Nature</a>, researchers have found that the <!--del_lnk--> copy number variation (CNV) of DNA sequences in humans and other mammals, can be considerable. Deletions, insertions, duplications and complex multi-site variants, collectively termed copy number variations (CNVs) or copy number polymorphisms (CNPs), are found in all humans and other mammals examined.<div class="printfooter"> Retrieved from &quot;<!--del_lnk--> http://en.wikipedia.org/wiki/DNA&quot;</div>
      <!-- end content -->
      <div class="visualClear">
      </div>
     </div>
    </div>
   </div>
   <!-- end of the left (by default at least) column -->
   <div class="visualClear">
   </div>
   <div id="footer">
    <div class="center"> This reference article is mainly selected from the English Wikipedia with only minor checks and changes (see www.wikipedia.org for details of authors and sources) and is available under the <nobr><a href="../../wp/w/Wikipedia_Text_of_the_GNU_Free_Documentation_License.htm">GNU Free Documentation License</a></nobr>. See also our <b><a href="../../disclaimer.htm">Disclaimer</a></b>. </div>
   </div>
   <script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
  </div>
  
 </body>
</html>
