<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html dir="ltr" lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
 <head>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
  <meta content="Polymerase chain reaction,1992,1993,1995,2004,Acquittal,Adenine,Affinity chromatography,Agarose gel electrophoresis,Amino acid,Amplicon" name="keywords" />
  <link href="../../favicon.ico" rel="shortcut icon" />
  <link href="../../wp/w/Wikipedia_Text_of_the_GNU_Free_Documentation_License.htm" rel="copyright" />
  <title>Polymerase chain reaction</title>
  <style media="screen,projection" type="text/css">/*<![CDATA[*/ @import "../../css/wp-monobook-main.css"; /*]]>*/</style>
  <link href="../../css/wp-commonPrint.css" media="print" rel="stylesheet" type="text/css" />
  <!--[if lt IE 5.5000]><style type="text/css">@import "../../css/IE50Fixes.css";</style><![endif]-->
  <!--[if IE 5.5000]><style type="text/css">@import "../../css/IE55Fixes.css";</style><![endif]-->
  <!--[if IE 6]><style type="text/css">@import "../../css/IE60Fixes.css";</style><![endif]-->
  <!--[if IE 7]><style type="text/css">@import "../../css/IE70Fixes.css";</style><![endif]-->
  <!--[if lt IE 7]><script type="text/javascript" src="../../js/IEFixes.js"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->
  <script type="text/javascript">
			var skin = "monobook";
			var stylepath = "/skins-1.5";

			var wgArticlePath = "/wiki/$1";
			var wgScriptPath = "/w";
			var wgServer = "http://en.wikipedia.org";
                        
			var wgCanonicalNamespace = "";
			var wgNamespaceNumber = 0;
			var wgPageName = "Polymerase_chain_reaction";
			var wgTitle = "Polymerase chain reaction";
			var wgArticleId = 23647;
			var wgIsArticle = true;
                        
			var wgUserName = null;
			var wgUserLanguage = "en";
			var wgContentLanguage = "en";
		</script>
  <script src="../../js/wikibits.js" type="text/javascript"><!-- wikibits js --></script>
  <script src="../../js/wp.js" type="text/javascript"><!-- site js --></script>
  <style type="text/css">/*<![CDATA[*/
@import "../../css/wp-common.css";
@import "../../css/wp-monobook.css";
@import "../../css/wp.css";
/*]]>*/</style>
  <!-- Head Scripts -->
 </head>
 <body class="mediawiki ns-0 ltr page-Polymerase_chain_reaction">
  <div id="globalWrapper">
   <div id="column-content">
    <div id="content"><a id="top" name="top"></a><h1 class="firstHeading">Polymerase chain reaction</h1>
     <div id="bodyContent">
      <h3 id="siteSub"><a href="../../index.htm">2007 Schools Wikipedia Selection</a>. Related subjects: <a href="../index/subject.Science.Biology.General_Biology.htm">General Biology</a></h3>
      <!-- start content -->
      <p><b>Polymerase chain reaction</b> (<b>PCR</b>) is a <!--del_lnk--> molecular biology technique  for <!--del_lnk--> enzymatically <!--del_lnk--> replicating <a href="../../wp/d/DNA.htm" title="DNA">DNA</a> without using a living <a href="../../wp/o/Organism.htm" title="Organism">organism</a>, such as <i><!--del_lnk--> E. coli</i> or <a href="../../wp/y/Yeast.htm" title="Yeast">yeast</a>. Like amplification using living organisms, the technique allows a small amount of DNA to be amplified exponentially. As PCR is an <i><!--del_lnk--> in vitro</i> technique, it can be performed without restrictions on the form of DNA and it can be extensively modified to perform a wide array of <!--del_lnk--> genetic manipulations.<p>PCR is commonly used in medical and biological research labs for a variety of tasks, such as the detection of <!--del_lnk--> hereditary diseases, the identification of <!--del_lnk--> genetic fingerprints, the <!--del_lnk--> diagnosis of <!--del_lnk--> infectious diseases, the <!--del_lnk--> cloning of <!--del_lnk--> genes, <!--del_lnk--> paternity testing, and <!--del_lnk--> DNA computing.<p>PCR was invented by <!--del_lnk--> Kary Mullis. At the time he thought up PCR in 1983, Mullis was working in <!--del_lnk--> Emeryville, California for Cetus, one of the first biotechnology companies. There, he was charged with making short chains of DNA for other scientists. Mullis has written that he conceived of PCR while cruising along the Pacific Coast Highway 1 one night in his car. He was playing in his mind with a new way of analyzing changes (mutations) in DNA when he realized that he had instead invented a method of amplifying any DNA region. Mullis has said that before his trip was over, he was already savoring the prospects of a Nobel Prize. He shared the <!--del_lnk--> Nobel Prize in Chemistry with <!--del_lnk--> Michael Smith in 1993.<p>As Mullis has written in the <!--del_lnk--> Scientific American: &quot;Beginning with a single molecule of the genetic material DNA, the PCR can generate 100 billion similar molecules in an afternoon. The reaction is easy to execute. It requires no more than a test tube, a few simple reagents, and a source of heat.&quot;<p>
       <script type="text/javascript">
//<![CDATA[
 if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } 
//]]>
</script><a id="PCR_in_practice" name="PCR_in_practice"></a><h2> <span class="mw-headline">PCR in practice</span></h2>
      <div class="thumb tright">
       <div style="width:122px;"><!--del_lnk--> Image:Pcr mjhgjgjhgjhgjuachine.jpg<div class="thumbcaption"><b>Figure 1</b>: A thermal cycler for PCR</div>
       </div>
      </div>
      <p>PCR is used to amplify specific regions of a DNA strand. This can be a single gene, just a part of a gene, or non-coding sequence. PCR typically amplifies only short DNA fragments, usually up to 10 <!--del_lnk--> kilo base pairs (kb). Certain methods can copy fragments up to 47 kb in size, which is still much less than the chromosomal DNA of a <a href="../../wp/e/Eukaryote.htm" title="Eukaryote">eukaryotic cell</a> - for example, a human cell contains about three billion base pairs.<p>PCR, as currently practiced, requires several basic components. These components are:<ul>
       <li><i>DNA template,</i> which contains the region of the DNA fragment to be amplified<li>Two <i><!--del_lnk--> primers,</i> which determine the beginning and end of the region to be amplified (see following section on primers)<li><i><!--del_lnk--> Taq polymerase</i> (or another durable polymerase), a <!--del_lnk--> DNA polymerase, which copies the region to be amplified<li><i><!--del_lnk--> Deoxynucleotide triphosphates,</i> (dNTPs) from which the DNA polymerase builds the new DNA<li><i><!--del_lnk--> Buffer,</i> which provides a suitable chemical environment for the DNA Polymerase</ul>
      <p>The PCR process is carried out in a <!--del_lnk--> thermal cycler. This is a machine that heats and cools the reaction tubes within it to the precise temperature required for each step of the reaction. To prevent evaporation of the reaction mixture (typically volumes between 15-100&micro;l per tube), a heated lid is placed on top of the reaction tubes, or a layer of oil is put on the surface of the reaction mixture. These machines cost more than $2,500 <!--del_lnk--> USD, as of <!--del_lnk--> 2004.<p><a id="Primers" name="Primers"></a><h3> <span class="mw-headline">Primers</span></h3>
      <p>The DNA fragment to be amplified is determined by selecting primers. Primers are short, artificial DNA strands &mdash; often not more than 50 and usually only 18 to 25 base pairs long &mdash; that are complementary to the beginning or the end of the DNA fragment to be amplified. They <!--del_lnk--> anneal by adhering to the DNA template at these starting and ending points, where the DNA polymerase binds and begins the synthesis of the new DNA strand.<p>The choice of the length of the primers and their <!--del_lnk--> melting temperature (T<sub>m</sub>) depends on a number of considerations. The melting temperature of a primer -- not to be confused with the melting temperature of the template <a href="../../wp/d/DNA.htm" title="DNA">DNA</a> -- is defined as the temperature at which half of the primer binding sites are occupied. Primers that are too short would anneal at several positions on a long <a href="../../wp/d/DNA.htm" title="DNA">DNA</a> template, which would result in non-specific copies. On the other hand, the length of a primer is limited by the maximum temperature allowed to be applied in order to melt it, as melting temperature increases with the length of the primer. Melting temperatures that are too high, i.e., above 80<!--del_lnk--> &deg;C, can cause problems since the DNA polymerase is less active at such temperatures. The optimum length of a primer is generally from 15 to 40 <!--del_lnk--> nucleotides with a melting temperature between 55&deg;C and 65&deg;C.<p>Sometimes <i>degenerate primers</i> are used. These are actually mixtures of similar, but not identical, primers. They may be convenient if the same <!--del_lnk--> gene is to be amplified from different <a href="../../wp/o/Organism.htm" title="Organism">organisms</a>, as the genes themselves are probably similar but not identical. The other use for degenerate primers is when primer design is based on <!--del_lnk--> protein sequence. As several different <a href="../../wp/g/Genetic_code.htm" title="Codon">codons</a> can code for one <!--del_lnk--> amino acid, it is often difficult to deduce which codon is used in a particular case. Therefore primer sequence corresponding to the <!--del_lnk--> amino acid <!--del_lnk--> isoleucine might be &quot;ATH&quot;, where A stands for <!--del_lnk--> adenine, T for <!--del_lnk--> thymine, and H for <!--del_lnk--> adenine, <!--del_lnk--> thymine, or <!--del_lnk--> cytosine. (See <a href="../../wp/g/Genetic_code.htm" title="Genetic code">genetic code</a> for further details about <a href="../../wp/g/Genetic_code.htm" title="Codon">codons</a>.) Use of degenerate primers can greatly reduce the specificity of the PCR amplification. This problem can be partly solved by using <!--del_lnk--> touchdown PCR.<p>The above mentioned considerations make primer design a very exacting process, upon which product yield depends:<ul>
       <li><!--del_lnk--> GC-content should be between 40-60%.<li>Calculated T<sub>m</sub> for both primers used in reaction should not differ &gt;5&deg;C, and T<sub>m</sub> of the amplification product should not differ from primers by &gt;10&deg;C.<li>Annealing temperature usually is 5&deg;C below the calculated lower T<sub>m</sub>. However, it should be chosen empirically for individual conditions.<li>Inner self-complementary hairpins of &gt;4 and of <!--del_lnk--> dimers &gt;8 should be avoided.<li>Primer 3&#39; terminus design is critical to PCR success since the primer extends from the 3&#39; end. The 3&#39; end should not be complementary over greater than 3-4 bases to any region of the other primer (or even the same primer) used in the reaction and must provide correct base matching to the template.</ul>
      <p>There are <!--del_lnk--> computer programs to help design primers (see External links).<p><a id="Procedure" name="Procedure"></a><h3> <span class="mw-headline">Procedure</span></h3>
      <p>The PCR process usually consists of a series of twenty to thirty-five cycles. Each cycle consists of three steps (Fig. 2).<ol>
       <li>The double-stranded DNA has to be heated to 94-96&deg;C (or 98&deg;C if extremely thermostable polymerases are used) in order to separate the strands. This step is called <i>denaturing</i>; it breaks apart the hydrogen bonds that connect the two DNA strands. Prior to the first cycle, the DNA is often denatured for an extended time to ensure that both the template DNA and the primers have completely separated and are now single-strand only. Time: usually 1-2 minutes, but up to 5 minutes. Also certain polymerases are activated at this step (see <!--del_lnk--> hot-start PCR).<li>After separating the DNA strands, the temperature is lowered so the primers can attach themselves to the single DNA strands. This step is called <i><!--del_lnk--> annealing</i>. The temperature of this stage depends on the primers and is usually 5&deg;C below their melting temperature (45-60&deg;C). A wrong temperature during the annealing step can result in primers not binding to the template DNA at all, or binding at random. Time: 1-2 minutes.<li>Finally, the DNA polymerase has to copy the DNA strands. It starts at the annealed primer and works its way along the DNA strand. This step is called <i>elongation</i>. The elongation temperature depends on the DNA polymerase. <!--del_lnk--> Taq polymerase elongates optimally at a temperature of 72 Celsius. The time for this step depends both on the DNA polymerase itself and on the length of the DNA fragment to be amplified. As a rule-of-thumb, this step takes 1 minute per thousand base pairs. A <i>final elongation</i> step is frequently used after the last cycle to ensure that any remaining single stranded DNA is completely copied. This differs from all other elongation steps, only in that it is longer, typically 10-15 minutes. This last step is highly recommendable if the PCR product is to be ligated into a <!--del_lnk--> T vector using <!--del_lnk--> TA-cloning.</ol>
      <div class="thumb tnone">
       <div style="width:554px;"><a class="internal" href="../../images/210/21000.png.htm" title="Figure 2: Schematic drawing of the PCR cycle. (1) Denaturing at 94-96&deg;C. (2) Annealing at (eg) 68&deg;C. (3) Elongation at 72&deg;C (P=Polymerase). (4) The first cycle is complete. The two resulting DNA strands make up the template DNA for the next cycle, thus doubling the amount of DNA duplicated for each new cycle (a total of three cycles is shown above)."><img alt="Figure 2: Schematic drawing of the PCR cycle. (1) Denaturing at 94-96&deg;C. (2) Annealing at (eg) 68&deg;C. (3) Elongation at 72&deg;C (P=Polymerase). (4) The first cycle is complete. The two resulting DNA strands make up the template DNA for the next cycle, thus doubling the amount of DNA duplicated for each new cycle (a total of three cycles is shown above)." height="555" longdesc="/wiki/Image:Pcr.png" src="../../images/210/21000.png" width="552" /></a><div class="thumbcaption"><b>Figure 2</b>: Schematic drawing of the PCR cycle. (1) Denaturing at 94-96&deg;C. (2) Annealing at (eg) 68&deg;C. (3) Elongation at 72&deg;C (P=Polymerase). (4) The first cycle is complete. The two resulting DNA strands make up the template DNA for the next cycle, thus doubling the amount of DNA duplicated for each new cycle (a total of three cycles is shown above).</div>
       </div>
      </div>
      <p><a id="Example" name="Example"></a><h3> <span class="mw-headline">Example</span></h3>
      <p>The times and temperatures given in this example are taken from a PCR program that was successfully used on a 250 bp fragment of the C-terminus of the <i><!--del_lnk--> insulin-like growth factor</i> (<i>IGF</i>).<div class="thumb tright">
       <div style="width:302px;"><a class="internal" href="../../images/210/21001.jpg.htm" title="Gel electrophoresis image of a standard PCR.  Two sets of specific primers were used to amplify one gene from three seperate tissues.  As the gel shows, Tissue #1 lacks that gene, whereas Tissue #2 and #3 possess that gene."><img alt="Gel electrophoresis image of a standard PCR.  Two sets of specific primers were used to amplify one gene from three seperate tissues.  As the gel shows, Tissue #1 lacks that gene, whereas Tissue #2 and #3 possess that gene." height="283" longdesc="/wiki/Image:Roland_Gel.JPG" src="../../images/210/21001.jpg" width="300" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/210/21001.jpg.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div><!--del_lnk--> Gel electrophoresis image of a standard PCR. Two sets of specific <!--del_lnk--> primers were used to amplify one <!--del_lnk--> gene from three seperate tissues. As the gel shows, Tissue #1 lacks that gene, whereas Tissue #2 and #3 possess that gene.</div>
       </div>
      </div>
      <p>The reaction mixture consists of<ul>
       <li>1.0 &micro;l DNA template (100 ng/&micro;l)<li>2.5 &micro;l of primer, 1.25 &micro;l per primer (100 ng/&micro;l)<li>1.0 &micro;l Pfu-Polymerase<li>1.0 &micro;l nucleotides<li>5.0 &micro;l buffer solution<li>89.5 &micro;l water</ul>
      <p>A 200 &micro;l reaction tube containing the 100 &micro;l mixture is inserted into the thermocycler.<p>The PCR process consists of the following steps:<ol>
       <li>Initialization. The mixture is heated at 96&deg;C for 5 minutes to ensure that the DNA strands as well as the primers have melted. The DNA Polymerase can be present at initialization, or it can be added after this step.<li>Melting, where it is heated at 96&deg;C for 30 seconds. For each cycle, this is usually enough time for the DNA to denature.<li>Annealing by heating at 68&deg;C for 30 seconds:The primers are jiggling around, caused by the Brownian motion. Short bondings are constantly formed and broken between the single stranded primer and the single stranded template. The more stable bonds last a little bit longer (primers that fit exactly) and on that little piece of double stranded DNA (template and primer), the polymerase can attach and starts copying the template. Once there are a few bases built in, the Tm of the double-stranded region between the template and the primer is greater than the annealing or extension temperature.<li>Elongation by heating 72&deg;C for 45 seconds:This is the ideal working temperature for the polymerase. The primers, having been extended for a few bases, already have a stronger hydrogen bond to the template than the forces breaking these attractions. Primers that are on positions with no exact match, melt away from the template (because of the higher temperature) and are not extended.</ol>
      <p>The bases (complementary to the template) are coupled to the primer on the 3&#39; side (the polymerase adds dNTP&#39;s from 5&#39; to 3&#39;, reading the template from 3&#39; to 5&#39; side, bases are added complementary to the template)<ol>
       <li>Steps 2-4 are repeated 25 times, but with good primers and fresh polymerase, 15 to 20 cycles is sufficient.<li>Mixture is held at 7&deg;C. This is useful if one starts the PCR in the evening just before leaving the lab, so it can run overnight. The DNA will not be damaged at 7&deg;C after just one night.</ol>
      <p>The PCR product can be identified by its size using <!--del_lnk--> agarose gel electrophoresis. Agarose gel electrophoresis is a procedure that consists of injecting DNA into agarose gel and then applying an electric current to the gel. As a result, the smaller DNA strands move faster than the larger strands through the gel toward the positive current. The size of the PCR product can be determined by comparing it with a <i>DNA ladder</i>, which contains DNA fragments of known size, also within the gel (Fig. 3).<p><a id="PCR_optimization" name="PCR_optimization"></a><h3> <span class="mw-headline">PCR optimization</span></h3>
      <p>Since PCR is very sensitive, adequate measures to avoid contamination from other DNA present in the lab environment (<a href="../../wp/b/Bacteria.htm" title="Bacteria">bacteria</a>, viruses, lab staff&#39;s skin etc.) should be taken. Thus DNA sample preparation, reaction mixture assemblage and the PCR process, in addition to the subsequent reaction product analysis, should be performed in separate areas. For the preparation of reaction mixture, a <!--del_lnk--> laminar flow cabinet with UV lamp is recommended. Fresh gloves should be used for each PCR step as well as displacement pipettes with aerosol filters. The reagents for PCR should be prepared separately and used solely for this purpose. Aliquots should be stored separately from other DNA samples. A control reaction (inner control), omitting template DNA, should always be performed, to confirm the absence of contamination or primer multimer formation.<p><a id="Difficulties_with_polymerase_chain_reaction" name="Difficulties_with_polymerase_chain_reaction"></a><h3> <span class="mw-headline">Difficulties with polymerase chain reaction</span></h3>
      <p>Polymerase chain reaction is not perfect, and errors and mistakes can occur. These are some common errors and problems that may occur.<p><a id="Polymerase_errors" name="Polymerase_errors"></a><h4> <span class="mw-headline">Polymerase errors</span></h4>
      <p><!--del_lnk--> Taq polymerase lacks a 3&#39; to 5&#39; exonuclease activity. This makes it impossible for it to check the base it has inserted and remove it if it is incorrect, a process common in higher organisms. This in turn results in a high error rate of approximately 1 in 10,000 bases, which, if an error occurs early, can alter large proportions of the final product.<p>Other polymerases are available for accuracy in vital uses such as amplification for sequencing. Examples of polymerases with 3&#39;to 5&#39; exonuclease activity include: KOD DNA polymerase, a recombinant form of <i>Thermococcus kodakaraensis</i> KOD1; Vent, which is extracted from <i>Thermococcus litoralis</i>; <!--del_lnk--> Pfu DNA polymerase, which is extracted from <i><!--del_lnk--> Pyrococcus furiosus</i>; and Pwo, which is extracted from <i>Pyrococcus woesii</i>.<p><a id="Size_limitations" name="Size_limitations"></a><h4> <span class="mw-headline">Size limitations</span></h4>
      <p>PCR works readily with <a href="../../wp/d/DNA.htm" title="DNA">DNA</a> of lengths two to three thousand basepairs, but above this length the polymerase tends to fall off, and the typical heating cycle does not leave enough time for polymerisation to complete. It is possible to amplify larger pieces of up to 50,000 base pairs with a slower heating cycle and special polymerases. These special polymerases are often polymerases <!--del_lnk--> fused to a DNA-binding protein, making them literally &quot;stick&quot; to the DNA longer.<p><a id="Non_specific_priming" name="Non_specific_priming"></a><h4> <span class="mw-headline">Non specific priming</span></h4>
      <p>The non specific binding of primers is always a possibility due to sequence duplications, non-specific binding and partial primer binding, leaving the 5&#39; end unattached. This is increased by the use of degenerate sequences or bases in the primer. Manipulation of <!--del_lnk--> annealing temperature and <a href="../../wp/m/Magnesium.htm" title="Magnesium">magnesium</a> ion (which stabilise <a href="../../wp/d/DNA.htm" title="DNA">DNA</a> and <!--del_lnk--> RNA interactions) concentrations can increase specificity. Non-specific priming can be prevented during the low temperatures of reaction preparation by use of &quot;hot-start&quot; polymerase enzymes where the active site is blocked by an antibody or chemical that only dislodges once the reaction is heated to 95&#x2DA;C during the <!--del_lnk--> denaturation step of the first cycle.<p>Other methods to increase specificity include <!--del_lnk--> Nested PCR and <!--del_lnk--> Touchdown PCR.<p><a id="Practical_modifications_to_the_PCR_technique" name="Practical_modifications_to_the_PCR_technique"></a><h3> <span class="mw-headline">Practical modifications to the PCR technique</span></h3>
      <ul>
       <li><b>Nested PCR</b> - <!--del_lnk--> Nested PCR is intended to reduce the contaminations in products due to the amplification of unexpected primer binding sites. Two sets of primers are used in two successive PCR runs, the second set intended to amplify a secondary target within the first run product. This is very successful, but requires more detailed knowledge of the sequences involved.<li><b>Intersequence specific (ISSR) PCR</b><li><b>Ligation-mediated PCR</b></ul>
      <ul>
       <li><b>Inverse PCR</b> - <!--del_lnk--> Inverse PCR is a method used to allow PCR when only one internal sequence is known. This is especially useful in identifying <!--del_lnk--> flanking sequences to various <!--del_lnk--> genomic inserts. This involves a series of <!--del_lnk--> digestions and <!--del_lnk--> self ligation before cutting by an <!--del_lnk--> endonuclease, resulting in known sequences at either end of the unknown sequence.</ul>
      <ul>
       <li><b>RT-PCR</b> - <!--del_lnk--> RT-PCR (<b>R</b>everse <b>T</b>ranscription PCR) is the method used to amplify, isolate or identify a known sequence from a <a href="../../wp/c/Cell_%2528biology%2529.htm" title="Cell (biology)">cell</a> or <!--del_lnk--> tissues <!--del_lnk--> RNA library. Essentially normal PCR preceded by <!--del_lnk--> transcription by <!--del_lnk--> Reverse transcriptase (to convert the <!--del_lnk--> RNA to <!--del_lnk--> cDNA) this is widely used in <!--del_lnk--> expression mapping, determining when and where certain <!--del_lnk--> genes are <!--del_lnk--> expressed.</ul>
      <ul>
       <li><b>Assembly PCR</b> - Assembly PCR is the completely artificial synthesis of long gene products by performing PCR on a pool of long oligonucleotides with short overlapping segments. The oligonucleotides alternate between sense and antisense directions, and the overlapping segments serve to order the PCR fragments so that they selectively produce their final product.</ul>
      <ul>
       <li><b>Asymmetric PCR</b> - Asymmetric PCR is used to preferentially amplify one strand of the original <a href="../../wp/d/DNA.htm" title="DNA">DNA</a> more than the other. It finds use in some types of <!--del_lnk--> sequencing and <!--del_lnk--> hybridization probing where having only one of the two complementary stands is ideal. PCR is carried out as usual, but with a great excess of the primers for the chosen strand. Due to the slow (<a href="../../wp/a/Arithmetic.htm" title="Arithmetic">arithmetic</a>) amplification later in the reaction after the limiting primer has been used up, extra cycles of PCR are required. A recent modification on this process, known as <b>L</b>inear-<b>A</b>fter-<b>T</b>he-<b>E</b>xponential-PCR (<!--del_lnk--> LATE-PCR), uses a limiting primer with a higher melting temperature (<!--del_lnk--> Tm) than the excess primer to maintain reaction efficiency as the limiting primer concentration decreases mid-reaction.</ul>
      <ul>
       <li><b>Quantitative PCR</b> - <!--del_lnk--> Q-PCR (<b>Q</b>uantitative PCR) is used to rapidly measure the quantity of PCR product (preferably real-time), thus is an indirect method for quantitatively measuring starting amounts of DNA, cDNA or RNA. This is commonly used for the purpose of determining whether a sequence is present or not, and if it is present the number of copies in the sample. There are 3 main methods which vary in difficulty and detail.</ul>
      <ul>
       <li><b>Quantitative real-time PCR</b> is often confusingly known as <!--del_lnk--> RT-PCR (<b>R</b>eal <b>T</b>ime PCR) and RQ-PCR. QRT-PCR or RTQ-PCR are more appropriate contractions. RT-PCR can also refer to <!--del_lnk--> reverse transcription PCR, which even more confusingly, is often used in conjunction with Q-PCR. This method uses fluorescent dyes and probes to measure the amount of amplified product in real time.</ul>
      <ul>
       <li><b>Touchdown PCR</b> - <!--del_lnk--> Touchdown PCR is a variant of PCR that reduces nonspecific primer annealing by more gradually lowering the annealing temperature between cycles. As higher temperatures give greater specificity for primer binding, primers anneal first as the temperature passes through the zone of greatest specificity.</ul>
      <ul>
       <li><b>Hot-start PCR</b> is a technique that reduces non-specific priming that occurs during the preparation of the reaction components. The technique may be performed manually by simply heating the reaction components briefly at the melting temperature before adding the polymerase. Specialized enzyme systems have been developed that inhibit the polymerase&#39;s activity at ambient temperature, either by the binding of an <!--del_lnk--> antibody or by the presence of covalently bound inhibitors that only dissociate after a high-temperature activation step.</ul>
      <ul>
       <li><b>Colony PCR</b> - Bacterial clones (<!--del_lnk--> E.coli) can be screened for the correct ligation products. Selected colonies are picked with a sterile toothpick from an agarose plate and dabbed into the master mix or sterile water. Primers (and the master mix) are added - the PCR protocol has to be started with an extended time at 95^^C.</ul>
      <ul>
       <li><b>RACE-PCR</b> - Rapid amplification of cDNA ends.</ul>
      <ul>
       <li><b>Multiplex-PCR</b> - The use of multiple, unique primer sets within a single PCR reaction to produce <!--del_lnk--> amplicons of varying sizes specific to different DNA sequences. By targeting multiple genes at once, additional information may be elicited from a single test run that otherwise would require several times the reagents and technician time to perform. Annealing temperatures for each of the primer sets must be optimized to work correctly within a single reaction and amplicon sizes should be separated by enough difference in final base pair length to form distinct bands via gel electrophoresis.</ul>
      <ul>
       <li><b>Methylation Specific PCR</b> - Methylation Specific PCR (MSP) is used to detect methylation of CpG islands in genomic DNA. DNA is first treated with sodium bisulfite, which converts unmethylated cytosine bases to uracil, which is recognized by PCR primers as thymine. Two PCR reactions are then carried out on the modified DNA, using primer sets identical except at any CpG islands within the primer sequences. At these points, one primer set recognizes DNA with cytosines to amplify methylated DNA, and one set recognizes DNA with uracil or thymine to amplify unmethylated DNA. MSP using qPCR can also be performed to obtain quantitative rather than qualitative information about methylation.</ul>
      <p><a id="Recent_developments_in_PCR_techniques" name="Recent_developments_in_PCR_techniques"></a><h3> <span class="mw-headline">Recent developments in PCR techniques</span></h3>
      <ul>
       <li>A more recent method which excludes a temperature cycle, but uses enzymes, is <!--del_lnk--> helicase-dependent amplification.<li><!--del_lnk--> TAIL-PCR, developed by Liu <i>et al.</i> in <!--del_lnk--> 1995, is the <b>thermal asymmetric interlaced</b> PCR.<li><!--del_lnk--> Meta-PCR, developed by Andrew Wallace, allows to optimize amplification and direct sequence analysis of complex genes. Details at <!--del_lnk--> National Genetic Reference Laboratory, Manchester, UK</ul>
      <p><a id="Uses_of_PCR" name="Uses_of_PCR"></a><h2> <span class="mw-headline">Uses of PCR</span></h2>
      <p>PCR can be used for a broad variety of experiments and analyses. Some examples are discussed below.<p><a id="Genetic_fingerprinting" name="Genetic_fingerprinting"></a><h3> <span class="mw-headline">Genetic fingerprinting</span></h3>
      <p><!--del_lnk--> Genetic fingerprinting is a forensic technique used to identify a person by comparing his or her DNA with a given sample. An example is <!--del_lnk--> blood from a <!--del_lnk--> crime scene being genetically compared to blood from a suspect. The sample may contain only a tiny amount of DNA (obtained from a source such as <!--del_lnk--> blood, <!--del_lnk--> semen, <!--del_lnk--> saliva, <!--del_lnk--> hair, or other organic material)). Theoretically, just a single strand is needed. First, one breaks the DNA sample into fragments; then amplifies them using PCR. The amplified fragments are then separated using gel electrophoresis. The overall layout of the DNA fragments is called a <i>DNA fingerprint</i>. Since there is a very tiny possibility that two individuals may have the same sequences (one in several million), the technique is more effective at <!--del_lnk--> acquitting a suspect than proving the suspect guilty. This small possibility was exploited by defense lawyers in the <!--del_lnk--> controversial <!--del_lnk--> O.J. Simpson case. A match however usually remains an extremely strong indicator also in the question of guilt.<p><a id="Paternity_testing" name="Paternity_testing"></a><h3> <span class="mw-headline">Paternity testing</span></h3>
      <div class="thumb tright">
       <div style="width:202px;"><a class="internal" href="../../images/210/21002.png.htm" title="Figure 4: Electrophoresis of PCR-amplified DNA fragments. (1) Father. (2) Child. (3) Mother. The child has inherited some, but not all of the fingerprint of each of its parents, giving it a new, unique fingerprint."><img alt="Figure 4: Electrophoresis of PCR-amplified DNA fragments. (1) Father. (2) Child. (3) Mother. The child has inherited some, but not all of the fingerprint of each of its parents, giving it a new, unique fingerprint." height="301" longdesc="/wiki/Image:Pcr_fingerprint.png" src="../../images/210/21002.png" width="200" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/210/21002.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div><b>Figure 4</b>: Electrophoresis of PCR-amplified DNA fragments. (1) Father. (2) Child. (3) Mother. The child has inherited some, but not all of the fingerprint of each of its parents, giving it a new, unique fingerprint.</div>
       </div>
      </div>
      <p>Although these resulting &#39;fingerprints&#39; are unique (except for identical twins), genetic relationships, for example, parent-child or siblings, can be determined from two or more genetic fingerprints, which can be used for paternity tests (Fig. 4). A variation of this technique can also be used to determine evolutionary relationships between organisms.<p><a id="Detection_of_hereditary_diseases" name="Detection_of_hereditary_diseases"></a><h3> <span class="mw-headline">Detection of hereditary diseases</span></h3>
      <p>The detection of hereditary diseases in a given genome is a long and difficult process, which can be shortened significantly by using PCR. Each gene in question can easily be amplified through PCR by using the appropriate primers and then <!--del_lnk--> sequenced to detect mutations.<p><!--del_lnk--> Viral diseases, too, can be detected using PCR through amplification of the viral DNA. This analysis is possible right after infection, which can be from several days to several months before actual symptoms occur. Such early diagnoses give physicians a significant lead in treatment.<p><a id="Cloning_genes" name="Cloning_genes"></a><h3> <span class="mw-headline">Cloning genes</span></h3>
      <p>Cloning a gene, not to be confused with <!--del_lnk--> cloning a whole organism, describes the process of isolating a gene from one organism and then inserting it into another organism (now termed a <!--del_lnk--> genetically modified organism (GMO)). PCR is often used to amplify the gene, which can then be inserted into a <!--del_lnk--> vector (a <i>vector</i> is a piece of DNA which &#39;carries&#39; the gene into the GMO) such as a <!--del_lnk--> plasmid (a circular DNA molecule) (Fig. 5). The DNA can then be transferred into an organism (the GMO) where the gene and its product can be studied more closely. Expressing a cloned gene (when a gene is <i>expressed</i> the gene product (usually <a href="../../wp/p/Protein.htm" title="Protein">protein</a> or <!--del_lnk--> RNA) is produced by the GMO) can also be a way of mass-producing useful proteins, for example medicines or the enzymes in biological washing powders. The incorporation of an affinity tag on a <!--del_lnk--> recombinant protein will generate a <!--del_lnk--> fusion protein which can be more easily purified by <!--del_lnk--> affinity chromatography.<div class="thumb tnone">
       <div style="width:641px;"><a class="internal" href="../../images/210/21003.png.htm" title="Figure 5: Cloning a gene using a plasmid.(1) Chromosomal DNA of organism A. (2) PCR. (3) Multiple copies of a single gene from organism A. (4) Insertion of the gene into a plasmid. (5) Plasmid with gene from organism A. (6) Insertion of the plasmid in organism B. (7) Multiplication or expression of the gene, originally from organism A, occurring in organism B."><img alt="Figure 5: Cloning a gene using a plasmid.(1) Chromosomal DNA of organism A. (2) PCR. (3) Multiple copies of a single gene from organism A. (4) Insertion of the gene into a plasmid. (5) Plasmid with gene from organism A. (6) Insertion of the plasmid in organism B. (7) Multiplication or expression of the gene, originally from organism A, occurring in organism B." height="167" longdesc="/wiki/Image:Pcr_clone.png" src="../../images/210/21003.png" width="639" /></a><div class="thumbcaption"><b>Figure 5</b>: Cloning a gene using a plasmid.<br /> (1) Chromosomal DNA of organism A. (2) PCR. (3) Multiple copies of a single gene from organism A. (4) Insertion of the gene into a plasmid. (5) Plasmid with gene from organism A. (6) Insertion of the plasmid in organism B. (7) Multiplication or expression of the gene, originally from organism A, occurring in organism B.</div>
       </div>
      </div>
      <p><a id="Mutagenesis" name="Mutagenesis"></a><h3> <span class="mw-headline">Mutagenesis</span></h3>
      <p><i><!--del_lnk--> Mutagenesis</i> is a way of making changes to the sequence of nucleotides in the DNA. There are situations in which one is interested in <i>mutated</i> (changed) copies of a given DNA strand, for example, when trying to assess the function of a gene or in <i>in-vitro</i> protein <a href="../../wp/e/Evolution.htm" title="Evolution">evolution</a> (also known as <!--del_lnk--> Directed evolution). Mutations can be introduced into copied DNA sequences in two fundamentally different ways in the PCR process. <i><!--del_lnk--> Site-directed mutagenesis</i> allows the experimenter to introduce a mutation at a specific location on the DNA strand. Usually, the desired mutation is incorporated in the primers used for the PCR program. <i>Random mutagenesis,</i> on the other hand, is based on the use of error-prone polymerases in the PCR process. In the case of random mutagenesis, the location and nature of the mutations cannot be controlled. One application of random mutagenesis is to analyze structure-function relationships of a protein. By randomly altering a DNA sequence, one can compare the resulting protein with the original and determine the function of each part of the protein.<p><a id="Analysis_of_ancient_DNA" name="Analysis_of_ancient_DNA"></a><h3> <span class="mw-headline">Analysis of ancient DNA</span></h3>
      <p>Using PCR, it becomes possible to analyze DNA that is thousands of years old. PCR techniques have been successfully used on animals, such as a forty-thousand-year-old <a href="../../wp/m/Mammoth.htm" title="Mammoth">mammoth</a>, and also on human DNA, in applications ranging from the analysis of Egyptian <a href="../../wp/m/Mummy.htm" title="Mummy">mummies</a> to the identification of a <a href="../../wp/r/Russia.htm" title="Russia">Russian</a> <!--del_lnk--> Tsar.<p><a id="Genotyping_of_specific_mutations" name="Genotyping_of_specific_mutations"></a><h3> <span class="mw-headline">Genotyping of specific mutations</span></h3>
      <p>Through the use of allele-specific PCR, one can easily determine which allele of a mutation or polymorphism an individual has. Here, one of the two primers is common, and would anneal a short distance away from the mutation, while the other anneals right on the variation. The 3&#39; end of the allele-specific primer is modified, to only anneal if it matches one of the alleles. If the mutation of interest is a T or C <!--del_lnk--> single nucleotide polymorphism (T/C SNP), one would use two reactions, one containing a primer ending in T, and the other ending in C. The common primer would be the same. Following PCR, these two sets of reactions would be run out on an agarose gel, and the band pattern will tell you if the individual is homozygous T, homozygous C, or heterozygous T/C. This methodology has several applications, such as amplifying certain haplotypes (when certain alleles at 2 or more SNPs occur together on the same chromosome <!--del_lnk--> Linkage Disequilibrium) or detection of recombinant chromosomes and the study of meiotic recombination.<p><a id="Comparison_of_gene_expression" name="Comparison_of_gene_expression"></a><h3> <span class="mw-headline">Comparison of gene expression</span></h3>
      <p>Researchers have used traditional PCR as a way to estimate changes in the amount of a <!--del_lnk--> gene&#39;s expression. <!--del_lnk--> Ribonucleic acid (RNA) is the molecule into which DNA is transcribed prior to making a protein, and those strands of RNA that hold the instructions for protein sequence are known as messenger RNA (mRNA). Once RNA is isolated it can be <!--del_lnk--> reverse transcribed back into DNA (complementary DNA to be precise, known as cDNA), at which point traditional PCR can be applied to amplify the gene, this methodology is called <!--del_lnk--> RT-PCR. In most cases if there is more starting material (mRNA) of a gene then during PCR more copies of the gene will be generated. When the products of the PCR process are run on an agarose gel (see Figure 3 above) a band, corresponding to a gene, will appear larger on the gel (note that the band remains in the same location relative to the ladder, it will just appear fatter or brighter). By running samples of amplified cDNA from differently treated organisms one can get a general idea of which sample expressed more of the gene of interest. A quantative RT-PCR method has been developed, it is called <!--del_lnk--> Real-time PCR .<p><a id="History" name="History"></a><h2> <span class="mw-headline">History</span></h2>
      <p>Polymerase chain reaction was invented by <!--del_lnk--> Kary Mullis. He was awarded the <!--del_lnk--> Nobel Prize in Chemistry in <!--del_lnk--> 1993 for his invention, only seven years after he and his colleagues at Cetus first reduced his proposal to practice. The idea was to develop a process by which DNA could be artificially multiplied through repeated cycles of duplication driven by an <!--del_lnk--> enzyme called <!--del_lnk--> DNA polymerase.<p>DNA polymerase occurs naturally in living organisms. In cells it functions to duplicate DNA when cells divide in <!--del_lnk--> mitosis and <!--del_lnk--> meiosis. Polymerase works by binding to a single DNA strand and creating the complementary strand. In the first of many original processes, the <!--del_lnk--> enzyme was used <!--del_lnk--> in vitro (in a controlled environment outside an organism). The double-stranded DNA was separated into two single strands by heating it to 94&deg;C (201&deg;F). At this temperature, however, the DNA polymerase used at the time were destroyed, so the enzyme had to be replenished after the heating stage of each cycle. The original procedure was very inefficient, since it required a great deal of time, large amounts of DNA polymerase, and continual attention throughout the process.<p>Later, this original PCR process was greatly improved by the use of DNA polymerase taken from <!--del_lnk--> thermophilic bacteria grown in <!--del_lnk--> geysers at a temperature of over 110&deg;C (230&deg;F). The DNA polymerase taken from these organisms is stable at high temperatures and, when used in PCR, does not break down when the mixture was heated to separate the DNA strands. Since there was no longer a need to add new DNA polymerase for each cycle, the process of copying a given DNA strand could be simplified and automated.<p>One of the first thermostable DNA polymerases was obtained from <i><!--del_lnk--> Thermus aquaticus</i> and was called &quot;Taq.&quot; Taq polymerase is widely used in current PCR practice. A disadvantage of Taq is that it sometimes makes mistakes when copying DNA, leading to <!--del_lnk--> mutations (errors) in the DNA sequence, since it lacks 3&#39;&rarr;5&#39; proofreading exonuclease activity. Polymerases such as <i>Pwo</i> or <i>Pfu,</i> obtained from <i><!--del_lnk--> Archaea,</i> have <i>proofreading mechanisms</i> (mechanisms that check for errors) and can significantly reduce the number of mutations that occur in the copied DNA sequence. However these enzymes polymerise DNA at a much slower rate than Taq. Combinations of both <i>Taq</i> and <i>Pfu</i> are available nowadays that provide both high processivity (fast polymerisation) and high fidelity (accurate duplication of DNA).<p>PCR has been performed on DNA larger than 10 kilobases, but the average PCR is only several hundred to a few thousand bases of DNA. The problem with long PCR is that there is a balance between accuracy and processivity of the enzyme. Usually, the longer the fragment, the greater the probability of errors.<p><a id="Patent_wars" name="Patent_wars"></a><h2> <span class="mw-headline">Patent wars</span></h2>
      <p>The PCR technique was patented by <!--del_lnk--> Cetus Corporation, where Mullis worked when he invented the technique in 1983. The Taq polymerase enzyme is also covered by patents. There have been several high-profile lawsuits related to the technique, including an unsuccessful lawsuit brought by <!--del_lnk--> DuPont. The pharmaceutical company <!--del_lnk--> Hoffmann-La Roche purchased the rights to the patents in <!--del_lnk--> 1992 and currently holds those that are still protected.<p>A related patent battle over the Taq polymerase enzyme is still ongoing in several jurisdictions around the world between Roche and <!--del_lnk--> Promega. Interestingly, it seems possible that the legal arguments will extend beyond the life of the original PCR and Taq polymerase patents, which expire in 2006.<div class="printfooter"> Retrieved from &quot;<!--del_lnk--> http://en.wikipedia.org/wiki/Polymerase_chain_reaction&quot;</div>
      <!-- end content -->
      <div class="visualClear">
      </div>
     </div>
    </div>
   </div>
   <!-- end of the left (by default at least) column -->
   <div class="visualClear">
   </div>
   <div id="footer">
    <div class="center"> This reference article is mainly selected from the English Wikipedia with only minor checks and changes (see www.wikipedia.org for details of authors and sources) and is available under the <nobr><a href="../../wp/w/Wikipedia_Text_of_the_GNU_Free_Documentation_License.htm">GNU Free Documentation License</a></nobr>. See also our <b><a href="../../disclaimer.htm">Disclaimer</a></b>. </div>
   </div>
   <script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
  </div>
  
 </body>
</html>
