<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html dir="ltr" lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
 <head>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
  <meta content="Protein,1838,ATP,Acid,Actin,Active site,Adaptive immune system,Adenylate kinase,Albumins,Alfred Mirsky,Alpha carbon" name="keywords" />
  <link href="../../favicon.ico" rel="shortcut icon" />
  <link href="../../wp/w/Wikipedia_Text_of_the_GNU_Free_Documentation_License.htm" rel="copyright" />
  <title>Protein</title>
  <style media="screen,projection" type="text/css">/*<![CDATA[*/ @import "../../css/wp-monobook-main.css"; /*]]>*/</style>
  <link href="../../css/wp-commonPrint.css" media="print" rel="stylesheet" type="text/css" />
  <!--[if lt IE 5.5000]><style type="text/css">@import "../../css/IE50Fixes.css";</style><![endif]-->
  <!--[if IE 5.5000]><style type="text/css">@import "../../css/IE55Fixes.css";</style><![endif]-->
  <!--[if IE 6]><style type="text/css">@import "../../css/IE60Fixes.css";</style><![endif]-->
  <!--[if IE 7]><style type="text/css">@import "../../css/IE70Fixes.css";</style><![endif]-->
  <!--[if lt IE 7]><script type="text/javascript" src="../../js/IEFixes.js"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->
  <script type="text/javascript">
			var skin = "monobook";
			var stylepath = "/skins-1.5";

			var wgArticlePath = "/wiki/$1";
			var wgScriptPath = "/w";
			var wgServer = "http://en.wikipedia.org";
                        
			var wgCanonicalNamespace = "";
			var wgNamespaceNumber = 0;
			var wgPageName = "Protein";
			var wgTitle = "Protein";
			var wgArticleId = 23634;
			var wgIsArticle = true;
                        
			var wgUserName = null;
			var wgUserLanguage = "en";
			var wgContentLanguage = "en";
		</script>
  <script src="../../js/wikibits.js" type="text/javascript"><!-- wikibits js --></script>
  <script src="../../js/wp.js" type="text/javascript"><!-- site js --></script>
  <style type="text/css">/*<![CDATA[*/
@import "../../css/wp-common.css";
@import "../../css/wp-monobook.css";
@import "../../css/wp.css";
/*]]>*/</style>
  <!-- Head Scripts -->
 </head>
 <body class="mediawiki ns-0 ltr page-Protein">
  <div id="globalWrapper">
   <div id="column-content">
    <div id="content"><a id="top" name="top"></a><h1 class="firstHeading">Protein</h1>
     <div id="bodyContent">
      <h3 id="siteSub"><a href="../../index.htm">2007 Schools Wikipedia Selection</a>. Related subjects: <a href="../index/subject.Science.Biology.General_Biology.htm">General Biology</a></h3>
      <!-- start content -->
      <div class="thumb tright">
       <div style="width:202px;"><a class="internal" href="../../images/210/21090.png.htm" title="A representation of the 3D structure of myoglobin, showing coloured alpha helices. This protein was the first to have its structure solved by X-ray crystallography."><img alt="A representation of the 3D structure of myoglobin, showing coloured alpha helices. This protein was the first to have its structure solved by X-ray crystallography." height="204" longdesc="/wiki/Image:Myoglobin.png" src="../../images/210/21090.png" width="200" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/210/21090.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> A representation of the 3D structure of <!--del_lnk--> myoglobin, showing coloured <!--del_lnk--> alpha helices. This protein was the first to have its structure solved by <!--del_lnk--> X-ray crystallography.</div>
       </div>
      </div>
      <p><b>Proteins</b> are large <!--del_lnk--> organic compounds made of <!--del_lnk--> amino acids arranged in a linear chain and joined together between the <!--del_lnk--> carboxyl atom of one amino acid and the <!--del_lnk--> amine nitrogen of another. This bond is called a <!--del_lnk--> peptide bond. The sequence of amino acids in a protein is defined by a <!--del_lnk--> gene and encoded in the <a href="../../wp/g/Genetic_code.htm" title="Genetic code">genetic code</a>. Although this genetic code specifies 20 &quot;standard&quot; amino acids, the residues in a protein are often chemically altered in <!--del_lnk--> post-translational modification: either before the protein can function in the cell, or as part of control mechanisms. Proteins can also work together to achieve a particular function, and they often associate to form stable <!--del_lnk--> complexes.<p>Like other biological <!--del_lnk--> macromolecules such as <!--del_lnk--> polysaccharides and <!--del_lnk--> nucleic acids, proteins are essential parts of all living organisms and participate in every process within <a href="../../wp/c/Cell_%2528biology%2529.htm" title="Cell (biology)">cells</a>. Many proteins are <!--del_lnk--> enzymes that <!--del_lnk--> catalyze biochemical reactions, and are vital to <!--del_lnk--> metabolism. Other proteins have structural or mechanical functions, such as the proteins in the <!--del_lnk--> cytoskeleton, which forms a system of <!--del_lnk--> scaffolding that maintains cell shape. Proteins are also important in <!--del_lnk--> cell signaling, <!--del_lnk--> immune responses, <!--del_lnk--> cell adhesion, and the <!--del_lnk--> cell cycle. Protein is also a necessary component in our diet, since animals cannot synthesise all the amino acids and must obtain <!--del_lnk--> essential amino acids from food. Through the process of <!--del_lnk--> digestion, animals break down ingested protein into free amino acids that can be used for protein synthesis.<p>The name <i>protein</i> comes from the <!--del_lnk--> Greek <i>&pi;&rho;&#x3CE;&tau;&alpha;</i> (&quot;prota&quot;), meaning &quot;<i>of primary importance</i>&quot; and were first described and named by <!--del_lnk--> J&ouml;ns Jakob Berzelius in <!--del_lnk--> 1838. However, their central role in living organisms was not fully appreciated until 1926, when <!--del_lnk--> James B. Sumner showed that the enzyme <!--del_lnk--> urease was a protein. The first protein structures to be solved included <a href="../../wp/i/Insulin.htm" title="Insulin">insulin</a> and <!--del_lnk--> myoglobin; the first was by <!--del_lnk--> Sir Frederick Sanger who won a 1958 Nobel Prize for it, and the second by <!--del_lnk--> Max Perutz and <!--del_lnk--> Sir John Cowdery Kendrew in 1958. Both proteins&#39; three-dimensional structures were amongst the first determined by x-ray diffraction analysis; the myoglobin structure won the <!--del_lnk--> Nobel Prize in Chemistry for its discoverers.<p>
       <script type="text/javascript">
//<![CDATA[
 if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } 
//]]>
</script><a id="Biochemistry" name="Biochemistry"></a><h2> <span class="mw-headline">Biochemistry</span></h2>
      <dl>
       <dd>
      </dl>
      <div class="thumb tright">
       <div style="width:302px;"><a class="internal" href="../../images/210/21091.png.htm" title="Resonance structures of the peptide bond that links individual amino acids to form a protein polymer."><img alt="Resonance structures of the peptide bond that links individual amino acids to form a protein polymer." height="65" longdesc="/wiki/Image:Peptide_group_resonance.png" src="../../images/210/21091.png" width="300" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/210/21091.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div><!--del_lnk--> Resonance structures of the <!--del_lnk--> peptide bond that links individual amino acids to form a protein <!--del_lnk--> polymer.</div>
       </div>
      </div>
      <div class="thumb tright">
       <div style="width:302px;"><a class="internal" href="../../images/210/21092.jpg.htm" title="Section of a protein structure showing serine and alanine residues linked together by peptide bonds. Carbons are shown in white and hydrogens are omitted for clarity."><img alt="Section of a protein structure showing serine and alanine residues linked together by peptide bonds. Carbons are shown in white and hydrogens are omitted for clarity." height="211" longdesc="/wiki/Image:Peptide_bond.jpg" src="../../images/210/21092.jpg" width="300" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/210/21092.jpg.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Section of a protein structure showing serine and alanine residues linked together by peptide bonds. Carbons are shown in white and hydrogens are omitted for clarity.</div>
       </div>
      </div>
      <p>Proteins are linear polymers built from 20 different L-alpha-<!--del_lnk--> amino acids. All amino acids share common structural features including an <!--del_lnk--> alpha carbon to which an <!--del_lnk--> amino group, a <!--del_lnk--> carboxyl group, and a variable <!--del_lnk--> side chain are <!--del_lnk--> bonded. Only proline shows little difference in a fashion by containing an unusual ring to the N-end amine group, which forces the CO-NH amide sequence into a fixed conformation. The side chains of the standard amino acids, detailed in the <!--del_lnk--> list of standard amino acids, have varying chemical properties that produce proteins&#39; three-dimensional structure and are therefore critical to protein function. The amino acids in a polypeptide chain are linked by <!--del_lnk--> peptide bonds formed in a <!--del_lnk--> dehydration reaction. Once linked in the protein chain, an individual amino acid is called a <i>residue</i> and the linked series of carbon, nitrogen, and oxygen atoms are known as the <i>main chain</i> or <i>protein backbone</i>. The peptide bond has two <!--del_lnk--> resonance forms that contribute some <!--del_lnk--> double bond character and inhibit rotation around its axis, so that the alpha carbons are roughly <!--del_lnk--> coplanar. The other two <!--del_lnk--> dihedral angles in the peptide bond determine the local shape assumed by the protein backbone.<p>Due to the chemical structure of the individual amino acids, the protein chain has directionality. The end of the protein with a free carboxyl group is known as the <!--del_lnk--> C-terminus or carboxy terminus, while the end with a free amino group is known as the <!--del_lnk--> N-terminus or amino terminus.<p>There is some ambiguity between the usage of the words <i>protein</i>, <i><!--del_lnk--> polypeptide</i>, and <i><!--del_lnk--> peptide</i>. <i>Protein</i> is generally used to refer to the complete biological molecule in a stable <!--del_lnk--> conformation, while <i>peptide</i> is generally reserved for a short amino acid oligomers often lacking a stable 3-dimensional structure. However, the boundary between the two is ill-defined and usually lies near 20-30 residues. <i>Polypeptide</i> can refer to any single linear chain of amino acids, usually regardless of length, but often implies an absence of a single defined <!--del_lnk--> conformation.<p><a id="Synthesis" name="Synthesis"></a><h2> <span class="mw-headline">Synthesis</span></h2>
      <dl>
       <dd>
      </dl>
      <div class="floatleft"><span><a class="image" href="../../images/210/21093.gif.htm" title=""><img alt="" height="145" longdesc="/wiki/Image:One_gene%2C_one_enzyme.gif" src="../../images/210/21093.gif" width="125" /></a></span></div>
      <p>Proteins are assembled from amino acids using information encoded in <!--del_lnk--> genes. Each protein has its own unique amino acid sequence that is specified by the <!--del_lnk--> nucleotide sequence of the gene encoding this protein. The <a href="../../wp/g/Genetic_code.htm" title="Genetic code">genetic code</a> is a set of three-nucleotide sets called <a href="../../wp/g/Genetic_code.htm" title="Codon">codons</a> and each three-nucleotide combination stands for an amino acid, for example ATG stands for <!--del_lnk--> methionine. Because <a href="../../wp/d/DNA.htm" title="DNA">DNA</a> contains four nucleotides, the total number of possible codons is 64; hence, there is some redundancy in the genetic code and some amino acids are specified by more than one codon. Genes encoded in DNA are first <!--del_lnk--> transcribed into pre-<!--del_lnk--> messenger RNA (mRNA) by proteins such as <!--del_lnk--> RNA polymerase. Most organisms then process the pre-mRNA (also known as a <i>primary transcript</i>) using various forms of <!--del_lnk--> post-transcriptional modification to form the mature mRNA, which is then used as a template for protein synthesis by the <!--del_lnk--> ribosome. In <!--del_lnk--> prokaryotes the mRNA may either be used as soon as it is produced, or be bound by a ribosome after having moved away from the <!--del_lnk--> nucleoid. In contrast, <a href="../../wp/e/Eukaryote.htm" title="Eukaryote">eukaryotes</a> make mRNA in the <!--del_lnk--> cell nucleus and then translocate it across the <!--del_lnk--> nuclear membrane into the <!--del_lnk--> cytoplasm, where protein synthesis then takes place. The rate of protein synthesis is higher in prokaryotes than eukaryotes and can reach up to 20 amino acids per second.<p>The process of synthesizing a protein from an mRNA template is known as <!--del_lnk--> translation. The mRNA is loaded onto the ribosome and is read three nucleotides at a time by matching each codon to its <!--del_lnk--> base pairing <!--del_lnk--> anticodon located on a <!--del_lnk--> transfer RNA molecule, which carries the amino acid corresponding to the codon it recognizes. The enzyme <!--del_lnk--> aminoacyl tRNA synthetase &quot;charges&quot; the tRNA molecules with the correct amino acids. The growing polypeptide is often termed the <i>nascent chain</i>. Proteins are always biosynthesized from N-terminus to C-terminus.<p>The size of a synthesized protein can be measured by the number of amino acids it contains and by its total <!--del_lnk--> molecular mass, which is normally reported in units of <i>daltons</i> (synonymous with <!--del_lnk--> atomic mass units), or the derivative unit <!--del_lnk--> kilodalton (kDa). <a href="../../wp/y/Yeast.htm" title="Yeast">Yeast</a> proteins are on average 466 amino acids long and 53 kDa in mass. The largest known proteins are the <!--del_lnk--> titins, a component of the <!--del_lnk--> muscle <!--del_lnk--> sarcomere, with a molecular mass of almost 3,000 kDa and a total length of almost 27,000 amino acids.<p><a id="Chemical_synthesis" name="Chemical_synthesis"></a><h3> <span class="mw-headline">Chemical synthesis</span></h3>
      <p>Short proteins can also be synthesized chemically in the laboratory by a family of methods known as <!--del_lnk--> peptide synthesis, which rely on <!--del_lnk--> organic synthesis techniques such as <!--del_lnk--> chemical ligation to produce peptides in high yield. Chemical synthesis allows for the introduction of non-natural amino acids into polypeptide chains, such as attachment of <!--del_lnk--> fluorescent probes to amino acid side chains. These methods are useful in laboratory <!--del_lnk--> biochemistry and <!--del_lnk--> cell biology, though generally not for commercial applications. Chemical synthesis is inefficient for polypeptides longer than about 300 amino acids, and the synthesized proteins may not readily assume their native <!--del_lnk--> tertiary structure. Most chemical synthesis methods proceed from C-terminus to N-terminus, opposite the biological reaction.<p><a id="Structure_of_proteins" name="Structure_of_proteins"></a><h2> <span class="mw-headline">Structure of proteins</span></h2>
      <dl>
       <dd>
      </dl>
      <div class="thumb tright">
       <div style="width:502px;"><a class="internal" href="../../images/210/21094.png.htm" title="Three possible representations of the three-dimensional structure of the protein triose phosphate isomerase. Left: all-atom representation colored by atom type. Middle: &quot;cartoon&quot; representation illustrating the backbone conformation, colored by secondary structure. Right: Solvent-accessible surface representation colored by residue type (acidic residues red, basic residues blue, polar residues green, nonpolar residues white)."><img alt="Three possible representations of the three-dimensional structure of the protein triose phosphate isomerase. Left: all-atom representation colored by atom type. Middle: &quot;cartoon&quot; representation illustrating the backbone conformation, colored by secondary structure. Right: Solvent-accessible surface representation colored by residue type (acidic residues red, basic residues blue, polar residues green, nonpolar residues white)." height="200" longdesc="/wiki/Image:Proteinviews-1tim.png" src="../../images/210/21094.png" width="500" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/210/21094.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Three possible representations of the three-dimensional structure of the protein <!--del_lnk--> triose phosphate isomerase. Left: all-atom representation colored by atom type. Middle: &quot;cartoon&quot; representation illustrating the backbone conformation, colored by secondary structure. Right: Solvent-accessible surface representation colored by residue type (acidic residues red, basic residues blue, polar residues green, nonpolar residues white).</div>
       </div>
      </div>
      <p>Most proteins <!--del_lnk--> fold into unique 3-dimensional structures. The shape into which a protein naturally folds is known as its <!--del_lnk--> native state. Although many proteins can fold unassisted simply through the structural propensities of their component amino acids, others require the aid of molecular <!--del_lnk--> chaperones to efficiently fold to their native states. Biochemists often refer to four distinct aspects of a protein&#39;s structure:<ul>
       <li><i><!--del_lnk--> Primary structure</i>: the <!--del_lnk--> amino acid sequence<li><i><!--del_lnk--> Secondary structure</i>: regularly repeating local structures stabilized by <!--del_lnk--> hydrogen bonds. The most common examples are the <!--del_lnk--> alpha helix and <!--del_lnk--> beta sheet. Because secondary structures are local, many regions of different secondary structure can be present in the same protein molecule.<li><i><!--del_lnk--> Tertiary structure</i>: the overall shape of a single protein molecule; the spatial relationship of the secondary structures to one another. Tertiary structure is generally stabilized by nonlocal interactions, most commonly the formation of a <!--del_lnk--> hydrophobic core, but also through <!--del_lnk--> salt bridges, hydrogen bonds, <!--del_lnk--> disulfide bonds, and even <!--del_lnk--> post-translational modifications. The term &quot;tertiary structure&quot; is often used as synonymous with the term <i>fold</i>.<li><i><!--del_lnk--> Quaternary structure</i>: the shape or structure that results from the <!--del_lnk--> interaction of more than one protein molecule, usually called <i><!--del_lnk--> protein subunits</i> in this context, which function as part of the larger assembly or <!--del_lnk--> protein complex.</ul>
      <p>In addition to these levels of structure, proteins may shift between several related structures in performing their biological function. In the context of these functional rearrangements, these tertiary or quaternary structures are usually referred to as &quot;<!--del_lnk--> conformations,&quot; and transitions between them are called <i>conformational changes.</i> Such changes are often induced by the binding of a <!--del_lnk--> substrate molecule to an enzyme&#39;s <!--del_lnk--> active site, or the physical region of the protein that participates in chemical catalysis.<div class="thumb tright">
       <div style="width:502px;"><a class="internal" href="../../images/210/21095.jpg.htm" title="Molecular surface of several proteins showing their comparative sizes. From left to right are: Antibody (IgG), Hemoglobin, Insulin (a hormone), Adenylate kinase (an enzyme), and Glutamine synthetase (an enzyme)."><img alt="Molecular surface of several proteins showing their comparative sizes. From left to right are: Antibody (IgG), Hemoglobin, Insulin (a hormone), Adenylate kinase (an enzyme), and Glutamine synthetase (an enzyme)." height="144" longdesc="/wiki/Image:Protein_Composite.jpg" src="../../images/210/21095.jpg" width="500" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/210/21095.jpg.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Molecular surface of several proteins showing their comparative sizes. From left to right are: <!--del_lnk--> Antibody (IgG), <!--del_lnk--> Hemoglobin, <a href="../../wp/i/Insulin.htm" title="Insulin">Insulin</a> (a hormone), <!--del_lnk--> Adenylate kinase (an enzyme), and <!--del_lnk--> Glutamine synthetase (an enzyme).</div>
       </div>
      </div>
      <p>Proteins can be informally divided into three main classes, which correlate with typical tertiary structures: <!--del_lnk--> globular proteins, <!--del_lnk--> fibrous proteins, and <!--del_lnk--> membrane proteins. Almost all globular proteins are <!--del_lnk--> soluble and many are enzymes. Fibrous proteins are often structural; membrane proteins often serve as <!--del_lnk--> receptors or provide channels for polar or charged molecules to pass through the cell membrane.<p>A special case of intramolecular hydrogen bonds within proteins, poorly shielded from water attack and hence promoting their own <!--del_lnk--> dehydration, are called <!--del_lnk--> dehydrons.<p><a id="Structure_determination" name="Structure_determination"></a><h3> <span class="mw-headline">Structure determination</span></h3>
      <p>Discovering the tertiary structure of a protein, or the quaternary structure of its complexes, can provide important clues about how the protein performs its function. Common experimental methods of structure determination include <!--del_lnk--> X-ray crystallography and <!--del_lnk--> NMR spectroscopy, both of which can produce information at <a href="../../wp/a/Atom.htm" title="Atom">atomic</a> resolution. <!--del_lnk--> Cryoelectron microscopy is used to produce lower-resolution structural information about very large protein complexes, including assembled <a href="../../wp/v/Virus.htm" title="Virus">viruses</a>; a variant known as <!--del_lnk--> electron crystallography can also produce high-resolution information in some cases, especially for two-dimensional crystals of membrane proteins. Solved structures are usually deposited in the <!--del_lnk--> Protein Data Bank (PDB), a freely available resource from which structural data about thousands of proteins can be obtained in the form of <!--del_lnk--> Cartesian coordinates for each atom in the protein.<p>There are many more known gene sequences than there are solved protein structures. Further, the set of solved structures is biased toward those proteins that can be easily subjected to the experimental conditions required by one of the major structure determination methods. In particular, globular proteins are comparatively easy to <!--del_lnk--> crystallize in preparation for X-ray crystallography, which remains the oldest and most common structure determination technique. Membrane proteins, by contrast, are difficult to crystallize and are underrepresented in the PDB. <!--del_lnk--> Structural genomics initiatives have attempted to remedy these deficiencies by systematically solving representative structures of major fold classes. <!--del_lnk--> Protein structure prediction methods attempt to provide a means of generating a plausible structure for a proteins whose structures have not been experimentally determined.<p><a id="Cellular_functions" name="Cellular_functions"></a><h2> <span class="mw-headline">Cellular functions</span></h2>
      <p>Proteins are the chief actors within the cell, said to be carrying out the duties specified by the information encoded in genes. With the exception of certain types of <!--del_lnk--> RNA, most other biological molecules are relatively inert elements upon which proteins act. Proteins make up half the dry weight of an <!--del_lnk--> E. coli cell, while other macromolecules such as DNA and RNA make up only 3% and 20% respectively. The total complement of proteins expressed in a particular cell or cell type is known as its <!--del_lnk--> proteome.<div class="thumb tright">
       <div style="width:352px;"><a class="internal" href="../../images/210/21096.png.htm" title="The enzyme hexokinase is shown as a simple ball-and-stick molecular model. To scale in the top right-hand corner are its two substrates, ATP and glucose."><img alt="The enzyme hexokinase is shown as a simple ball-and-stick molecular model. To scale in the top right-hand corner are its two substrates, ATP and glucose." height="250" longdesc="/wiki/Image:Hexokinase_ball_and_stick_model%2C_with_substrates_to_scale_copy.png" src="../../images/210/21096.png" width="350" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/210/21096.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> The enzyme <!--del_lnk--> hexokinase is shown as a simple ball-and-stick molecular model. To scale in the top right-hand corner are its two substrates, <!--del_lnk--> ATP and <!--del_lnk--> glucose.</div>
       </div>
      </div>
      <p>The chief characteristic of proteins that enables them to carry out their diverse cellular functions is their ability to bind other molecules specifically and tightly. The region of the protein responsible for binding another molecule is known as the <!--del_lnk--> binding site and is often a depression or &quot;pocket&quot; on the molecular surface. This binding ability is mediated by the tertiary structure of the protein, which defines the binding site pocket, and by the chemical properties of the surrounding amino acids&#39; side chains. Protein binding can be extraordinarily tight and specific; for example, the <!--del_lnk--> ribonuclease inhibitor protein binds to human <!--del_lnk--> angiogenin with a sub-femtomolar <!--del_lnk--> dissociation constant (&lt;10<sup>-15</sup> M) but does not bind at all to its amphibian homolog <!--del_lnk--> onconase (&gt;1 M). Extremely minor chemical changes such as the addition of a single methyl group to a binding partner can sometimes suffice to nearly eliminate binding; for example, the <!--del_lnk--> aminoacyl tRNA synthetase specific to the amino acid <!--del_lnk--> valine discriminates against the very similar side chain of the amino acid <!--del_lnk--> isoleucine.<p>Proteins can bind to other proteins as well as to small-molecule substrates. When proteins bind specifically to other copies of the same molecule, they can <!--del_lnk--> oligomerize to form fibrils; this process occurs often in structural proteins that consist of globular monomers that self-associate to form rigid fibers. <!--del_lnk--> Protein-protein interactions also regulate enzymatic activity, control progression through the <!--del_lnk--> cell cycle, and allow the assembly of large <!--del_lnk--> protein complexes that carry out many closely related reactions with a common biological function. Proteins can also bind to, or even be integrated into, cell membranes. The ability of binding partners to induce conformational changes in proteins allows the construction of enormously complex <!--del_lnk--> signaling networks.<p><a id="Enzymes" name="Enzymes"></a><h3> <span class="mw-headline">Enzymes</span></h3>
      <dl>
       <dd>
      </dl>
      <p>The best-known role of proteins in the cell is their duty as <!--del_lnk--> enzymes, which <!--del_lnk--> catalyze chemical reactions. Enzymes are usually highly specific catalysts that accelerate only one or a few chemical reactions. Enzymes effect most of the reactions involved in <!--del_lnk--> metabolism and <!--del_lnk--> catabolism as well as <!--del_lnk--> DNA replication, <a href="../../wp/d/DNA_repair.htm" title="DNA repair">DNA repair</a>, and <!--del_lnk--> RNA synthesis. Some enzymes act on other proteins to add or remove chemical groups in a process known as <!--del_lnk--> post-translational modification. About 4,000 reactions are known to be catalyzed by enzymes. The rate acceleration conferred by enzymatic catalysis is often enormous - as much as 10<sup>17</sup>-fold increase in rate over the uncatalyzed reaction in the case of <!--del_lnk--> orotate decarboxylase.<p>The molecules bound and acted upon by enzymes are known as <!--del_lnk--> substrates. Although enzymes can consist of hundreds of amino acids, it is usually only a small fraction of the residues that come in contact with the substrate. and an even smaller fraction - 3-4 residues on average - that are directly involved in catalysis. The region of the enzyme that binds the substrate and contains the catalytic residues is known as the <!--del_lnk--> active site.<p><a id="Cell_signalling_and_ligand_transport" name="Cell_signalling_and_ligand_transport"></a><h3> <span class="mw-headline">Cell signalling and ligand transport</span></h3>
      <div class="thumb tright">
       <div style="width:152px;"><a class="internal" href="../../images/210/21097.png.htm" title="A mouse antibody against cholera that binds a carbohydrate antigen."><img alt="A mouse antibody against cholera that binds a carbohydrate antigen." height="224" longdesc="/wiki/Image:Mouse-cholera-antibody-1f4x.png" src="../../images/210/21097.png" width="150" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/210/21097.png.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> A <!--del_lnk--> mouse antibody against <a href="../../wp/c/Cholera.htm" title="Cholera">cholera</a> that binds a <!--del_lnk--> carbohydrate antigen.</div>
       </div>
      </div>
      <p>Many proteins are involved in the process of <!--del_lnk--> cell signaling and <!--del_lnk--> signal transduction. Some proteins, such as <a href="../../wp/i/Insulin.htm" title="Insulin">insulin</a>, are extracellular proteins that transmit a signal from the cell in which they were synthesized to other cells in distant <!--del_lnk--> tissues. Others are <!--del_lnk--> membrane proteins that act as <!--del_lnk--> receptors whose main function is to bind a signaling molecule and induce a biochemical response in the cell. Many receptors are <!--del_lnk--> membrane proteins that have a binding site exposed on the cell surface and an effector domain within the cell, which may have enzymatic activity or may undergo a <!--del_lnk--> conformational change detected by other proteins within the cell.<p><!--del_lnk--> Antibodies are protein components of <!--del_lnk--> adaptive immune system whose main function is to bind <!--del_lnk--> antigens, or foreign substances in the body, and target them for destruction. Antibodies can be <!--del_lnk--> secreted into the extracellular environment or anchored in the membranes of specialized <!--del_lnk--> B cells known as <!--del_lnk--> plasma cells. While enzymes are limited in their binding affinity for their substrates by the necessity of conducting their reaction, antibodies have no such constraints. An antibody&#39;s binding affinity to its target is extraordinarily high.<p>Many ligand transport proteins bind particular small biomolecules and transport them to other locations in the body of a multicellular organism. These proteins must have a high binding affinity when their <!--del_lnk--> ligand is present in high concentrations but must also release the ligand when it is present at low concentrations in the target tissues. The canonical example of a ligand-binding protein is <!--del_lnk--> haemoglobin, which transports <a href="../../wp/o/Oxygen.htm" title="Oxygen">oxygen</a> from the <!--del_lnk--> lungs to other organs and tissues in all <a href="../../wp/v/Vertebrate.htm" title="Vertebrate">vertebrates</a> and has close <!--del_lnk--> homologs in every biological <!--del_lnk--> kingdom.<p><!--del_lnk--> Transmembrane proteins can also serve as ligand transport proteins that alter the <!--del_lnk--> permeability of the cell&#39;s membrane to small molecules and ions. The membrane alone has a <!--del_lnk--> hydrophobic core through which <!--del_lnk--> polar or charged molecules cannot <!--del_lnk--> diffuse. Membrane proteins contain internal channels that allow such molecules to enter and exit the cell. Many <!--del_lnk--> ion channel proteins are specialized to select for only a particular ion; for example, <a href="../../wp/p/Potassium.htm" title="Potassium">potassium</a> and <a href="../../wp/s/Sodium.htm" title="Sodium">sodium</a> channels often discriminate for only one of the two ions.<p><a id="Structural_proteins" name="Structural_proteins"></a><h3> <span class="mw-headline">Structural proteins</span></h3>
      <p>Structural proteins confer stiffness and rigidity to otherwise fluid biological components. Most structural proteins are <!--del_lnk--> fibrous proteins; for example, <!--del_lnk--> actin and <!--del_lnk--> tubulin are globular and soluble as monomers but <!--del_lnk--> polymerize to form long, stiff fibers that comprise the <!--del_lnk--> cytoskeleton, which allows the cell to maintain its shape and size. <!--del_lnk--> Collagen and <!--del_lnk--> elastin are critical components of <!--del_lnk--> connective tissue such as <!--del_lnk--> cartilage, and <!--del_lnk--> keratin is found in hard or filamentous structures such as <!--del_lnk--> hair, <!--del_lnk--> nails, <a href="../../wp/f/Feather.htm" title="Feather">feathers</a>, <!--del_lnk--> hooves, and some <!--del_lnk--> animal shells.<p>Other proteins that serve structural functions are <!--del_lnk--> motor proteins such as <!--del_lnk--> myosin, <!--del_lnk--> kinesin, and <!--del_lnk--> dynein, which are capable of generating mechanical forces. These proteins are crucial for cellular <!--del_lnk--> motility of single-celled organisms and the <!--del_lnk--> sperm of many sexually reproducing multicellular organisms. They also generate the forces exerted by contracting <!--del_lnk--> muscles.<p><a id="Methods_of_study" name="Methods_of_study"></a><h2> <span class="mw-headline">Methods of study</span></h2>
      <dl>
       <dd>
      </dl>
      <p>As some of the most commonly studied biological molecules, the activities and structures of proteins are examined both <i>in vitro</i> and <i>in vivo</i>. <i>In vitro</i> studies of purified proteins in controlled environments are useful for learning how a protein carries out its function: for example, <a href="../../wp/e/Enzyme_kinetics.htm" title="Enzyme kinetics">enzyme kinetics</a> studies explore the <!--del_lnk--> chemical mechanism of an enzyme&#39;s catalytic activity and its relative affinity for various possible substrate molecules. By contrast, <i>in vivo</i> experiments on proteins&#39; activities within cells or even within whole organisms can provide complementary information about where a protein functions and how it is regulated.<p><a id="Protein_purification" name="Protein_purification"></a><h3> <span class="mw-headline">Protein purification</span></h3>
      <dl>
       <dd>
      </dl>
      <p>In order to perform <i>in vitro</i> analyses, a protein must be purified away from other cellular components. This process usually begins with <!--del_lnk--> cell lysis, in which a cell&#39;s membrane is disrupted and its internal contents released into a solution known as a <!--del_lnk--> crude lysate. The resulting mixture can be purified using <!--del_lnk--> ultracentrifugation, which fractionates the various cellular components into fractions containing soluble proteins; membrane <a href="../../wp/l/Lipid.htm" title="Lipid">lipids</a> and proteins; cellular <!--del_lnk--> organelles, and <!--del_lnk--> nucleic acids. <!--del_lnk--> Precipitation by a method known as <!--del_lnk--> salting out can concentrate the proteins from this lysate. Various types of <!--del_lnk--> chromatography are then used to isolate the protein or proteins of interest based on properties such as molecular weight, net charge and binding affinity. The level of purification can be monitored using <!--del_lnk--> gel electrophoresis if the desired protein&#39;s molecular weight is known, by <!--del_lnk--> spectroscopy if the protein has distinguishable spectroscopic features, or by <!--del_lnk--> enzyme assays if the protein has enzymatic activity.<p>For natural proteins, a series of purification steps may be necessary to obtain protein sufficiently pure for laboratory applications. To simplify this process, <!--del_lnk--> genetic engineering is often used to add chemical features to proteins that make them easier to purify without affecting their structure or activity. Here, a &quot;tag&quot; consisting of a specific amino acid sequence, often a series of <!--del_lnk--> histidine residues (a &quot;<!--del_lnk--> His-tag&quot;), is attached to one terminus of the protein. As a result, when the lysate is passed over a chromatography column containing <a href="../../wp/n/Nickel.htm" title="Nickel">nickel</a>, the histidine residues ligate the nickel and attach to the column while the untagged components of the lysate pass unimpeded.<p><a id="Cellular_localization" name="Cellular_localization"></a><h3> <span class="mw-headline">Cellular localization</span></h3>
      <div class="thumb tright">
       <div style="width:302px;"><a class="internal" href="../../images/210/21098.jpg.htm" title="Proteins in different cellular compartments and structures tagged with green fluorescent protein."><img alt="Proteins in different cellular compartments and structures tagged with green fluorescent protein." height="395" longdesc="/wiki/Image:Localisations02eng.jpg" src="../../images/210/21098.jpg" width="300" /></a><div class="thumbcaption">
         <div class="magnify" style="float:right"><a class="internal" href="../../images/210/21098.jpg.htm" title="Enlarge"><img alt="Enlarge" height="11" src="../../images/0/1.png" width="15" /></a></div> Proteins in different <!--del_lnk--> cellular compartments and structures tagged with <!--del_lnk--> green fluorescent protein.</div>
       </div>
      </div>
      <p>The study of proteins <i>in vivo</i> is often concerned with the synthesis and localization of the protein within the cell. Although many intracellular proteins are synthesized in the <!--del_lnk--> cytoplasm and membrane-bound or secreted proteins in the <!--del_lnk--> endoplasmic reticulum, the specifics of how proteins are <!--del_lnk--> targeted to specific organelles or cellular structures is often unclear. A useful technique for assessing cellular localization uses genetic engineering to express in a cell a <!--del_lnk--> fusion protein or <!--del_lnk--> chimera consisting of the natural protein of interest linked to a &quot;<!--del_lnk--> reporter&quot; such as <!--del_lnk--> green fluorescent protein (GFP). The fused protein&#39;s position within the cell can be cleanly and efficiently visualized using <!--del_lnk--> microscopy, as shown in the figure opposite.<p>Through another genetic engineering application known as <!--del_lnk--> site-directed mutagenesis, researchers can alter the protein sequence and hence its structure, cellular localization, and susceptibility to regulation, which can be followed <i>in vivo</i> by GFP tagging or <i>in vitro</i> by <a href="../../wp/e/Enzyme_kinetics.htm" title="Enzyme kinetics">enzyme kinetics</a> and binding studies.<p><a id="Proteomics_and_bioinformatics" name="Proteomics_and_bioinformatics"></a><h3> <span class="mw-headline">Proteomics and bioinformatics</span></h3>
      <dl>
       <dd>
      </dl>
      <p>The total complement of proteins present in a cell or cell type is known as its <!--del_lnk--> proteome, and the study of such large-scale data sets defines the field of <!--del_lnk--> proteomics, named by analogy to the related field of <!--del_lnk--> genomics. Key experimental techniques in proteomics include <!--del_lnk--> protein microarrays, which allow the detection of the relative levels of a large number of proteins present in a cell, and <!--del_lnk--> two-hybrid screening, which allows the systematic exploration of <!--del_lnk--> protein-protein interactions. The total complement of biologically possible such interactions is known as the <!--del_lnk--> interactome. A systematic attempt to determine the structures of proteins representing every possible fold is known as <!--del_lnk--> structural genomics.<p>The large amount of genomic and proteomic data available for a variety of organisms, including the <!--del_lnk--> human genome, allows researchers to efficiently identify <!--del_lnk--> homologous proteins in distantly related organisms by <a href="../../wp/s/Sequence_alignment.htm" title="Sequence alignment">sequence alignment</a>. <!--del_lnk--> Sequence profiling tools can perform more specific sequence manipulations such as <!--del_lnk--> restriction enzyme maps, <!--del_lnk--> open reading frame analyses for <!--del_lnk--> nucleotide sequences, and <!--del_lnk--> secondary structure prediction. From this data <!--del_lnk--> phylogenetic trees can be constructed and <a href="../../wp/e/Evolution.htm" title="Evolution">evolutionary</a> hypotheses developed using special software like <!--del_lnk--> ClustalW regarding the ancestry of modern organisms and the genes they express. The field of <a href="../../wp/b/Bioinformatics.htm" title="Bioinformatics">bioinformatics</a> seeks to assemble, annotate, and analyze genomic and proteomic data, applying <a href="../../wp/c/Computer_science.htm" title="Computer science">computational</a> techniques to biological problems such as <!--del_lnk--> gene finding and <!--del_lnk--> cladistics.<p><a id="Structure_prediction_and_simulation" name="Structure_prediction_and_simulation"></a><h3> <span class="mw-headline">Structure prediction and simulation</span></h3>
      <p>Complementary to the field of structural genomics, <!--del_lnk--> protein structure prediction seeks to develop efficient ways to provide plausible models for proteins whose structures have not yet been determined experimentally. The most successful type of structure prediction, known as <!--del_lnk--> homology modeling, relies on the existence of a &quot;template&quot; structure with sequence similarity to the protein being modeled; structural genomics&#39; goal is to provide sufficient representation in solved structures to model most of those that remain. Although producing accurate models remains a challenge when only distantly related template structures are available, it has been suggested that sequence alignment is the bottleneck in this process, as quite accurate models can be produced if a &quot;perfect&quot; sequence alignment is known. Many structure prediction methods have served to inform the emerging field of <!--del_lnk--> protein engineering, in which novel protein folds have already been designed. A more complex computational problem is the prediction of intermolecular interactions, such as in <!--del_lnk--> molecular docking and <!--del_lnk--> protein-protein interaction prediction.<p>The processes of protein folding and binding can be simulated using techniques derived from <!--del_lnk--> molecular dynamics, which increasingly take advantage of <!--del_lnk--> distributed computing as in the <!--del_lnk--> Folding@Home project. The folding of small alpha-helical protein domains such as the <!--del_lnk--> villin headpiece and the <a href="../../wp/h/HIV.htm" title="HIV">HIV</a> accessory protein have been successfully simulated <i>in silico</i>, and hybrid methods that combine standard molecular dynamics with <a href="../../wp/q/Quantum_mechanics.htm" title="Quantum mechanics">quantum mechanics</a> calculations have allowed exploration of the electronic states of <!--del_lnk--> rhodopsins.<p><a id="Nutrition" name="Nutrition"></a><h2> <span class="mw-headline">Nutrition</span></h2>
      <p>Most <!--del_lnk--> microorganisms and plants can biosynthesize all 20 standard amino acids, while animals must obtain some of the amino acids from the <!--del_lnk--> diet. Key enzymes in the biosynthetic pathways that synthesize certain amino acids - such as <!--del_lnk--> aspartokinase, which catalyzes the first step in the synthesis of <!--del_lnk--> lysine, <!--del_lnk--> methionine, and <!--del_lnk--> threonine from <!--del_lnk--> aspartate - are not present in animals. The amino acids that an organism cannot synthesize on its own are referred to as <!--del_lnk--> essential amino acids. (This designation is often used to specifically identify those essential to <a href="../../wp/h/Human.htm" title="Human">humans</a>.) If amino acids are present in the environment, most microorganisms can conserve energy by taking up the amino acids from the environment and downregulating their own biosynthetic pathways. <a href="../../wp/b/Bacteria.htm" title="Bacteria">Bacteria</a> are often <!--del_lnk--> engineered in the laboratory to lack the genes necessary for synthesizing a particular amino acid, providing a <!--del_lnk--> selectable marker for the success of <!--del_lnk--> transfection, or the introduction of foreign DNA.<p>In animals, amino acids are obtained through the consumption of foods containing protein. Ingested proteins are broken down through <!--del_lnk--> digestion, which typically involves <!--del_lnk--> denaturation of the protein through exposure to <!--del_lnk--> acid and degradation by the action of enzymes called <!--del_lnk--> proteases. Ingestion of essential amino acids is critical to the health of the organism, since the biosynthesis of proteins that include these amino acids is inhibited by their low concentration. Amino acids are also an important dietary source of <a href="../../wp/n/Nitrogen.htm" title="Nitrogen">nitrogen</a>. Some ingested amino acids, especially those that are not essential, are not used directly for protein biosynthesis. Instead, they are converted to <!--del_lnk--> carbohydrates through <!--del_lnk--> gluconeogenesis, which is also used under <!--del_lnk--> starvation conditions to generate <!--del_lnk--> glucose from the body&#39;s own proteins, particularly those found in <!--del_lnk--> muscle.<p><a id="History" name="History"></a><h2> <span class="mw-headline">History</span></h2>
      <p>Proteins were recognized as a distinct class of biological molecules in the eighteenth century by <!--del_lnk--> Antoine Fourcroy and others. Members of this class (called the &quot;albuminoids&quot;, <i>Eiweissk&ouml;rper</i>, or <i>mati&egrave;res albuminoides</i>) were recognized by their ability to <!--del_lnk--> coagulate or <!--del_lnk--> flocculate under various treatments such as heat or acid; well-known examples at the start of the nineteenth century included albumen from <!--del_lnk--> egg whites, <!--del_lnk--> blood <!--del_lnk--> serum albumin, <!--del_lnk--> fibrin, and <a href="../../wp/w/Wheat.htm" title="Wheat">wheat</a> <!--del_lnk--> gluten. The similarity between the cooking of egg whites and the curdling of milk was recognized even in ancient times; for example, the name <i>albumen</i> for the egg-white protein was coined by <!--del_lnk--> Pliny the Elder from the <!--del_lnk--> Latin <i>albus ovi</i> (egg white).<p>With the advice of <!--del_lnk--> J&ouml;ns Jakob Berzelius, the Dutch chemist <!--del_lnk--> Gerhardus Johannes Mulder carried out <!--del_lnk--> elemental analyses of common animal and plant proteins. To everyone&#39;s surprise, all proteins had nearly the same <!--del_lnk--> empirical formula, roughly C<sub>400</sub>H<sub>620</sub>N<sub>100</sub>O<sub>120</sub> with individual sulfur and phosphorus atoms. Mulder published his findings in two papers (1837,1838) and hypothesized that there was one basic substance (<i>Grundstoff</i>) of proteins, and that it was synthesized by plants and absorbed from them by animals in digestion. Berzelius was an early proponent of this theory and proposed the name &quot;protein&quot; for this substance in a letter dated <!--del_lnk--> 10 July 1838<blockquote>
       <p>The name protein that I propose for the organic oxide of <!--del_lnk--> fibrin and <!--del_lnk--> albumin, I wanted to derive from [the <!--del_lnk--> Greek word] &pi;&rho;&omega;&tau;&epsilon;&iota;&omicron;&sigmaf;, because it appears to be the primitive or principal substance of animal nutrition.</blockquote>
      <p>Mulder went on to identify the products of protein degradation such as the <!--del_lnk--> amino acid, <!--del_lnk--> leucine, for which he found a (nearly correct) molecular weight of 131 <!--del_lnk--> Da.<p>The minimum molecular weight suggested by Mulder&#39;s analyses was roughly 9 <!--del_lnk--> kDa, hundreds of times larger than other molecules being studied. Hence, the chemical structure of proteins (their <!--del_lnk--> primary structure) was an active area of research until 1949, when <!--del_lnk--> Fred Sanger sequenced <a href="../../wp/i/Insulin.htm" title="Insulin">insulin</a>. The (correct) theory that proteins were linear polymers of <!--del_lnk--> amino acids linked by <!--del_lnk--> peptide bonds was proposed independently and simultaneously by <!--del_lnk--> Franz Hofmeister and <!--del_lnk--> Emil Fischer at the same conference in 1902. However, some scientists were sceptical that such long <!--del_lnk--> macromolecules could be stable in solution. Consequently, numerous alternative theories of the protein <!--del_lnk--> primary structure were proposed, e.g., the colloidal hypothesis that proteins were assemblies of small molecules, the <!--del_lnk--> cyclol hypothesis of <!--del_lnk--> Dorothy Wrinch, the diketopiperazine hypothesis of <!--del_lnk--> Emil Abderhalden and the pyrrol/piperidine hypothesis of Troensgard (1942). Most of these theories had difficulties in accounting for the fact that the digestion of proteins yielded <!--del_lnk--> peptides and <!--del_lnk--> amino acids. Proteins were finally shown to be macromolecules of well-defined composition (and not colloidal mixtures) by <!--del_lnk--> Theodor Svedberg using <!--del_lnk--> analytical ultracentrifugation. The possibility that some proteins are non-covalent associations of such macromolecules was shown by <!--del_lnk--> Gilbert Smithson Adair (by measuring the <!--del_lnk--> osmotic pressure of <!--del_lnk--> hemoglobin) and, later, by <!--del_lnk--> Frederic M. Richards in his studies of ribonuclease S. The <!--del_lnk--> mass spectrometry of proteins has long been a useful technique for identifying <!--del_lnk--> posttranslational modifications and, more recently, for probing protein structure.<p>Most proteins are difficult to <!--del_lnk--> purify in more than milligram quantities, even using the most modern methods. Hence, early studies focused on proteins that could be purified in large quantities, e.g., those of <!--del_lnk--> blood, <!--del_lnk--> egg white, various <!--del_lnk--> toxins, and digestive/metabolic enzymes obtained from <!--del_lnk--> slaughterhouses. Many techniques of protein purification were developed during <a href="../../wp/w/World_War_II.htm" title="World War II">World War II</a> in a project led by <!--del_lnk--> Edwin Joseph Cohn to purify blood proteins to help keep soldiers alive. In the late 1950&#39;s, the <!--del_lnk--> Armour Hot Dog Co. purified 1 kg (= one million milligrams) of pure bovine pancreatic <!--del_lnk--> ribonuclease A and made it freely available to scientists around the world. This generous act made RNase A the main protein for basic research for the next few decades, resulting in several Nobel Prizes.<p>The study of protein folding began in 1910 with a famous paper by Henrietta Chick and C. J. Martin, in which they showed that the <!--del_lnk--> flocculation of a protein was composed of two distinct processes: the <!--del_lnk--> precipitation of a protein from solution was <i>preceded</i> by another process called <!--del_lnk--> denaturation, in which the protein became much less soluble, lost its enzymatic activity and became more chemically reactive. In the mid-1920&#39;s, <!--del_lnk--> Tim Anson and <!--del_lnk--> Alfred Mirsky proposed that denaturation was a reversible process, a correct hypothesis that was initially lampooned by some scientists as &quot;unboiling the egg&quot;. Anson also suggested that denaturation was a two-state (&quot;all-or-none&quot;) process, in which one fundamental molecular transition resulted in the drastic changes in solubility, enzymatic activity and chemical reactivity; he further noted that the free energy changes upon denaturation were much smaller than those typically involved in chemical reactions. In 1929, <!--del_lnk--> Hsien Wu hypothesized that denaturation was protein folding, a purely conformational change that resulted in the exposure of amino acid side chains to the solvent. According to this (correct) hypothesis, exposure of aliphatic and reactive side chains to solvent rendered the protein less soluble and more reactive, whereas the loss of a specific conformation caused the loss of enzymatic activity. Although considered plausible, Wu&#39;s hypothesis was not immediately accepted, since so little was known of protein structure and enzymology and other factors could account for the changes in solubility, enzymatic activity and chemical reactivity. In the early 1960&#39;s, <!--del_lnk--> Chris Anfinsen showed that the folding of <!--del_lnk--> ribonuclease A was fully reversible with no external cofactors needed, verifying the &quot;thermodynamic hypothesis&quot; of protein folding that the folded state represents the global minimum of <!--del_lnk--> free energy for the protein.<p>The hypothesis of protein folding was followed by research into the physical interactions that stabilize folded protein structures. The crucial role of <!--del_lnk--> hydrophobic interactions was hypothesized by <!--del_lnk--> Dorothy Wrinch and <!--del_lnk--> Irving Langmuir, as a mechanism that might stabilize her <!--del_lnk--> cyclol structures. Although supported by <!--del_lnk--> J. D. Bernal and others, this (correct) hypothesis was rejected along with the cyclol hypothesis, which was disproven in the 1930&#39;s by <a href="../../wp/l/Linus_Pauling.htm" title="Linus Pauling">Linus Pauling</a> (among others). Instead, Pauling championed the idea that protein structure was stabilized mainly by <!--del_lnk--> hydrogen bonds, an idea advanced initially by <!--del_lnk--> William Astbury (1933). Remarkably, Pauling&#39;s incorrect theory about H-bonds resulted in his <i>correct</i> models for the <!--del_lnk--> secondary structure elements of proteins, the <!--del_lnk--> alpha helix and the <!--del_lnk--> beta sheet. The hydrophobic interaction was restored to its correct prominence by a famous article in 1959 by <!--del_lnk--> Walter Kauzman on <!--del_lnk--> denaturation, based partly on work by <!--del_lnk--> Kaj Linderstrom-Lang. The ionic nature of proteins was demonstrated by Bjerrum, Weber and <!--del_lnk--> Arne Tiselius, but Linderstrom-Lang showed that the charges were generally accessible to solvent and not bound to each other (1949).<p>The <!--del_lnk--> secondary and low-resolution <!--del_lnk--> tertiary structure of globular proteins was investigated initially by hydrodynamic methods, such as <!--del_lnk--> analytical ultracentrifugation and <!--del_lnk--> flow birefringence. Spectroscopic methods to probe protein structure (such as <!--del_lnk--> circular dichroism, fluorescence, near-ultraviolet and infrared absorbance) were developed in the 1950&#39;s. The first atomic-resolution structures of proteins were solved by <!--del_lnk--> X-ray crystallography in the 1960&#39;s and by <!--del_lnk--> NMR in the 1980&#39;s. As of 2006, the <!--del_lnk--> Protein Data Bank has nearly 40,000 atomic-resolution structures of proteins. In more recent times, <!--del_lnk--> cryo-electron microscopy of large macromolecular assemblies and computational <!--del_lnk--> protein structure prediction of small protein <!--del_lnk--> domains are two methods approaching atomic resolution.<p>
       <br />
      <p><a id="See_also" name="See_also"></a><div class="printfooter"> Retrieved from &quot;<!--del_lnk--> http://en.wikipedia.org/wiki/Protein&quot;</div>
      <!-- end content -->
      <div class="visualClear">
      </div>
     </div>
    </div>
   </div>
   <!-- end of the left (by default at least) column -->
   <div class="visualClear">
   </div>
   <div id="footer">
    <div class="center"> This reference article is mainly selected from the English Wikipedia with only minor checks and changes (see www.wikipedia.org for details of authors and sources) and is available under the <nobr><a href="../../wp/w/Wikipedia_Text_of_the_GNU_Free_Documentation_License.htm">GNU Free Documentation License</a></nobr>. See also our <b><a href="../../disclaimer.htm">Disclaimer</a></b>. </div>
   </div>
   <script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
  </div>
  
 </body>
</html>
